<!--?xml version="1.0" encoding="utf-8"?--><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5287289/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Translation from unconventional 5′ start sites drives tumour initiation</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5287289/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature"><meta name="citation_title" content="Translation from unconventional 5′ start sites drives tumour initiation"><meta name="citation_authors" content="Ataman Sendoel, Joshua G. Dunn, Edwin H. Rodriguez, Shruti Naik, Nicholas C. Gomez, Brian Hurwitz, John Levorse, Brian D. Dill, Daniel Schramek, Henrik Molina, Jonathan S. Weissman, Elaine Fuchs"><meta name="citation_date" content="26 January 2017"><meta name="citation_issue" content="7638"><meta name="citation_volume" content="541"><meta name="citation_firstpage" content="494"><meta name="citation_doi" content="10.1038/nature21036"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5287289/?report=abstract"><meta name="citation_pmid" content="28077873"><meta name="DC.Title" content="Translation from unconventional 5′ start sites drives tumour initiation"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Ataman Sendoel"><meta name="DC.Contributor" content="Joshua G. Dunn"><meta name="DC.Contributor" content="Edwin H. Rodriguez"><meta name="DC.Contributor" content="Shruti Naik"><meta name="DC.Contributor" content="Nicholas C. Gomez"><meta name="DC.Contributor" content="Brian Hurwitz"><meta name="DC.Contributor" content="John Levorse"><meta name="DC.Contributor" content="Brian D. Dill"><meta name="DC.Contributor" content="Daniel Schramek"><meta name="DC.Contributor" content="Henrik Molina"><meta name="DC.Contributor" content="Jonathan S. Weissman"><meta name="DC.Contributor" content="Elaine Fuchs"><meta name="DC.Date" content="2017 Jan 26"><meta name="DC.Identifier" content="10.1038/nature21036"><meta name="DC.Language" content="en"><meta property="og:title" content="Translation from unconventional 5′ start sites drives tumour initiation"><meta property="og:type" content="article"><meta property="og:description" content="We are just beginning to understand how translational control affects tumour initiation <mark>and</mark> malignancy. Here we use an epidermis-specific, in vivo ribosome profiling strategy to investigate the translational landscape during the transition from normal ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5287289/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.8/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.8/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'/pmc/utils/ctxp/'"><script type="text/javascript" src="/corehtml/pmc/ctxp/jquery.citationexporter.js">//</script><link rel="stylesheet" href="/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5287289/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5287289/pdf/nihms845657.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="F4FC036F9A5ED16100000000004E0040">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/css/3411343/3882866/4157116.css" media="print"><style type="text/css" media="print">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }.print-log li { list-style-image: url('https://www.ncbi.nlm.nih.gov/stat?jsevent=print&ncbi_app=pmc&ncbi_db=pmc&ncbi_pcid=%2Farticles%2FPMC5287289%2F&ncbi_pdid=article&ncbi_phid=F4FC036F9A5ED16100000000004E0040'); }</style><style type="text/css">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none; "></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	&lt;p class="nojs"&gt;
	&lt;strong&gt;Warning:&lt;/strong&gt;
	The NCBI web site requires JavaScript to function. 
	&lt;a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank"&gt;more...&lt;/a&gt;
	&lt;/p&gt;
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="https://www.ncbi.nlm.nih.gov/account/?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5287289%2F" id="sign_in">Sign in to NCBI</a><a href="https://www.ncbi.nlm.nih.gov/account/signout/?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5287289%2F" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="toolkitbookgh">ToolKitBookgh</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input type="text" name="term" id="term" title="Search PMC. Use up <mark>and</mark> down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="OFF" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="#" style="visibility: hidden; "><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC5287289</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="small"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nature</a></span>. Author manuscript; available in PMC 2017 Feb 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28077873" target="pmc_ext" ref="reftype=publisher&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal"><span class="cit">Nature. 2017 Jan 26; 541(7638): 494–499. </span></a></div><span class="fm-vol-iss-date">Published online 2017 Jan 11. </span> <span class="doi">doi:&nbsp; <a href="//dx.doi.org/10.1038%2Fnature21036" target="pmc_ext" ref="reftype=other&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">10.1038/nature21036</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5287289</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS845657</span></div></div></div></div></div><h1 class="content-title">Translation from unconventional 5′ start sites drives tumour initiation</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sendoel%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Ataman Sendoel</a>,<sup>1</sup> <a href="/pubmed/?term=Dunn%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Joshua G. Dunn</a>,<sup>2</sup> <a href="/pubmed/?term=Rodriguez%20EH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Edwin H. Rodriguez</a>,<sup>2</sup> <a href="/pubmed/?term=Naik%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Shruti Naik</a>,<sup>1</sup> <a href="/pubmed/?term=Gomez%20NC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Nicholas C. Gomez</a>,<sup>1</sup> <a href="/pubmed/?term=Hurwitz%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Brian Hurwitz</a>,<sup>1</sup> <a href="/pubmed/?term=Levorse%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">John Levorse</a>,<sup>1</sup> <a href="/pubmed/?term=Dill%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Brian D. Dill</a>,<sup>3</sup> <a href="/pubmed/?term=Schramek%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Daniel Schramek</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Molina%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Henrik Molina</a>,<sup>3</sup> <a href="/pubmed/?term=Weissman%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Jonathan S. Weissman</a>,<sup>2</sup> <mark>and</mark>  <a href="/pubmed/?term=Fuchs%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28077873">Elaine Fuchs</a><sup>1</sup></div></div><div class="fm-panel small half_rhythm"><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139773359012896_ai" style="display:none"><div class="fm-affl" lang="en"><sup>1</sup>Robin Chemers Neustein Laboratory of Mammalian Development <mark>and</mark> Cell Biology, Howard Hughes Medical Institute, The Rockefeller University, New York, New York 10065, USA</div><div class="fm-affl" lang="en"><sup>2</sup>Department of Cellular <mark>and</mark> Molecular Pharmacology, Howard Hughes Medical Institute, University of California, San Francisco, California 94158, USA</div><div class="fm-affl" lang="en"><sup>3</sup>Proteomics Resource Center, The Rockefeller University, New York, New York 10065, USA</div><div id="FN1">Correspondence <mark>and</mark> requests for materials should be addressed to E.F. (<a href="mailto:dev@null" data-email="ude.rellefekcor@blshcuf" class="oemail">ude.rellefekcor@blshcuf</a>)</div><div id="FN2"><sup>†</sup>Present address: The Lunenfeld-Tanenbaum Research Institute, Mount Sinai Hospital, Toronto M5G 1X5, Canada.</div></div><div class="togglers"><a href="#" class="pmctoggle" rid="idm139773359012896_ai">Author information <span>►</span></a> <a href="#" class="pmctoggle" rid="idm139773359012896_cpl">Copyright <mark>and</mark> License information <span>►</span></a></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="fm-cpl-info fm-panel hide half_rhythm" id="idm139773359012896_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a>  <mark>and</mark> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-copyright half_rhythm"><div class="license">Reprints <mark>and</mark> permissions information is available at <a href="http://www.nature.com/reprints" ref="reftype=extlink&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">www.nature.com/reprints</a></div></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="small"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=28077873" target="pmc_ext" ref="reftype=publisher&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal">Nature</a></div></div></div></div><div class="sec"></div><div id="__abstractidm139773288589168" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__abstractidm139773288589168title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">We are just beginning to understand how translational control affects tumour initiation <mark>and</mark> malignancy. Here we use an epidermis-specific, <em>in vivo</em> ribosome profiling strategy to investigate the translational landscape during the transition from normal homeostasis to malignancy. Using a mouse model of inducible SOX2, which is broadly expressed in oncogenic RAS-associated cancers, we show <mark>that</mark> despite widespread reductions in translation <mark>and</mark> protein synthesis, certain oncogenic mRNAs are spared. During tumour initiation, the translational apparatus is redirected towards unconventional upstream initiation sites, enhancing the translational efficiency of oncogenic mRNAs. An <em>in vivo</em> RNA interference screen of translational regulators revealed <mark>that</mark> depletion of conventional eIF2 complexes has adverse effects on normal but not oncogenic growth. Conversely, the alternative initiation factor eIF2A is essential for cancer progression, during which it mediates initiation at these upstream sites, differentially skewing translation <mark>and</mark> protein expression. Our findings unveil a role for the translation of 5′ untranslated regions in cancer, <mark>and</mark> expose new targets for therapeutic intervention.</p></div></div><div id="__bodyidm139773288586496" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Translational control is a key determinant of protein abundance, which in turn defines cellular states<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078341">1</a></sup>. Its impact may intensify during the transition from homeostasis to malignancy, as revealed by the surprisingly low correlations between mRNA <mark>and</mark> protein levels in genome-wide human cancer databases<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078390">2</a></sup>. Moreover, oncogenic drivers, such as mTOR, c-MYC <mark>and</mark> RAS, can influence the activity of eukaryotic initiation factors (eIFs) <mark>and</mark> ribosomal proteins<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078366">3</a>–<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078340">6</a></sup>. Thus, by generating aberrant downstream networks of translational regulators, oncogenes might impose altered protein synthesis programs <mark>that</mark> become the driving force for tumour formation <mark>and</mark> malignant progression.</p><p id="P3" class="p">Here we test this hypothesis by focusing on squamous cell carcinomas (SCCs), which are among the most common <mark>and</mark> life-threatening cancers worldwide. In mice, the RAS–MAPK pathway is essential for benign tumours <mark>and</mark> SCCs<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078361">7</a></sup>. Downstream of RAS<sup>G12V</sup>–MAPK is SOX2, an essential transcription factor induced by SCC-initiating (stem) cells<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078368">8</a>–<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078337">11</a></sup>. Notably, <em>SOX2</em> is also recurrently amplified in human SCCs of the lung, head <mark>and</mark> neck, oesophagus <mark>and</mark> cervix<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078403">12</a></sup>. Given its broad effect on these cancers, we used an established, inducible SOX2 mouse model to interrogate its effects on translational regulation of skin epidermis, at a time preceding overt phenotypic <mark>and</mark> proliferative changes associated <mark>with</mark> tumorigenesis. Our studies led us to an unexpected shift to unconventional translation <mark>that</mark> functions crucially in tumour initiation.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Translational landscapes</h2><p id="P4" class="p p-first">Embryonic epidermis is an excellent model for studying a rapidly growing tissue <mark>that</mark> relies on a fine-tuned balance between proliferation <mark>and</mark> differentiation<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078379">13</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078385">14</a></sup>. To assess how SOX2 perturbs this balance, we crossed <em>R26-LSL-Sox2-IRES-eGFP<sup>fl/fl</sup></em> <mark>and</mark> <em>K14-cre</em><sup>+/</sup><em><sup>wt</sup></em> mice<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078334">15</a></sup>, yielding newborn litters <mark>with</mark> unaffected (green fluorescent protein (GFP)-negative; GFP<sup>−</sup>) or SOX2<em>-</em>expressing (GFP<sup>+</sup>) epidermal progenitors. We focused on postnatal days 0–4 (P0–P4) when SOX2<sup>+</sup> epidermis was pre-phenotypic (<a href="/pmc/articles/PMC5287289/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a, b</span></a>, <a href="/pmc/articles/PMC5287289/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Extended Data Fig. 1a</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a class="icnblk_img figpopup" href="/pmc/articles/PMC5287289/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><img src="/pmc/articles/PMC5287289/bin/nihms845657f1.gif" class="small-thumb" alt="Figure 1" title="Figure 1" src-large="/pmc/articles/PMC5287289/bin/nihms845657f1.jpg"></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC5287289/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a4--><div><span><!--caption a8--><strong>The translational landscape of the epidermis during premalignant transformation</strong></span></div></div></div><p id="P5">To determine the effect of SOX2 on translation, we adopted a strategy previously used to map ribosome-protected mRNA sequences in yeast <mark>and</mark> cultured cell lines<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078394">16</a>–<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078378">18</a></sup>. After treating freshly isolated skins <mark>with</mark> translation elongation inhibitor (cycloheximide), we prepared an epidermal suspension enriched for basal progenitors <mark>and</mark> then isolated <mark>and</mark> sequenced ribosome-protected mRNA fragments to generate a genome-wide <em>in vivo</em> epidermal translational landscape (<a href="/pmc/articles/PMC5287289/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Extended Data Fig. 1b</span></a>). In parallel, we performed ribosome profiling on HRAS<sup>G12V</sup>-transformed SCCs <mark>and</mark> primary keratinocytes, <mark>and</mark> we also mapped initiation sites using harringtonine, which blocks initiation during the first round of elongation<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078405">18</a></sup>. Ribosome-protected fragment-length assays generated a 31-nucleotide peak (<a href="/pmc/articles/PMC5287289/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1c</span></a>). Across deep-sequencing replicates, our <em>in vivo</em> ribosomal profiling was highly reproducible (<em>R</em><sup>2</sup> values &gt;0.9, <a href="/pmc/articles/PMC5287289/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Extended Data Fig. 1b, c</span></a>).</p><p id="P6">By conducting matched transcriptional RNA sequencing (RNA-seq) <mark>and</mark> ribosome profiling, we contrasted genome-wide transcriptional <mark>and</mark> translational differences. As confirmed by immunofluorescence, some genes, such as <em>Krt5</em> <mark>and</mark> <em>Krt14</em>, showed SOX2-independent translation, whereas others, such as <em>Krt6</em>, <em>Krt16</em> <mark>and</mark> <em>Sox2</em>, displayed increased translation in SOX2<sup>+</sup> epidermis<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078371">19</a></sup> (<a href="/pmc/articles/PMC5287289/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b, d–f</span></a>). Notably, 573 translatome-only differences were found, which included mRNAs encoding proteins such as <em>Cd44</em>, <em>Fos</em>, <em>Erbb3</em> <mark>and</mark> <em>Irs2</em> <mark>that</mark> have well-known functions in tumorigenesis. By performing parallel, comparative mass spectrometry, we found <mark>that</mark> ribosome profiling correlated well <mark>with</mark> SOX2-induced protein differences (Spearman correlation coefficient (<em>r</em><sub>s</sub>) =0.85, <a href="/pmc/articles/PMC5287289/figure/F8/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Figs 2a</span></a>, <a href="/pmc/articles/PMC5287289/figure/F16/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F16" rid-ob="ob-F16" co-legend-rid="lgnd_F16"><span>10b</span></a>).</p><p id="P7">Somatic stem cells <mark>and</mark> cancer-propagating cells have low protein synthesis rates, a feature implicated in driving stemness<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078388">20</a>,<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078347">21</a></sup>. Therefore, we sought to determine whether SOX2 <mark>and</mark> HRAS<sup>G12V</sup> also affected the cell-based rate of protein synthesis. We first used <em>O</em>-propargyl-puromycin (OPP) incorporation as a proxy for total protein synthesis<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078344">20</a></sup>. OPP led to a robust increase in fluorescence, which could be largely blocked by prior cycloheximide treatment. On the basis of this assay, pre-malignant <mark>and</mark> SCC SOX2<sup>+</sup> keratinocytes both showed lower protein synthesis rates than their wild-type counterparts (<a href="/pmc/articles/PMC5287289/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a, b</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a class="icnblk_img figpopup" href="/pmc/articles/PMC5287289/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><img src="/pmc/articles/PMC5287289/bin/nihms845657f2.gif" class="small-thumb" alt="Figure 2" title="Figure 2" src-large="/pmc/articles/PMC5287289/bin/nihms845657f2.jpg"></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC5287289/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a4--><div><span><!--caption a8--><strong>Overall protein synthesis <mark>and</mark> translational efficiency are decreased in premalignant <mark>and</mark> SCC states</strong></span></div></div></div><p id="P8">To explore further how SOX2 functions in translational regulation, we next focused on translational efficiency by determining the reads per kilobase of transcript per million mapped reads (RPKM) of coding sequences (CDS) in ribosome profiling versus the RPKM in exons of RNA-seq (RPKM<sub>ribosome profiling</sub>/RPKM<sub>RNA-seq</sub>) in wild-type <mark>and</mark> premalignant states. Overall, translational efficiency in SOX2<sup>+</sup> epidermis was markedly lower than in wild-type epidermis (<a href="/pmc/articles/PMC5287289/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2c</span></a>). Translational efficiency was also reduced in cultured SOX2-transformed keratinocytes, indicating <mark>that</mark> this difference was intrinsic to SOX2 levels (<a href="/pmc/articles/PMC5287289/figure/F8/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 2b</span></a>). Moreover, the cells grew comparably <em>in vivo</em>, ruling out proliferation rates as the root of these differences (<a href="/pmc/articles/PMC5287289/figure/F8/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 2c</span></a>). We also found no differences in the expression of phospho-eIF4E-BP1 or NSUN2, which could have contributed to reduced translational efficiency<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078355">21</a></sup> (<a href="/pmc/articles/PMC5287289/figure/F8/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 2d, e</span></a>).</p><p id="P9" class="p p-last">Reduced protein synthesis in pre-malignancy correlated well <mark>with</mark> reduced translational efficiency. Notably, however, a small cohort of mRNAs deviated from this trend. In particular, while increased transcripts were enriched for stress pathways, the cohort of efficiently translated mRNAs in SOX2<sup>+</sup> epidermis displayed a notable association <mark>with</mark> cancer-related pathways (<a href="/pmc/articles/PMC5287289/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2d</span></a>, <a href="/pmc/articles/PMC5287289/figure/F9/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Extended Data Fig. 3</span></a>). Thus, in the face of global translational reduction during the early stages of malignancy, a translationally controlled subset of cancer genes escaped this suppression by maintaining efficient translation.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Tumour-induced shifts in translation initiation</h2><p id="P10" class="p p-first">In addition to these differences in translational efficiency, there were also qualitative differences in the patterns of ribosomal occupancies. Oncogene-dependent differential skewing of ribosome occupancy was particularly evident within 5′ untranslated regions (UTRs). We identified distinct SOX2-induced patterns of translated upstream open reading frames (uORFs), reflected in their triplet periodicity<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078342">22</a></sup>, <mark>that</mark> were 5′ of annotated CDS (<a href="/pmc/articles/PMC5287289/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3a, b</span></a>, <a href="/pmc/articles/PMC5287289/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Figs 4</span></a>, <a href="/pmc/articles/PMC5287289/figure/F11/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,5a).</span></span><span>5a</span></a>). Quantification of 5′ UTR translation of 1,830 mRNAs revealed <mark>that</mark> uORF translation increased substantially in SOX2<sup>+</sup> compared to wild-type epidermis (median ratio 1.84), as measured by the ratio of ribosome profiling reads within the 5′ UTR relative to reads within the annotated downstream CDS.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a class="icnblk_img figpopup" href="/pmc/articles/PMC5287289/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><img src="/pmc/articles/PMC5287289/bin/nihms845657f3.gif" class="small-thumb" alt="Figure 3" title="Figure 3" src-large="/pmc/articles/PMC5287289/bin/nihms845657f3.jpg"></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC5287289/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a4--><div><span><!--caption a8--><strong>Translation is shifted towards upstream open reading frames in premalignant <mark>and</mark> SCC states</strong></span></div></div></div><p id="P11">Notably, some extensions in ribosomal coverage along 5′ UTRs in pre-malignant epidermis also generated peptide diversity (<a href="/pmc/articles/PMC5287289/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Figs 4</span></a>, <a href="/pmc/articles/PMC5287289/figure/F11/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,5).</span></span><span>5</span></a>). Previous studies underscored the difficulties in detecting uORF peptides by mass spectrometry<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078336">23</a></sup>, <mark>and</mark> indeed, even when we specifically enriched for N-terminal fragments (terminal amine isotopic labelling of substrates, TAILS), we only detected 13 uORF peptides.</p><p id="P12">Independent of whether uORFs are translated into stable peptides or N-terminal extensions, or represent stalled or poised ribosomes, they still could serve an important regulatory role. Moreover, upon scrutinizing 5′ UTRs of the top 10% of efficiently translated uORFs in SOX2<sup>+</sup> epidermis, we discovered a marked correlation between enhanced translation, increased length <mark>and</mark> decreased minimum free energy of 5′ UTRs (<a href="/pmc/articles/PMC5287289/figure/F12/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 6a, b</span></a>). When normalized for length, reduced folding energy still correlated <mark>with</mark> increased translation (<a href="/pmc/articles/PMC5287289/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3c</span></a>), indicating <mark>that</mark> SOX2-dependent re-direction towards uORF translation may target 5′ UTRs <mark>with</mark> increased secondary structure.</p><p id="P13">Pathway analysis revealed <mark>that</mark> highly translated uORFs in SOX2<sup>+</sup> epidermis have downstream ORFs involved in mechanisms of cancer, stem-cell pluripotency <mark>and</mark> Wnt/β-catenin signalling. This contrasted <mark>with</mark> wild-type epidermis, where eIF2 signalling was the primary target of differential uORF usage (<a href="/pmc/articles/PMC5287289/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3e</span></a>). Additionally, significant overlap existed between mRNAs <mark>with</mark> high uORF translation <mark>and</mark> mRNAs refractory to oncogenic suppression of translational efficiency (<a href="/pmc/articles/PMC5287289/figure/F12/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 6c</span></a>). Many of these SOX2-induced, ribosome-protected uORFs within 5′ UTRs displayed CUG, rather than the conventional AUG initiation site of canonical ORFs (<a href="/pmc/articles/PMC5287289/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3d</span></a>). For instance, nucleophosmin (<em>Npm1</em>), which is frequently mutated or translocated in cancer<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078392">24</a></sup>, exhibited increased ribosome occupancy in the 5′ upstream CUG uORF (<a href="/pmc/articles/PMC5287289/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3f</span></a>).</p><p id="P14" class="p p-last">These findings suggest <mark>that</mark> even before visible signs of tumorigenesis, the translational initiation apparatus is redirected towards upstream ORFs of a cohort of cancer-related mRNAs. If this hypothesis is correct, then we should see differential uORF usage in established cancers. We therefore examined uORF usage in a tumour allograft model, in which oncogenic HRAS<sup>G12V</sup> in combination <mark>with</mark> loss of TGFβ receptor II rapidly progresses in a SOX2-dependent manner into SCCs<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078400">25</a></sup> (<a href="/pmc/articles/PMC5287289/figure/F12/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 6d</span></a>). Indeed, when compared to keratinocytes from normal epidermis, SCC keratinocytes displayed an approximately 1.7-fold increase in uORF translation of mRNAs, similar to <mark>that</mark> seen <mark>with</mark> premalignant epidermis (<a href="/pmc/articles/PMC5287289/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3g</span></a>). Moreover, a notable overlap existed between uORFs regulated by SOX2 <em>in vivo</em> <mark>and</mark> HRAS<sup>G12V</sup>
<em>in vitro</em> (<a href="/pmc/articles/PMC5287289/figure/F12/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 6e</span></a>). Together, these findings underscore the existence of a mechanism to redirect the translational initiation machinery specifically in SCCs for the purpose of enhancing the relative 5′ UTR translational efficiency of the uORFs of certain cancer-related mRNAs.</p></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title"><em>In vivo</em> RNAi screen exposes an eIF2 regulatory node</h2><p id="P15" class="p p-first">Recognition of upstream <mark>and</mark> conventional ORF start codons is thought to involve a similar set of initiation factors (eIFs) <mark>and</mark> ribosomal subunits<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078333">26</a></sup>. Therefore, we posited <mark>that</mark> differential activity or abundance of either eIFs or ribosomal proteins might be responsible for SOX2-induced changes of the translational landscape.</p><p id="P16">To find such regulatory nodes, we designed an RNA interference (RNAi)-based screen to determine whether knockdown of any of those genes exhibits a differential growth phenotype in SOX2 versus wild-type epidermis. To this end, we pooled 750 short hairpin RNA (shRNA)-containing lentiviruses targeting translation-related genes <mark>and</mark> then introduced this library <em>in utero</em> into the amniotic sacs of living embryonic day (E)9.5 embryos (<a href="/pmc/articles/PMC5287289/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4a</span></a>, <a href="/pmc/articles/PMC5287289/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 7a</span></a>). This technique selectively <mark>and</mark> efficiently transduces skin epidermal progenitors<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078349">14</a></sup>. To prevent multi-infections <mark>and</mark> still ensure a coverage of more than 400 (cells/shRNA), we used a 10% infection rate on 27–49 embryos of each genotype <mark>and</mark> assessed shRNA representation in P0 epidermis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a class="icnblk_img figpopup" href="/pmc/articles/PMC5287289/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><img src="/pmc/articles/PMC5287289/bin/nihms845657f4.gif" class="small-thumb" alt="Figure 4" title="Figure 4" src-large="/pmc/articles/PMC5287289/bin/nihms845657f4.jpg"></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC5287289/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a4--><div><span><!--caption a8--><strong>An epidermal-specific <em>in vivo</em> RNAi screen identifies the eIF2 ternary complex as a regulatory node in premalignancy</strong></span></div></div></div><p id="P17">As expected, most cells containing ribosomal protein <mark>and</mark> <em>Eif</em> shRNAs were depleted in skins of both genotypes, <mark>with</mark> similar overall distribution of enriched <mark>and</mark> depleted shRNAs (<a href="/pmc/articles/PMC5287289/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4b, c</span></a>). However, by analysing differential shRNA representation, we began to unveil regulatory nodes downstream of SOX2 (<a href="/pmc/articles/PMC5287289/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4d</span></a>). As judged by the RIGER algorithm<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078372">27</a></sup>, <em>Eif2s1</em>, <em>Eif2s2</em> <mark>and</mark> <em>Eif5</em> were among the top six genes in which the shRNAs were specifically enriched in SOX2<sup>+</sup> epidermis (<a href="/pmc/articles/PMC5287289/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4e, f</span></a>, <a href="/pmc/articles/PMC5287289/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 7b, c</span></a>).</p><p id="P18" class="p p-last">eIF2α (encoded by <em>Eif2s1</em>) <mark>and</mark> eIF2β (<em>Eif2s2</em>) are part of the eIF2 ternary complex, which initiates canonical translation together <mark>with</mark> methionine initiator-tRNA; eIF5 is required for recycling this complex to launch subsequent rounds of translation<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078404">5</a>,<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078398">28</a></sup> (<a href="/pmc/articles/PMC5287289/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4g</span></a>). On the basis of our screen, we posited <mark>that</mark> pre-malignant SOX2-expressing epidermal cells may be using a translational program independent of canonical eIF2-mediated translation initiation. Interestingly, both SOX2<sup>+</sup> <mark>and</mark> <em>Hras<sup>G12V</sup>;Tgfbr2-</em>null SCCs displayed higher eIF2α-Ser51 phosphorylation (<a href="/pmc/articles/PMC5287289/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4h</span></a>), a modification also seen in the integrated stress response (ISR), where it diminishes eIF2 activity<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078339">29</a></sup>.</p></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">eIF2A is an essential factor for tumour initiation</h2><p id="P19" class="p p-first">To understand how SCC translation is coordinated independently of eIF2, we first asked whether alternative initiation factors coordinate translation in the context of tumour formation. Four alternative initiation factors MCT1–DENR, eIF2D (also known as <em>Lgtn</em> or ligatin), eIF5B <mark>and</mark> eIF2A are known to deliver tRNAs to P sites of the ribosome<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078357">30</a>–<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078384">34</a></sup>. <em>Eif5b</em> was not differentially depleted in our screen (<a href="/pmc/articles/PMC5287289/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 7b</span></a>), but <em>Mct1-Denr</em>, <em>Eif2d</em> <mark>and</mark> <em>Eif2a</em> shRNAs had not been included in our initial library. We focused on eIF2A as it is a crucial factor for uORF translation<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078370">35</a></sup>, it has been implicated in leucine-tRNA recruitment<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078395">34</a></sup> <mark>and</mark> may be directly involved in initiating CUG (leucine) uORF start codons<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078399">34</a>,<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078383">35</a></sup>, which showed the highest prevalence in SOX2<sup>+</sup> epidermis<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078373">18</a></sup> (<a href="/pmc/articles/PMC5287289/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3e</span></a>).</p><p id="P20">To test whether SOX2<sup>+</sup> epidermis has enhanced dependency on eIF2A, we first performed an <em>in vivo</em> mini-screen <mark>with</mark> now available <em>Eif2a</em> shRNAs. Relative to 35 control shRNAs, <em>Eif2a</em> shRNAs were under-represented in SOX2<sup>+</sup> epidermis, correlating to an extent <mark>with</mark> mRNA-knockdown efficiency (<a href="/pmc/articles/PMC5287289/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 7d</span></a>). Similar results were obtained by using <em>in utero</em> CRISPR/Cas9-mediated gene targeting to ablate <em>Eif2a</em> fully (<a href="/pmc/articles/PMC5287289/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4i</span></a>). Thus, whereas translation in wild-type epidermis depended on eIF2α, SOX2<sup>+</sup> epidermis showed increased reliance on eIF2A.</p><p id="P21" class="p p-last">To test whether eIF2A is required for translation in SCCs, we used CRISPR/Cas9 to establish three independent <em>Eif2a-</em>null clones of <em>Hras<sup>G12V</sup>;Tgfbr2-</em>null SCCs<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078331">25</a></sup> (<a href="/pmc/articles/PMC5287289/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5a</span></a>, <a href="/pmc/articles/PMC5287289/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 7e</span></a>). In serum-rich media, <em>Eif2a-</em>ablation did not affect proliferation, total protein synthesis or the translational landscape of SCC cells (<a href="/pmc/articles/PMC5287289/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 7f–h</span></a>). However, whereas control SCCs formed tumours within 1–2 weeks of engraftment, <em>Eif2a-</em>null SCCs rarely developed tumours even after 2 months (<a href="/pmc/articles/PMC5287289/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5b</span></a>). Moreover, in limiting dilution assays, tumour-initiating SCC cells were roughly 100 times fewer in isogenic <em>Eif2a-</em>null versus control SCC lines (<a href="/pmc/articles/PMC5287289/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5c</span></a>). Importantly, after reintroducing eIF2A into our <em>Eif2a-</em>null lines, tumour formation was rescued (<a href="/pmc/articles/PMC5287289/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5d</span></a>), thereby establishing an essential <mark>and</mark> hitherto unappreciated role for eIF2A-mediated translation in tumour initiation.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a class="icnblk_img figpopup" href="/pmc/articles/PMC5287289/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><img src="/pmc/articles/PMC5287289/bin/nihms845657f5.gif" class="small-thumb" alt="Figure 5" title="Figure 5" src-large="/pmc/articles/PMC5287289/bin/nihms845657f5.jpg"></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC5287289/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a4--><div><span><!--caption a8--><strong>eIF2A controls tumour formation</strong></span></div></div></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">eIF2A, protein synthesis <mark>and</mark> SCC prognosis</h2><p id="P22" class="p p-first">In performing ribosome profiling, we found a pronounced decline in the ratio of uORF to ORF translation in <em>Eif2a-</em>null compared <mark>with</mark> control SCC keratinocytes (<a href="/pmc/articles/PMC5287289/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6a</span></a>). Having established a direct role for eIF2A in mediating genome-wide translation from 5′ UTRs, we next monitored the consequences of eIF2A-dependent uORF usage on protein production. Since eIF2A loss had no appreciable effects on translation in rich media (<a href="/pmc/articles/PMC5287289/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 7h</span></a>), we added arsenite to phosphorylate <mark>and</mark> inhibit eIF2α, thus increasing eIF2A dependency. To track newly synthesized proteins in control <mark>and</mark> <em>Eif2a</em>-null SCCs, we used a pulsed SILAC (stable isotope labelling <mark>with</mark> amino acids in cell culture) approach, administering the eIF2α block concomitant <mark>with</mark> the switch from light (L)- to heavy (H)-labelled SILAC medium.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F6" co-legend-rid="lgnd_F6"><a class="icnblk_img figpopup" href="/pmc/articles/PMC5287289/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><img src="/pmc/articles/PMC5287289/bin/nihms845657f6.gif" class="small-thumb" alt="Figure 6" title="Figure 6" src-large="/pmc/articles/PMC5287289/bin/nihms845657f6.jpg"></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC5287289/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6</a></div><!--caption a4--><div><span><!--caption a8--><strong>eIF2A promotes translation of select cancer genes <mark>and</mark> leads to poor prognosis in human SCC</strong></span></div></div></div><p id="P23">Of 2,045 proteins detected by mass spectrometry, 368 showed an eIF2A-dependent, temporal increase in H-labelled peptides in SCCs (<a href="/pmc/articles/PMC5287289/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6b</span></a>, <a href="/pmc/articles/PMC5287289/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 7i</span></a>). This cohort included cancer- associated proteins such as NRAS, CD44, KRT17, Ki67, NRP1 <mark>and</mark> RAC1. Moreover, eIF2A-targeted uORF genes showed a significant eIF2A-dependent overall increase in H-labelled proteins when SCCs were cultured under stress (<a href="/pmc/articles/PMC5287289/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6c</span></a>). Thus, under conditions of eIF2α phosphorylation, eIF2A specifically augments protein synthesis of uORF-containing mRNAs.</p><p id="P24">To address whether this principle pertains to early stages of tumorigenesis <em>in vivo</em>, we performed transplantation assays (<a href="/pmc/articles/PMC5287289/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6d</span></a>, <a href="/pmc/articles/PMC5287289/figure/F14/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F14" rid-ob="ob-F14" co-legend-rid="lgnd_F14"><span>Extended Data Fig. 8a–c</span></a>). As early as day 5 after transplantation, SCC mRNAs containing eIF2A-targeted uORFs exhibited a marked increase in both translation <mark>and</mark> translational efficiency in the absence of appreciable transcriptional differences. These data suggest <mark>that</mark> during early tumorigenesis, cancer-associated transcripts <mark>that</mark> contain eIF2A-targeted uORFs selectively increase their translational efficiency. In agreement, mutations in the eIF2A-regulated uORF of <em>Ctnnb1</em>, a key oncogene for SCC progression, resulted in diminished tumorigenic potential (<a href="/pmc/articles/PMC5287289/figure/F14/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F14" rid-ob="ob-F14" co-legend-rid="lgnd_F14"><span>Extended Data Fig. 8d</span></a>).</p><p id="P25" class="p p-last">Analysis of The Cancer Genome Atlas (TCGA)<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078345">36</a></sup> revealed <mark>that</mark> the human <em>EIF2A</em> locus is amplified in 29% of patients <mark>with</mark> lung SCC, 15% of patients <mark>with</mark> head <mark>and</mark> neck SCC <mark>and</mark> 15% of patients <mark>with</mark> oesophageal carcinoma (<a href="/pmc/articles/PMC5287289/figure/F15/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F15" rid-ob="ob-F15" co-legend-rid="lgnd_F15"><span>Extended Data Fig. 9</span></a>). Notably, although initiation factors are typically regulated post-translationally, higher <em>EIF2A</em> mRNA levels correlated significantly <mark>with</mark> shorter overall survival <mark>and</mark> shorter disease-free survival (<a href="/pmc/articles/PMC5287289/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6f</span></a>).</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Discussion</h2><p id="P26" class="p p-first">Translational control allows a cell to orchestrate rapid changes in protein synthesis <mark>and</mark> tailor newly synthesized proteins to its specific needs, thereby ensuring <mark>that</mark> cellular resources are conserved<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078402">37</a></sup>. Our studies not only add to evidence <mark>that</mark> suggests <mark>that</mark> eIF2α phosphorylation <mark>and</mark> inactivation may be induced during tumour formation<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078351">38</a></sup>, but also further indicate <mark>that</mark> oncogenes can direct the translational machinery towards eIF2A-dependent uORF translation in tumorigenesis.</p><p id="P27">We found <mark>that</mark> whereas eIF2α phosphorylation globally reduces protein synthesis, translation is not inhibited on all mRNAs, but rather reprogrammed to translate selected eIF2A-dependent cancer-associated mRNAs efficiently. eIF2A competes, albeit very poorly, <mark>with</mark> eIF2α for delivery of the initiator tRNA to the 40S ribosomal subunit<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078359">32</a>,<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078396">39</a>,<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078369">40</a></sup>, thus accentuating its relevance only when eIF2α is inhibited. Our findings support a model in which tumour initiation factors <mark>and</mark>/or cellular stressors<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078332">38</a>,<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078401">41</a></sup> lead to eIF2α inhibition early in tumorigenesis, freeing eIF2A of its competitive disadvantage <mark>and</mark> orchestrating its preferential translation of genes important for malignant progression.</p><p id="P28">eIF2A could mediate tumorigenesis in several ways (<a href="/pmc/articles/PMC5287289/figure/F16/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F16" rid-ob="ob-F16" co-legend-rid="lgnd_F16"><span>Extended Data Fig. 10a</span></a>). First, if a preferred 5′ upstream start codon is in-frame, eIF2A could generate an N-terminally extended protein, <mark>and</mark> as described for the <em>Pten</em> tumour suppressor<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078343">42</a></sup>, this could yield a function distinct from its parent ORF. Second, as shown by our proteomics experiments, uORF translation can generate small peptides, which if bioactive, could directly impact cellular behaviour. Third, uORF translation could have a regulatory role by amplifying or diminishing translation of downstream ORFs. This becomes particularly important in stem cells <mark>and</mark> the early stages of tumorigenesis, where overall protein synthesis is suppressed<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078348">20</a>,<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078360">21</a></sup> <mark>and</mark> yet marked changes in protein production must occur. Indeed, we identified a subset of oncogenic mRNAs <mark>that</mark> contain eIF2A-targeted uORFs <mark>and</mark> which are preferentially translated at early stages of tumorigenesis.</p><p id="P29" class="p p-last">In summary, we discovered <mark>that</mark> during tumour initiation, oncogene- induced changes in eukaryotic initiation factors profoundly impact the translational landscape. By introducing shifts in the translation of uORFs, they can render specific cohorts of cancer-related mRNAs refractory to global reductions in protein synthesis <mark>that</mark> often accompany a stem-like <mark>and</mark>/or cancerous state. Given the poor prognosis associated <mark>with</mark> <em>EIF2A</em> mRNA levels in human cancers, our findings form a foundation for future investigations into whether eIF2A-mediated translation <mark>and</mark>/or translational regulation by uORFs can be exploited for therapeutic interventions.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">METHODS</h2><div id="S8" class="sec sec-first"><h3>Ribosome profiling</h3><p id="P30" class="p p-first"><em>In vivo</em> sample preparation: mouse skins of P4 mice were collected immediately after euthanization <mark>and</mark> placed in ice-cold PBS supplemented <mark>with</mark> 8 mg ml<sup>−1</sup> cycloheximide (CHX, in dimethyl sulfoxide) for 5 min. This concentration of CHX has been shown to reduce potential CHX-related artefacts<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078367">44</a></sup> <mark>and</mark> ensures rapid inhibition of translation within the skin. Skins were then placed into 2 ml of dispase (Corning 354235) supplemented <mark>with</mark> 8 mg ml<sup>−1</sup> CHX for 20 min at 37 °C. Epidermis was separated from dermis under a dissection scope using fine forceps. Epidermis was then placed immediately in 4 ml trypsin supplemented <mark>with</mark> 4 mg ml<sup>−1</sup> CHX <mark>and</mark> incubated for 12 min at 37 °C. The resulting cell suspension, enriched for basal epidermal keratinocytes, was then filtered through a 40-μm cell strainer, spun down <mark>and</mark> resuspended in lysis buffer as reported previously<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078335">45</a></sup>. Cells were lysed on ice for 10 min, centrifuged at 16,000<em>g</em> at 4 °C for 10 min <mark>and</mark> supernatant was flash-frozen in liquid nitrogen. RNA concentration was determined using the Quant-it Ribogreen assay kit (<a id="__tag_587078418" class="tag_hotlink" href="/nuccore/R11490" ref="/nuccore/R11490"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"R11490","term_id":"764225","term_text":"R11490"}}</span>R11490</a>, Thermo Fisher). In most experiments, three P4 mice were pooled per sample. A sample was taken for RNA-seq analysis for comparison to ribosome profiling <mark>and</mark> calculation of translational efficiency.</p><p id="P31"><em>In vitro</em> sample preparation: keratinocytes were treated <mark>with</mark> Harringtonine (Abcam, 141941) for 5 min (2 μg ml<sup>−1</sup>) to block initiation-specific translation, CHX (Sigma) for 1 min (100 μg ml<sup>−1</sup>) to block translational elongation or no drug, as indicated. Cell lysis was performed in the dish. A sample was taken for RNA-seq analysis.</p><p id="P32"><em>In vivo</em> tumour sample preparation: tumours were collected immediately after euthanization <mark>and</mark> placed in ice-cold PBS supplemented <mark>with</mark> 8 mg ml<sup>−1</sup> CHX (in dimethyl sulfoxide) for 5 min. Tumours were broken up into small pieces using a scalpel <mark>and</mark> incubated for 20 min in 0.5% collagenase supplemented <mark>with</mark> 8mg ml<sup>−1</sup> CHX (in dimethyl sulfoxide). Samples were spun down <mark>and</mark> resuspended 4 ml trypsin supplemented <mark>with</mark> 4 mg ml<sup>−1</sup> CHX for 12 min at 37 °C. The resulting cell suspension was then filtered through a 40-μm cell strainer, spun down <mark>and</mark> resuspended in lysis buffer.</p><p id="P33">The ribosomal profiling technique was carried out as reported previously<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078397">45</a></sup>, <mark>with</mark> a few modifications as described below. Lysates were treated <mark>with</mark> 2.5 U RNase I (Ambion) per microgram RNA. Ribosome-protected fragments were isolated using sephacryl S400 columns (GE Healthcare) in TE buffer. rRNAs were removed using the Ribo-zero magnetic kit (Illumina, MRZH11124). Finally, ribosome-protected fragments were amplified in 7–9 PCR cycles using Phusion polymerase (NEB). Resulting amplicons were run on an 8% acrylamide non-denaturing gel, excised <mark>and</mark> incubated in 300 mM NaCl, 10 mM Tris (pH 8) <mark>and</mark> 1 mM EDTA-containing buffer overnight at 20 °C. The ribosome-protected fragments library was then precipitated in 2 μl of glycogen <mark>and</mark> 700 μl isopropanol. Samples were analysed on a bioanalyzer before sequencing. Libraries were sequenced on HiSeq2500 <mark>and</mark> HiSeq4000 platforms.</p><p id="P34" class="p p-last">For parallel RNA-seq, RNA was purified using Direct-zol RNA MiniPrep kit (Zymo Research) per manufacturer’s instructions. Quality of the RNA was determined using Agilent 2100 Bioanalyzer, <mark>with</mark> all samples passing the quality threshold of RNA integrity numbers (RIN) &gt; 8. Library preparation using Illumina TrueSeq mRNA sample preparation kit was performed at the Weill Cornell Medical College Genomic Core facility, <mark>and</mark> cDNA was sequenced on Illumina HiSeq 2500. Reads were mapped to mm10 build of the mouse genome using TopHat2, <mark>and</mark> differential expression was determined using DESeq2.</p></div><div id="S9" class="sec"><h3>Sequencing alignment <mark>and</mark> mapping</h3><p id="P35" class="p p-first-last">Sequencing reads were clipped <mark>and</mark> trimmed using fastx_clipper <mark>and</mark> fastx_trimmer from the Hannon laboratory. Fragments derived from rRNA were removed using Bowtie2 (ref. <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078356">46</a>), aligning to the 45S preRNA sequence. The remaining non-ribosomal reads were then aligned to the mm10 mouse genome using TopHat2 (ref. <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078391">47</a>).</p></div><div id="S10" class="sec"><h3>Gene expression quantitation</h3><p id="P36" class="p p-first">For all analyses we used GRCm38.78 of the mouse genome transcript annotation from Ensembl. Sets of genes whose transcripts shared one or more exact exons were collapsed to ‘merged’ genes. Genomic coordinates occupied by more than one merged gene on the same strand were excluded from analysis. Within each merged gene, the remaining nucleotide positions were then divided into the following classes: ‘exon’ was the union of all positions across all transcripts belonging to the merged gene; for coding genes, ‘5′ UTR’ contained all positions <mark>that</mark> were uniquely labelled as 5′ UTR in all transcripts; ‘CDS’ contained all positions <mark>that</mark> were uniquely labelled as CDS in all transcripts; <mark>and</mark> ‘3′ UTR’ contained positions labelled as 3′ UTR in all transcripts.</p><p id="P37">The total number of mRNA fragments <mark>and</mark> ribosome footprints aligning to each class of positions was tabulated for each merged gene. Gene merging, position classification, <mark>and</mark> expression counting were performed using the cs script from the Plastid toolkit<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078386">48</a></sup>.</p><p id="P38">These count data were then taken into DESeq2, an R package designed for the analysis of Illumina sequencing-based assays, which estimates <mark>and</mark> accounts for biological variability in a statistical test based on the negative binomial distribution<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078365">43</a></sup>.</p><p id="P39">Ribosome occupancy was quantified for mRNAs <mark>with</mark> more than an average of 128 reads over all replicates. 5′ UTR translation was quantified for mRNAs <mark>with</mark> an average of more than 16 reads over all replicates. Throughout our study, we use 5′ UTR translation as a proxy for uORF translation. Relative 5′ UTR translation was calculated for each gene using the formula: relative 5′ UTR translation =counts in the 5′ UTR/counts in the CDS.</p><p id="P40">Translation efficiency (TE) was computed for each merged gene using the formula: 
</p><div class="disp-formula" id="FD1"><div class="f">TE =&nbsp;RPKM of CDS in ribosome profiling/RPKM in exon of RNA-seq.</div></div><p></p><p id="P41">Only RNAs <mark>with</mark> more than 256 reads were included for translation efficiency calculations.</p><p id="P42">For metagene analysis, annotated 5′ UTR from the list of quantified uORFs, translated mRNAs were selected from the ENSEMBL database. 5′ UTRs were then scaled to equal number of windows <mark>and</mark> average signal was plotted. For start codon usage (<a href="/pmc/articles/PMC5287289/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3d</span></a>), the top 5% of SOX2-regulated uORFs from P4 epidermis were quantified.</p><p id="P43">eIF2A-regulated uORFs were defined as ratio of 5′ UTR translation in SCC control/5′ UTR translation in SCC <em>Eif2a</em> knockout &gt;4 (<em>n</em> = 716). The remaining mRNAs <mark>with</mark> a ratio &lt;4 were defined as non-eIF2A regulated (<em>n</em> =746).</p><p id="P44" class="p p-last">All calculations were made in R, some graphs were plotted using ggplot2 in R. Gene lists were imported into the Ingenuity Pathway Analysis software (Ingenuity Systems), <mark>and</mark> analyses <mark>and</mark> graphic outputs of relative enrichment in functional gene categories were performed as recommended.</p></div><div id="S11" class="sec"><h3>Western blotting</h3><p id="P45" class="p p-first-last">Total cell lysates were prepared using RIPA (20 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1% Triton X-100, 0.5% deoxycorate, 0.1% SDS) supplemented <mark>with</mark> protease inhibitors (Complete mini, Roche) <mark>and</mark> Halt phosphatase inhibitor cocktail (Thermo Fisher, 1862495). The protein concentrations of clarified supernatants were measured by using BCA Protein Assay Kit (Thermo Scientific). Gel electrophoresis was performed using 4–12% NuPAGE Bis-Tris gradient gels (Life Technologies), transferred to nitro-cellulose membranes (GE Healthcare, 0.45 μm). Membranes were blocked for 90 min in 2% BSA in TBS <mark>with</mark> 0.1% Tween 20 (TBST). Membranes were then incubated <mark>with</mark> primary antibodies in the blocking buffer overnight at 4 °C. After washing <mark>with</mark> TBST, membranes were incubated <mark>with</mark> secondary antibodies in the blocking buffer at room temperature for 45 min. The membranes were washed in TBST, then incubated for 1 min <mark>with</mark> ECL western blotting detection reagent (GE Healthcare, RPN2209). Chemiluminescent protein bands were analysed using CL-X posure film (Thermo Scientific, 34091). The following primary antibodies were used: eIF2A (Proteintech, 11233-1-AP, 1:1,000), 4-E-BP1 (53H11, Cell Signaling, 1:1,000), p-4E-BP1 (236B4, Cell Signaling, 1:1,000), p-eIF2α (119A11, Cell Signaling, 1:1,000), eIF2α (D7D3, Cell Signaling, 1:1,000), β-actin (8H10D10, 1:1,000), GAPDH (14C10, Cell Signaling, 1:1,000), SOX2 (Abcam, 92494, 1:1,000).</p></div><div id="S12" class="sec"><h3>Immunofluorescence immunohistochemistry <mark>and</mark> imaging</h3><p id="P46" class="p p-first-last">For immunofluorescence microscopy, skin was embedded in optimal cutting temperature compound (OCT) (VWR). Cryosections were cut at a thickness of 12 μm on a Leica cryostat <mark>and</mark> mounted on SuperFrost Plus slides (VWR). Sections were incubated in 2% paraformaldehyde for 10 min, washed <mark>and</mark> blocked for 1 h in blocking buffer (5% normal donkey serum, 1% BSA, 2% fish gelatin, 0.3% Triton X-100 in PBS). Slides were incubated at 4 °C overnight in a primary antibodies diluted in blocking buffer. The following primary antibodies were used: SOX2 (Abcam, 92494, 1:100), integrin β4 (ITGB4; rat, 1:100, BD Pharmingen), K5 (guinea-pig, 1:500, Fuchs laboratory), After washing <mark>with</mark> PBS, sections were treated for 1 h at room temperature <mark>with</mark> secondary antibodies conjugated <mark>with</mark> Alexa 488, Alexa 594, or Alexa 647 (Life Technologies). Slides were washed, counterstained <mark>with</mark> 4′6′-diamidino- 2-phenilindole (DAPI), <mark>and</mark> mounted in Prolong Gold (Life Technologies). Images were acquired <mark>with</mark> an Axio Observer Z1 epifluorescence microscope equipped <mark>with</mark> a Hamamatsu ORCA-ER camera (Hamamatsu Photonics), <mark>and</mark> <mark>with</mark> an ApoTome.2 (Carl Zeiss) slider <mark>that</mark> reduces the light scatter in the fluorescent samples, using a 20× objective, controlled by Zen software (Carl Zeiss). RGB images were assembled using Imaris. Panels were labelled in Adobe Illustrator CS5. For immunohistochemistry, samples were processed, embedded in paraffin, <mark>and</mark> sectioned at 4 μm. Immunohistochemistry was performed on a Bond Rx auto-stainer (Leica Biosystems) <mark>with</mark> heat-mediated antigen retrieval using standard protocols. Antibodies used were rabbit monoclonal primary antibodies for phospho-4E-BP-1. Bond Polymer Refine Detection (Leica Biosystems) was used according to manufacturer’s protocol. Sections were then counterstained <mark>with</mark> haematoxycilin, dehydrated <mark>and</mark> film coverslipped using a TissueTek-Prisma <mark>and</mark> Coverslipper (Sakura). Whole slide scanning (40×) was performed on an Aperio AT2 (Leica Biosystems).</p></div><div id="S13" class="sec"><h3>Statistics</h3><p id="P47" class="p p-first-last">Data were analysed <mark>and</mark> statistics performed in Prism6 (GraphPad) <mark>and</mark> R. Significant differences between two groups were noted by asterisks or actual <em>P</em> values (*<em>P</em> &lt;0.05; **<em>P</em> &lt;0.01; ***<em>P</em> &lt; 0.001). Replicates (<em>n</em>) in this study refer to biological replicates.</p></div><div id="S14" class="sec"><h3>Analysis of human HNSCC patient data</h3><p id="P48" class="p p-first-last">We analysed the publicly available data-sets of the The Cancer Genome Atlas (TCGA; <a href="https://cancergenome.nih.gov/" ref="reftype=extlink&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">https://cancergenome.nih.gov/</a>). The cBioPortal for Cancer Genomics developed <mark>and</mark> maintained by the Computational Biology Center at Memorial Sloan-Kettering Cancer Center was used to mine the publicly available TCGA dataset on HNSCC<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078376">12</a></sup>.</p></div><div id="S15" class="sec"><h3>Lentivirus production <mark>and</mark> transduction</h3><p id="P49" class="p p-first">Production of vesicular stomatitis virus (VSV-G) pseudotyped lentivirus was performed by calcium phosphate transfection of 293FT cells (Invitrogen) <mark>with</mark> pLKO.1 <mark>and</mark> helper plasmids pMD2.G <mark>and</mark> psPAX2 (Addgene plasmid 12259 <mark>and</mark> 12260). Viral supernatant was collected 46 h after transfection <mark>and</mark> filtered through a 0.45-μm filter. For lentiviral infections in culture, cells were plated in 6-well plate at 1.0 ×10<sup>5</sup> cells per well <mark>and</mark> incubated <mark>with</mark> viruses in the presence of polybrene (20 mg ml<sup>−1</sup>) for 30 min, <mark>and</mark> then plates were spun at 1,100<em>g</em> for 30 min at 37 °C in a Thermo IEC CL40R centrifuge. Infected cells were selected <mark>with</mark> puromycin.</p><p id="P50" class="p p-last">For <em>in vivo</em> lentiviral transduction, the same viral supernatant as above was filtered (0.45-μm filter) <mark>and</mark> concentrated by ultracentrifugation. Final viral particle was resuspended in viral resuspension buffer (20 mM Tris pH 8.0, 250 mM NaCl, 10 mM MgCl<sub>2</sub>, 5% sorbitol) <mark>and</mark> 0.5 μl was <em>in utero</em> injected into E9.5 embryos, as described before<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078393">49</a></sup>. For knockdown experiments, we used clones from the Broad Institute’s Mission TRC mouse library. For shRNA screen, virus was diluted in order to obtain a 10% infection rate.</p></div><div id="S16" class="sec"><h3>Sample preparation <mark>and</mark> pre-amplification of shRNA screen</h3><p id="P51" class="p p-first">shRNA library was injected in E9.5 embryos <mark>and</mark> skins were collected as P0 postnatal animals. To distinguish between SOX2 <mark>and</mark> wild type, P0 pups were assessed for GFP expression (SOX2<sup>+</sup>). After euthanization, back <mark>and</mark> head skin was collected <mark>and</mark> epidermal cells were isolated from P0 mouse skin using previously established procedures<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078380">14</a></sup>. For <em>t</em> = 0, keratinocytes were infected <em>in vitro</em> in three independent experiments <mark>and</mark> gDNA was isolated 24 h later.</p><p id="P52" class="p p-last">Cells from individual embryos were used for genomic DNA isolation <mark>with</mark> the DNeasy Blood &amp; Tissue Kit (Qiagen). gDNAs from transduced embryos of independent experiments were pooled (3 independent experiments in wild type <mark>and</mark> SOX2, total coverage of &gt;400×). Typically two litters were pooled <mark>and</mark> processed separately as independent experiments (<em>n</em> = 3). Total DNA was used as template in pre-amplification reaction <mark>with</mark> 25 cycles <mark>and</mark> Phusion High-Fidelity DNA Polymerase (NEB). Per embryo, 4 μg of DNA was amplified. PCR products were then run on an 8% TBE gel <mark>and</mark> a clean ~240-bp band was isolated using DNA resuspension buffer (300 mM NaCl, 10 mM Tris (pH 8) <mark>and</mark> 1 mM EDTA) <mark>and</mark> incubated overnight at 20 °C. Samples were tested using the Agilent bioanalyzer. Final samples were then sent for Illumina HiSeq 2500 sequencing.</p></div><div id="S17" class="sec"><h3>Statistical analysis of relative shRNA abundance</h3><p id="P53" class="p p-first-last">For each genotype, pooled DNAs from three independent experiments were sequenced independently. Illumina reads were trimmed to the 21-nucleotide hairpin sequence using the fastx toolkit <mark>and</mark> aligned to the TRC 2 library <mark>with</mark> Bowtie using a maximum edit distance of 3. Only shRNAs <mark>that</mark> showed at <em>t</em> = 0 more than 150 reads over three independent experiments were included for further analysis, resulting in a total of 601 shRNAs. shRNA abundance was then normalized <mark>and</mark> representation of shRNAs was compared against a control library <mark>with</mark> 35 non-targeting shRNAs (ratio of shRNA to average behaviour of control library). Representation of shRNAs were then analysed using the RIGER algorithm<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078363">27</a></sup>, which ranks shRNAs according to their differential effects between two classes of samples, then identifies the genes targeted by the shRNAs at the top of the list. Please also see <a href="/pmc/articles/PMC5287289/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 7</span></a> for additional methods <mark>and</mark> metrics.</p></div><div id="S18" class="sec"><h3>Mouse strains</h3><p id="P54" class="p p-first"><em>Rosa26-CAG-loxP-stop-loxP-Sox2-IRES-eGFP</em> (<em>R26-LSL-Sox2-IRES-eGFP</em><sup>+/+</sup>) mice were donated by J. Que <mark>and</mark> were maintained in a B6/svev129 mixed background. <em>K14-cre</em>(tg) mice were maintained in a CD-1-ICR background. Nude mice (Nu/Nu) for tumour cell grafts were from Charles River. <em>Rosa26-CAG-loxP-stop-loxP-Cas9</em> mice were obtained from The Jackson Laboratory<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078338">50</a></sup>. For ribosome profiling studies, females <mark>and</mark> males were used; for tumour allografts, only females were used.</p><p id="P55">Mice were housed <mark>and</mark> cared for in an AAALAC-accredited facility, <mark>and</mark> all animal experiments were conducted in accordance <mark>with</mark> IACUC-approved protocols.</p><p id="P56" class="p p-last">Sample sizes to ensure adequate power were chosen using Gpower software<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078387">51</a></sup>. No animals were excluded from analysis. Randomization <mark>and</mark> blinding were not used in this study.</p></div><div id="S19" class="sec"><h3>Tumour formation</h3><p id="P57" class="p p-first-last">For allograft transplantation, 1.0 × 10<sup>5</sup> mouse primary tumour cells were subcutaneously injected <mark>with</mark> growth-factor reduced Matrigel (Corning, 356231) in nude female mice (6–8 weeks old). Tumour size was measured every 5 days <mark>and</mark> calculated using the formula <em>V</em> =π/6 ×length ×width<sup>2</sup>. For limit-dilution transplantation, 1.0 ×10<sup>2</sup>–1.0 ×10<sup>4</sup> mouse SCC cells were subcutaneously injected <mark>with</mark> growth-factor reduced Matrigel (Corning, 356231) in nude mice <mark>and</mark> tumour formation was assessed 4 weeks after injection. Estimated percentage of tumour-initiating cells were analysed by ELDA extreme limiting dilution analysis<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078364">52</a></sup>. The maximal tumour size allowed as per IACUC committee was 20 mm, or smaller if tumour necrosis or bleeding occurred.</p></div><div id="S20" class="sec"><h3><em>In vitro</em> cell culture experiments</h3><p id="P58" class="p p-first-last">Newborn, primary mouse epidermal keratinocytes from <em>R26-Sox2-IRES-eGFP<sup>fl</sup></em><sup>/+</sup> <mark>and</mark> <em>K14-cre; R26-Sox2-IRES-eGFP<sup>fl</sup></em><sup>/+</sup> were cultured on 3T3-S2 feeder layer in 0.05 mM Ca<sup>2+</sup> E-media supplemented <mark>with</mark> 15% serum<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078352">53</a></sup>. <em>Hras<sup>G12V</sup>; Tgfbr2</em> knockout cell line was generated previously in the Fuchs laboratory<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078346">25</a></sup>. Cell lines were cultured in E medium <mark>with</mark> 15% FBS <mark>and</mark> 50 mM CaCl<sub>2</sub>. Cell lines were not tested for mycoplasma infection.</p></div><div id="S21" class="sec"><h3>CRISPR in <em>Hras<sup>G12V</sup>; Tgfbr2</em> knockout SCC cells</h3><p id="P59" class="p p-first-last">Three sgRNAs against <em>Eif2a</em> were selected from the GeCKO library <mark>and</mark> cloned into a lenti-CRISPR v2 vector (v2 vector was a gift from F. Zhang, Addgene plasmid 52961)<sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078354">54</a></sup>. Non-targeting control sgRNA was used side-by-side <mark>with</mark> sgRNAs against target sites throughout the experiments to rule out phenotypic changes due to nonspecific editing. <em>Hras<sup>G12V</sup>; Tgfbr2</em> knockout mice were infected <mark>and</mark> selected <mark>with</mark> puromycin (3 μg ml<sup>−1</sup>) to obtain stable <em>Eif2a</em> knockout pools. To obtain clonal cell lines, single-cell sorting into 96-well plates was performed. Genomic DNAs from single clones were isolated, from which the targeted <em>Eif2a</em> locus was PCR amplified <mark>and</mark> Sanger sequenced to confirm editing. Three frameshift mutations were selected (targeted by sg1 <mark>and</mark> sg3). Knockout was confirmed by western blot.</p></div><div id="S22" class="sec"><h3>Sequences of the guides</h3><p id="P60" class="p p-first-last"><em>Eif2a</em> sg1: 5′-TATAATCAATGTCGCTAACA-3′; <em>Eif2a</em> sg2: 5′-TGTAAGGCTGCCACGTTGCC-3′; <em>Eif2a</em> sg3: 5′-ACCGTGCTTTC TGTGAAGTG-3′; non-targeting: 5′-GCGAGGTATTCGGCTCCGCG-3′; <em>Ctnnb1</em> uORF: 5′-GGCCTCCTGCACTGACGGCT-3′.</p></div><div id="S23" class="sec"><h3>Measurement of protein synthesis</h3><p id="P61" class="p p-first-last">For protein synthesis analysis, 10<sup>4</sup>–10<sup>5</sup> cells were plated in 12-well plates. OPP (Jena Biosciences) was added for 1 h at a concentration of 50 μM. Cells were then removed from wells <mark>and</mark> washed <mark>with</mark> PBS. The cell suspension was fixed in 0.5 ml of 1% paraformaldehyde in PBS for 15 min on ice <mark>and</mark> permeabilized in PBS containing 0.1% saponin (Sigma) for 5 min at room temperature. At this point the azide–alkyne cycloaddition reaction (using Alexa Fluor 594 azide, Life Technologies) was allowed to proceed for 30 min in the dark at room temperature using the Click-iT Cell Reaction Buffer Kit (Life Technologies). Cells were then washed three times in PBS supplemented <mark>with</mark> 2% FBS, resuspended in PBS, <mark>and</mark> immediately analysed by flow cytometry. As additional negative control, translational elongation inhibitor 100 μg ml<sup>−1</sup> CHX (Sigma) was added 30 min before OPP. OPP fluorescence signal was compared between different genotypes after subtracting fluorescence from control (no OPP) samples.</p></div><div id="S24" class="sec"><h3>Quantification of total proteome <mark>and</mark> pulsed SILAC labelling</h3><p id="P62" class="p p-first">Protein was extracted from mouse epidermis tissue for total proteome profiling <mark>and</mark> from <em>Hras<sup>G12V</sup>; Tgfbr2</em> or <em>Hras<sup>G12V</sup>; Tgfbr2; Eif2a</em> knockout cells pulsed <mark>with</mark> heavy arginine (R10) <mark>and</mark> lysine (K6) for 0, 12, 24, 48 <mark>and</mark> 72 h for pulsed SILAC to measure newly translated proteins. Cell pellets were disrupted in 8 M urea <mark>and</mark> passed through a 20-gauge needle for 10 cycles <mark>and</mark> placed in a bath sonicator on ice for 30 min. Fifty micrograms of protein for three replicates from each condition was processed in parallel as follows: cysteines were reduced <mark>with</mark> dithiothreitol (Sigma) before alkylation <mark>with</mark> iodoacetamide (Sigma). Proteins were digested <mark>with</mark> LysC (Wako Chemicals) followed by trypsin (Promega) <mark>and</mark> desalted <mark>with</mark> Empore C18 STaGETips (3M)<sup><a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078377">55</a></sup>.</p><p id="P63">One microgram of total protein was injected for nano-LC–MS/MS analysis. For the total proteome analysis, peptides were separated using a 12 cm ×75μm C18 column (Nikkyo Technos) at a flow rate of 200 nl min<sup>−1</sup>, <mark>with</mark> a 5–40% gradient over 160 min (buffer A 0.1% formic acid, buffer B 0.1% formic acid in acetonitrile), <mark>and</mark> a Q-Exactive Plus (ThermoScientific) was operated in data-dependent mode <mark>with</mark> a top 20 method. For the pulse SILAC analysis, peptides were separated <mark>with</mark> an EasySpray 50 cm column (ThermoScientific) over a 3 h gradient, at a flow rate of 300 nl min<sup>−1</sup> <mark>and</mark> analysed by an Orbitrap Lumos operated in ‘top speed’ mode, <mark>with</mark> HCD/ion trap MS/MS scans.</p><p id="P64" class="p p-last">Mass spectrometry data were analysed using MaxQuant (version 1.5) <mark>and</mark> Perseus software (version 1.4), searching against a Uniprot <em>Mus musculus</em> database (downloaded July 2014), allowing oxidation of methionine <mark>and</mark> protein N-terminal acetylation, <mark>and</mark> filtering at a 1% false discovery rate at the peptide <mark>and</mark> protein level. Proteins were quantified using LFQ values for the total proteome analysis; proteins were deemed significantly changing by <em>t</em>-test, using a corrected FDR less than 5%. For the pulse SILAC analysis, LFQ values from the heavy channel were used for quantitation.</p></div><div id="S25" class="sec"><h3>Identification of uORF peptides</h3><p id="P65" class="p p-first">To identify proteins expressed from the untranslated region, a peptidome enrichment from epidermis tissue <mark>and</mark> protein N-terminus enrichment from P4 epidermis <mark>and</mark> SCCs was conducted before LC–MS/MS analysis <mark>and</mark> searching against a 5′ UTR database. The database was generated by including all 5′ UTRs <mark>with</mark> an average of more than 16 reads over all P4 <em>in vivo</em> epidermis samples (3 wild type <mark>and</mark> 3 SOX2).</p><p id="P66">The peptidome was enriched by passing 100 μg of undigested protein through a 30 kDa molecular mass cut-off filter (Millipore). Flow-through was further enriched by C18 reversed Oasis phase solid phase extraction columns (Waters), eluting in 50% acetonitrile, 0.1% TFA. The enriched peptidome was either analysed directly, or tryptic digested as above, to cover both short <mark>and</mark> long uORF peptides.</p><p id="P67">Protein N-terminal peptides were enriched by the TAILS method<sup><a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078375">56</a></sup>. In brief, after protein denaturation, free amines were blocked by dimethyl labelling, digested, <mark>and</mark> internal (neo-N-terminal unblocked) peptides were depleted by crosslinking to HPG-ALD polymer. Enriched N termini were analysed by an Orbitrap Lumos operated in top speed mode, <mark>with</mark> alternating CID/ion trap <mark>and</mark> HCD/Orbitrap MS/MS scans.</p><p id="P68" class="p p-last">Data were searched <mark>with</mark> Proteome Discover 1.4 using the Mascot search engine. Spectra not matched to the mouse proteome database <mark>with</mark> at least medium confidence (Percolator, confidence &gt;95%) were exported for search of the 5′ UTR database, using 6-frame translation, semitryptic constraints, <mark>and</mark> peptide N-terminal acetylation. Data from the TAILS experiment were searched <mark>with</mark> semi-ArgC constraints <mark>with</mark> lysine dimethylation as a stable modification <mark>and</mark> N-terminal dimethylation as a variable modification. High confidence (&gt;99%) peptides within 5 p.p.m. accuracy were submitted to BLAST to confirm <mark>that</mark> they did not match a predicted protein or contaminant. Manual analysis of MS/MS matches confirmed 13 uORF peptide sequences (<a href="/pmc/articles/PMC5287289/figure/F11/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 5</span></a>).</p></div><div id="S26" class="sec"><h3>Minimum free energy calculation</h3><p id="P69" class="p p-first-last">Ribosome footprint reads were counted relative to a consensus 5′ UTR sequence for each gene as annotated by plastid. These genomic coordinates were used to build a library of 5′ UTR sequences using bed-tools. These sequences were input for the RNAfold algorithm of the ViennaRNA Package<sup><a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078353">57</a></sup>. The algorithm computes base pairing probabilities of nucleotides within an RNA sequence to determine a single thermodynamically favoured structure as well as its free energy. This library of structural data was used to evaluate genes <mark>with</mark> 5′ UTRs differentially regulated by SOX2.</p></div><div id="S27" class="sec"><h3>RiboTaper analysis</h3><p id="P70" class="p p-first-last">The Ribotaper<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078358">22</a></sup> pipeline was used to predict translated regions in ribosome profiling data from SOX2 keratinocytes (sample number 16).</p></div><div id="S28" class="sec sec-last"><h3>Data availability</h3><p id="P71" class="p p-first-last">The data <mark>that</mark> support the findings of this study have been deposited in the Gene Expression Omnibus (GEO) repository <mark>with</mark> the accession code <a id="__tag_587078406" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE83332" ref="/geo/query/acc.cgi?acc=GSE83332"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE83332","term_id":"83332"}}</span>GSE83332</a>. All other data are available from the corresponding author(s) upon reasonable request.</p></div></div><div id="S29" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S29title">Extended Data</h2><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F7"><h3>Extended Data Figure 1</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f7.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f7.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f7.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Consequence of SOX2 expression in the epidermis <mark>and</mark> correlation between <em>in vivo</em> ribosome profiling experiments</strong><p id="P72"><strong>a</strong>, Papilloma formation induced by SOX2 overexpression in the skin. SOX2 expression in tamoxifen-inducible <em>K14-creER; R26-Sox2-IRES-eGFP<sup>fl/fl</sup></em> mice results in hyperplasia <mark>and</mark> papilloma formation. Representative H&amp;E sections are shown. Animals develop severe skin lesions in the ventral epidermis 6–8 weeks after tamoxifen injection <mark>and</mark> require euthanasia. <strong>b</strong>, Experimental strategy to perform <em>in vivo</em> epidermis-specific ribosome profiling. <em>In vivo</em> epidermis-specific ribosome profiling strategy results in highly reproducible quantifications. Plots show RPKM correlations between the 3 independent replicate experiments in wild-type P4 epidermis. Quantified in this study were mRNAs <mark>with</mark> &gt;128 reads. <strong>c</strong>, N-terminal extension of the translated <em>Swi5</em> mRNA in wild-type <mark>and</mark> SOX2 P4 epidermis <em>in vivo</em>. Tracks show ribosome profiling reads along the <em>Swi5</em> mRNA for replicate samples of each genotype. The final track shows harringtonine-treated ribosome profiling reads of wild-type keratinocytes <em>in vitro.</em> Harringtonine blocks ribosomes at the translational start site <mark>and</mark> allows translation start site mapping<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078362">18</a></sup>. Red arrow indicates direction of translation. Green bar marks the annotated CDS. Blue bar denotes the actual translated coding sequence based upon ribosomal profiling.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F8"><h3>Extended Data Figure 2</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f8.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f8.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f8.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Global translational efficiency is decreased upon SOX2 expression in keratinocytes <em>in vitro</em></strong><p id="P73"><strong>a</strong>, Ribosome profiling data correlates <mark>with</mark> proteomics data. At a stage when proliferation <mark>and</mark> morphology were similar, freshly isolated, basally enriched keratinocytes from wild-type <mark>and</mark> SOX2-expressing P4 skins were subjected to <em>in vivo</em> ribosome profiling <mark>and</mark> to a label-free proteomics strategy. Plotted are proteomics fold changes compared to ribosome profiling fold changes. Comparisons are made in SOX2 versus wild-type samples for significantly changed proteins (false discovery rate (FDR) &lt;0.05). <strong>b</strong>, Translational efficiency is markedly reduced in SOX2-expressing premalignant keratinocytes <em>in vitro</em>. Keratinocytes were isolated from <em>R26-Sox2-IRES-eGFP<sup>fl</sup></em><sup>/+</sup> (WT) or <em>K14-cre; R26-Sox2-IRES-eGFP<sup>fl</sup></em><sup>/+</sup> (SOX2<sup>+</sup>) P0 animals <mark>and</mark> cultured <em>in vitro.</em> Quantified were genes <mark>with</mark> more than 256 reads in RNA-seq data. Histogram shows distribution of differential translational efficiency (TE =RPKM<sub>ribosome profiling</sub>/RPKM<sub>RNA-seq</sub>). Data are shown from 4 independent ribosome profiling experiments <mark>and</mark> 2 independent RNA-seq experiments. <strong>c</strong>, Wild-type <mark>and</mark> SOX2 keratinocytes have similar proliferation rates <em>in vivo</em>. Basal epidermal EdU incorporation in P0, P2 <mark>and</mark> P4 mice (<em>n</em> = 442/496, 385/449 <mark>and</mark> 903/841 cells from duplicate wild-type/SOX2 animals) 1 h after injection. Data are mean ±s.d. <strong>d</strong>, Phospho-4E-BP1 immunohistochemistry in the epidermis shows no difference in levels upon SOX2 induction. <strong>e</strong>, NSUN2 transcript, translation, <mark>and</mark> protein levels in SOX2 versus WT P4 epidermis.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F9"><h3>Extended Data Figure 3</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f9.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f9.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f9.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Translationally controlled genes show a shift towards cancer-related pathways</strong><p id="P74">Pathway analyses reveal a shift towards cancer-related pathways in mRNAs resistant to the SOX2-mediated global decrease in translational efficiency <mark>and</mark> in mRNAs <mark>with</mark> preferential uORF translation in premalignant SOX2<sup>+</sup> P4 epidermis. Ingenuity pathway analysis (IPA) was used to analyse genes differentially regulated at four levels of control: transcriptional, translational, translational efficiency, <mark>and</mark> uORF/ORF ratio level. Included were the top 10% most upregulated <mark>and</mark> top 10% most downregulated genes at the levels of the transcriptome, translatome, <mark>and</mark> uORF usage. The translational efficiency list was restricted to the top <mark>and</mark> bottom-most 500 genes, corresponding to 6.6% of all genes. Total number of genes quantified: 4,725 transcriptome, 4,725 translatome, 7,605 translational efficiency, 1,830 uORFs.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F10"><h3>Extended Data Figure 4</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f10.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f10.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f10.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Translation from 5′ UTRs</strong><p id="P75"><strong>a</strong>, 5′ UTR translation of <em>Myc</em> in the P4 epidermis <em>in vivo</em>. As described previously, <em>Myc</em> mRNA contains several translated uORFs<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078382">18</a></sup>. Right panels show higher magnification of the uORF start codons. <strong>b</strong>, 5′ UTR translation of <em>Eif4g2</em> (encoding eIF4γ2) in the P4 epidermis <em>in vivo</em>. Harringtonine track shows main translation initiation site in the 5′ UTR. Right panels show higher magnification of the uORF start codons. <strong>c</strong>, 5′ UTR translation of <em>Btg1</em> in the P4 epidermis <em>in vivo</em>. Harringtonine track shows main translation initiation site in the 5′ UTR. Right panels show higher magnification of the uORF start codons. <strong>d</strong>, Relative uORF translation in SOX2 versus wild-type keratinocytes using Harringtonine-treated samples. For each gene, the ratio of ribosome footprints in 5′ UTR of Harringtonine samples versus CHX-treated ORFs (for normalization) was calculated. Histogram shows distribution of log<sub>2</sub> fold changes in relative 5′ UTR translation.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F11"><h3>Extended Data Figure 5</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f11.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f11.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f11.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Proteomic detection of 13 peptides produced from 5′ UTRs</strong><p id="P76"><strong>a</strong>, The RiboTaper analysis pipeline<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078350">22</a></sup> was used to annotate upstream ORFs computationally in an <em>in vitro</em> SOX2 keratinocyte sample. RiboTaper exploits the triplet periodicity of ribosomal footprints to predict bona fide translated regions. Boxplot shows the distribution of length of 215 uORFs which start <mark>with</mark> an AUG codon <mark>with</mark> a median length of 54 nucleotides. As a comparison, the right panel shows boxplot <mark>with</mark> the distribution of length of main ORFs predicted by RiboTaper. <strong>b</strong>, Schematic overview of the terminal amine isotopic labelling of substrates (TAILS) strategy to specifically enrich for N-terminal fragments. <strong>c</strong>, Overview of the pre- <mark>and</mark> post-TAILS peptide counts <mark>and</mark> peptide-spectrum matches. <strong>d</strong>, MS/MS spectra for identified uORF peptides. Representative MS/MS spectra for identified uORF peptides <mark>with</mark> monoisotopic (<em>m</em>/<em>z</em>), parent mass error (parts per million), charge state, <mark>and</mark> Mascot ion score. Matched <em>y</em>-ion fragments are shown in blue, <em>b</em>-ions in red, <mark>and</mark> unfragmented parents in green. Peptide N termini were identified as naturally N-terminally acetylated or unmodified. Owing to the protein-level primary amine blocking step in the TAILS workflow, naturally unmodified N termini <mark>and</mark> all lysines carry a dimethyl chemical modification. <strong>e</strong>, Ribosome profiling tracks showing translated uORF in P4 epidermis of the hepatoma-derived growth factor <em>Hdgf</em> <mark>and</mark> of the glycolipid transfer <em>Gltp</em> gene. Encoded peptides identified by high-resolution/high-mass-accuracy mass spectrometry using proteomics <mark>and</mark> peptidomics are shown. Red amino acids correspond to the identified peptides, yellow nucleotides mark potential initiation sites.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F12"><h3>Extended Data Figure 6</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f12.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f12.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f12.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>SOX2-targeted 5′ UTRs are highly structured</strong><p id="P77"><strong>a</strong>, Genes <mark>with</mark> increased SOX2-regulated 5′ UTR translation also have longer 5′ UTRs <mark>and</mark> are more structured. The 10% of genes <mark>with</mark> the most increased 5′ UTR translation in SOX2 cells were evaluated relative to the 10% of genes <mark>with</mark> the largest decrease in 5′ UTR translation (error bars indicate range, <em>n</em> = 183 each, two-sided Wilcoxon test). <strong>b</strong>, Cumulative distribution plot showing length to structure comparison, an assessment for each gene’s 5′ UTR structure relative to its length. Analysis showing <mark>that</mark> SOX2-upregulated 5′ UTRs tend to have more favourable free energy at each length, suggesting <mark>that</mark> SOX2-regulated 5′ UTR secondary structures are more stable even when normalized for length (error bars indicate range, <em>n</em> = 183 each, two-sided Wilcoxon test). <strong>c</strong>, mRNAs showing preferential uORF translation in premalignant SOX2-expressing epidermis significantly overlap <mark>with</mark> mRNAs <mark>that</mark> are most resistant to the reduction in translational efficiency in premalignant SOX2 versus wild-type epidermis (hypergeometric test, <em>P</em> &lt; 0.001). All mRNAs <mark>with</mark> a relative increase in the uORF/ORF ratio in SOX2 versus wild type were compared to the top 10% of mRNAs most resistant to reduction in translational efficiency in P4 SOX2 versus wild-type epidermis. Pathway analysis for the overlapping 122 genes (ingenuity pathway analysis) revealed epithelial–mesenchymal transition (EMT), stem-cell pluripotency, <mark>and</mark> axonal guidance as the top most enriched pathways. <strong>d</strong>, <em>Hras<sup>G12V</sup>; Tgfbr2</em> knockout SCC tumour growth is dependent on SOX2 signalling. As shown in <a href="/pmc/articles/PMC5287289/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4h</span></a>, the <em>Hras<sup>G12V</sup>; Tgfbr2</em> knockout is sufficient to upregulate SOX2 levels. Graph showing SCC tumour growth post-injection of 10<sup>5</sup> cells. Data are mean ±s.e.m. (<em>n</em> = 8 for each genotype). <strong>e</strong>, Overlap between uORF translation <mark>that</mark> occurs preferentially in premalignant SOX2-expressing P4 epidermis <em>in vivo</em> <mark>and</mark> uORF translation of malignant SOX2-expressing, HRAS-regulated SCC <em>in vitro.</em> Included were all mRNAs <mark>with</mark> twofold difference SOX2 versus wild-type P4 epidermis <em>in vivo</em> <mark>and</mark> twofold difference <em>Hras<sup>G12V</sup>; Tgfbr2</em> knockout SCC versus wild-type keratinocytes <em>in vitro</em>. Hypergeometric test, <em>P</em> &lt;0.001.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F13"><h3>Extended Data Figure 7</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f13.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f13.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f13.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>An shRNA screen reveals regulatory nodes in premalignancy</strong><p id="P78"><strong>a</strong>, The 138 ribosomal genes <mark>and</mark> initiation factors targeted in our shRNA screen. <strong>b</strong>, Overview of the shRNA screen in wild-type versus premalignant SOX2-expressing epidermis from P0 mice. The RIGER algorithm<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078374">27</a></sup> was used <mark>with</mark> the following methods <mark>and</mark> metrics to convert hairpins to genes <mark>and</mark> to rank top hits. From left to the right: weighted sum, signal to noise (median); second best rank, signal to noise (median); weighted sum, signal to noise; weighted sum, fold change. <strong>c</strong>, Western blot shows knockdown efficiency of shRNAs targeting the top hit in our screen, <em>Eif2s1.</em> Note <mark>that</mark> the knockdown efficiency correlates well <mark>with</mark> the degree of shRNA depletion in the screen. <strong>d</strong>, Intra-amniotic injection of lentivirus library of <em>Eif2a</em> <mark>and</mark> 35 control shRNAs. This sub-library was injected into E9.5 embryos <mark>and</mark> representation of shRNAs was quantified in wild-type <mark>and</mark> SOX2 P0 skin. Top, knockdown efficiency; bottom, relative representation of <em>Eif2a</em> shRNAs (normalized against the control shRNA library) in wild-type versus SOX2 epidermis. <strong>e</strong>, DNA sequence of <em>Eif2a</em> knockout clonal cell lines used in our study. PAM region is highlighted in red, CRISPR target region in blue. <strong>f</strong>, <strong>g</strong>, Proliferation rates <mark>and</mark> overall protein synthesis rates are unchanged in <em>Eif2a</em> knockout cells under serum-rich conditions <em>in vitro</em>. SCC control <mark>and</mark> <em>Eif2a</em> knockout cells were quantified for EdU <mark>and</mark> OPP incorporation 1 h after administration. Data are mean ± s.d. of 4 independent experiments. <strong>h</strong>, The translational landscape is unchanged in <em>Eif2a</em> knockout cells under serum-rich conditions <em>in vitro.</em> SCC control <mark>and</mark> <em>Eif2a</em> knockout cells subjected to ribosome profiling <mark>and</mark> reads within the main ORFs were quantified <mark>and</mark> tested for differential expression using DESeq2. As opposed to 5′ UTR translation, no significant differences (adjusted <em>P</em> &lt; 0.1 DESeq2) in ORF translation were found between SCC control <mark>and</mark> SCC <em>Eif2a</em> knockout cells (<em>n</em> = 2 SCC control, <em>n</em> =2 SCC <em>Eif2a</em> knockout). Right panels show two representative H&amp;E sections of squamous cell carcinomas formed 25 days after subcutaneous injection of SCC cells. <strong>i</strong>, eIF2A-dependent changes of heavy-labelled peptides of CTNNB1 <mark>and</mark> CD44 under stress in pulsed SILAC. SCC control <mark>and</mark> SCC <em>Eif2a</em> knockout cells were grown in light-labelled medium <mark>and</mark> switched to heavy-labelled medium supplemented <mark>with</mark> 5 μM arsenite. Graphs show the relative difference between SCC control <mark>and</mark> SCC <em>Eif2a</em> knockout cells during the pulsed SILAC time course.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F14"><h3>Extended Data Figure 8</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f14.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f14.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f14.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>mRNAs containing eIF2A-targeted uORFs are preferentially translated during tumorigenesis</strong><p id="P79"><strong>a</strong>–<strong>c</strong>, Genes <mark>that</mark> contain eIF2A-targeted uORFs maintain increased translation <mark>and</mark> translational efficiency of their downstream ORFs during early stages of tumorigenesis. <em>Hras<sup>G12V</sup>; Tgfbr2</em> knockout SCCs were subcutaneously injected into nude mice. Day 5 or day 10 tumours were analysed by ribosome profiling <mark>and</mark> RNA-seq. Changes in transcription, translation <mark>and</mark> translational efficiency were assessed comparing <em>in vivo</em> against SCC <em>in vitro</em> data <mark>and</mark> represented as fold changes in Kernel density plots. Either only the significantly changed genes (DESeq2) or all genes were assessed. eIF2A-targeted versus non-eIF2A-targeted uORF genes refer to changes in uORF usage in SCC control versus SCC <em>Eif2a</em> knockout in <a href="/pmc/articles/PMC5287289/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6a</span></a> (n =2 SCC <em>in vitro</em>, <em>n</em> =2 SCC day 5, <em>n</em> = 2 SCC day 10, two-sample Kolmogorov–Smirnov test). <strong>d</strong>, Mutations in the eIF2A-regulated uORF of <em>Ctnnb1</em>, a key oncogene for SCC progression, diminishes its tumorigenic potential. Tracks show ribosome profiling reads in the 5′ UTR of <em>Ctnnb1</em> <mark>and</mark> in the uORF GUG start codon. <em>Hras<sup>G12V</sup>; Tgfbr2</em> knockout SCC keratinocytes were infected <mark>with</mark> either non-targeting control sgRNA or <em>Ctnnb1</em> uORF sgRNAs. Clonal lines were sequenced <mark>and</mark> an uORF mutant clonal line was established. Data are mean ± s.e.m. following subcutaneous injection of 10<sup>5</sup> cells (<em>n</em> =8 control, <em>n</em> = 14 clone 2B).</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F15"><h3>Extended Data Figure 9</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f15.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f15.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f15.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Human <em>EIF2A</em> is frequently amplified in human cancer</strong><p id="P80"><strong>a</strong>, Summary of cross-cancer alterations for <em>EIF2A</em> in human cancers<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_587078381">12</a></sup>. 29% of patients <mark>with</mark> lung carcinoma <mark>and</mark> 15% of patients <mark>with</mark> head <mark>and</mark> neck SCCs show an amplification of the <em>EIF2A</em> locus. <strong>b</strong>, Summary of the clinical information accompanying TCGA patients <mark>with</mark> head <mark>and</mark> neck SCC.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F16"><h3>Extended Data Figure 10</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5287289_nihms845657f16.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms845657f16.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5287289/bin/nihms845657f16.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Overview of 5′ UTR translation <mark>and</mark> proteomics analyses</strong><p id="P81"><strong>a</strong>, Summary of different analyses of 5′ UTR translation in this study. <strong>b</strong>, Significantly changed proteins in SOX2 versus wild-type P4 epidermis <em>in vivo</em> (FDR &lt;0.05).</p></div></div></div><div id="S30" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S30title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemental</h4><div class="sup-box half_rhythm" id="d36e1922"><a href="/pmc/articles/PMC5287289/bin/NIHMS845657-supplement-Supplemental.pdf">Click here to view.</a><sup>(94K, pdf)</sup></div></div></div><div id="S31" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S31title">Acknowledgments</h2><div class="sec"><p id="__p2">We thank J. Que for the <em>R26-Sox2-IRES-eGFP</em> mice, D. Xu <mark>and</mark> the Cornell Genomics Facility for sequencing support, the Rockefeller Proteomics Facility for protein/peptide analyses, members of the Fuchs’ laboratory for discussions, <mark>and</mark> L. Polak <mark>and</mark> M. Sribour for their support <mark>with</mark> tumorigenesis studies. We thank E. Heller for bioinformatics support, L. Calviello for support <mark>with</mark> RiboTaper, <mark>and</mark> F. Garcia-Quiroz <mark>and</mark> M. Jovanovic for critical reading of the manuscript. The Rockefeller University Proteomics Resource Center acknowledges funding from the Leona M. <mark>and</mark> Harry B. Helmsley Charitable Trust <mark>and</mark> Sohn Conferences Foundation for mass spectrometer instrumentation. The results published here are in part based upon data generated by the TCGA Research Network (<a href="http://cancergenome.nih.gov/" ref="reftype=extlink&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://cancergenome.nih.gov/</a>). A.S. was supported by the Human Frontier Science Program Organization (HFSP, LT000639-2013) <mark>and</mark> is currently supported by the People Programme (Marie Curie Actions) of the European Union’s Seventh Framework Programme FP7 under REA grant agreement no. 629861. S.N. is a Damon Runyon Fellow (DRG-2183-14). B.H. was supported by a Medical Scientist Training Program grant from the National Institute of General Medical Sciences of the National Institutes of Health under award number T32GM007739 to the Weill Cornell/Rockefeller/Sloan-Kettering Tri-Institutional MD-PhD Program. E.F. <mark>and</mark> J.S.W. are Investigators of the Howard Hughes Medical institute. This work was supported by grants to E.F. from the National Institutes of Health (R37-AR27883) <mark>and</mark> NYSTEM CO29559.</p></div></div><div id="__fn-groupidm139773289937872" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__fn-groupidm139773289937872title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="__p3" class="p p-first-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a> is available in the online version of the paper.</p><p></p><div><p id="__p4" class="p p-first-last"><strong>Author Contributions</strong> A.S. <mark>and</mark> E.F. conceived the project, designed the experiments <mark>and</mark> wrote the manuscript. A.S. <mark>and</mark> B.H. performed the experiments, <mark>and</mark> collected <mark>and</mark> analysed data. J.G.D., E.H.R., J.S.W. <mark>and</mark> N.C.G. contributed to ribosome profiling data analysis. D.S. contributed to control shRNA library generation <mark>and</mark> established <em>Hras<sup>G12V</sup>; Tgfbr2-</em>null cell lines. S.N. contributed to OPP experiments. J.L. carried out <em>in utero</em> lentiviral injections. H.M. <mark>and</mark> B.D.D. performed proteomics experiments <mark>and</mark> analysed proteomics data. E.F. supervised the project. All authors discussed the results <mark>and</mark> edited the manuscript.</p></div><p class="fn sec" id="FN5"></p><p id="__p5" class="p p-first-last">The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper.</p><p></p><p class="fn sec" id="FN6"></p><p id="__p6" class="p p-first-last"><strong>Online Content</strong> Methods, along <mark>with</mark> any additional Extended Data display items <mark>and</mark> Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.</p><p></p></div></div><div id="__ref-listidm139773289930048" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__ref-listidm139773289930048title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Schwanhäusser B, et al.  Global quantification of mammalian gene expression control. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">473</span>:337–342.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/21593866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Zhang B, et al.  Proteogenomic characterization of human colon <mark>and</mark> rectal cancer. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">513</span>:382–387.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4249766/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/25043054" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Mamane Y, Petroulakis E, LeBacquer O, Sonenberg N. mTOR, translation initiation <mark>and</mark> cancer. <span><span class="ref-journal">Oncogene. </span>2006;<span class="ref-vol">25</span>:6416–6422.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17041626" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Pelletier J, Graff J, Ruggero D, Sonenberg N. Targeting the eIF4F translation initiation complex: a critical nexus for cancer development. <span><span class="ref-journal">Cancer Res. </span>2015;<span class="ref-vol">75</span>:250–263.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4299928/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/25593033" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Sonenberg N, Hinnebusch AG. Regulation of translation initiation in eukaryotes: mechanisms <mark>and</mark> biological targets. <span><span class="ref-journal">Cell. </span>2009;<span class="ref-vol">136</span>:731–745.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3610329/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19239892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Hsieh AC, et al.  The translational landscape of mTOR signalling steers cancer initiation <mark>and</mark> metastasis. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">485</span>:55–61.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3663483/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22367541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Huang PY, Balmain A. Modeling cutaneous squamous carcinoma development in the mouse. <span><span class="ref-journal">Cold Spring Harb Perspect Med. </span>2014;<span class="ref-vol">4</span>:a013623.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4143107/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/25183851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Okubo T, Pevny LH, Hogan BLM. Sox2 is required for development of taste bud sensory cells. <span><span class="ref-journal">Genes Dev. </span>2006;<span class="ref-vol">20</span>:2654–2659.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC1578692/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17015430" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Que J, et al.  Multiple dose-dependent roles for Sox2 in the patterning <mark>and</mark> differentiation of anterior foregut endoderm. <span><span class="ref-journal">Development. </span>2007;<span class="ref-vol">134</span>:2521–2531.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3625644/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17522155" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Boumahdi S, et al.  SOX2 controls tumour initiation <mark>and</mark> cancer stem-cell functions in squamous-cell carcinoma. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">511</span>:246–250.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/24909994" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Rudin CM, et al.  Comprehensive genomic analysis identifies SOX2 as a frequently amplified gene in small-cell lung cancer. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:1111–1116.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3557461/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22941189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Gao J, et al.  Integrative analysis of complex cancer genomics <mark>and</mark> clinical profiles using the cBioPortal. <span><span class="ref-journal">Sci Signal. </span>2013;<span class="ref-vol">6</span>:pl1.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4160307/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23550210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Beronja S, et al.  RNAi screens in mice identify physiological regulators of oncogenic growth. <span><span class="ref-journal">Nature. </span>2013;<span class="ref-vol">501</span>:185–190.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3774280/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23945586" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Beronja S, Livshits G, Williams S, Fuchs E. Rapid functional dissection of genetic networks via tissue-specific transduction <mark>and</mark> RNAi in mouse embryos. <span><span class="ref-journal">Nat Med. </span>2010;<span class="ref-vol">16</span>:821–827.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2911018/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/20526348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Liu K, et al.  Sox2 cooperates <mark>with</mark> inflammation-mediated Stat3 activation in the malignant transformation of foregut basal progenitor cells. <span><span class="ref-journal">Cell Stem Cell. </span>2013;<span class="ref-vol">12</span>:304–315.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3594795/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23472872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Ingolia NT, Ghaemmaghami S, Newman JRS, Weissman JS. Genome-wide analysis <em>in vivo</em> of translation <mark>with</mark> nucleotide resolution using ribosome profiling. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">324</span>:218–223.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2746483/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19213877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Brar GA, et al.  High-resolution view of the yeast meiotic program revealed by ribosome profiling. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">335</span>:552–557.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3414261/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22194413" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Ingolia NT, Lareau LF, Weissman JS. Ribosome profiling of mouse embryonic stem cells reveals the complexity <mark>and</mark> dynamics of mammalian proteomes. <span><span class="ref-journal">Cell. </span>2011;<span class="ref-vol">147</span>:789–802.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3225288/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22056041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Tyner AL, Fuchs E. Evidence for posttranscriptional regulation of the keratins expressed during hyperproliferation <mark>and</mark> malignant transformation in human epidermis. <span><span class="ref-journal">J Cell Biol. </span>1986;<span class="ref-vol">103</span>:1945–1955.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2114388/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/2430980" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Signer RAJ, Magee JA, Salic A, Morrison SJ. Haematopoietic stem cells require a highly regulated protein synthesis rate. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">509</span>:49–54.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4015626/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/24670665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Blanco S, et al.  Stem cell function <mark>and</mark> stress response are controlled by protein synthesis. <span><span class="ref-journal">Nature. </span>2016;<span class="ref-vol">534</span>:335–340.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC5040503/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/27306184" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Calviello L, et al.  Detecting actively translated open reading frames in ribosome profiling data. <span><span class="ref-journal">Nat Methods. </span>2016;<span class="ref-vol">13</span>:165–170.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/26657557" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Slavoff SA, et al.  Peptidomic discovery of short open reading frame-encoded peptides in human cells. <span><span class="ref-journal">Nat Chem Biol. </span>2013;<span class="ref-vol">9</span>:59–64.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3625679/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23160002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Falini B, et al.  Translocations <mark>and</mark> mutations involving the nucleophosmin (NPM1) gene in lymphomas <mark>and</mark> leukemias. <span><span class="ref-journal">Haematologica. </span>2007;<span class="ref-vol">92</span>:519–532.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17488663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Yang H, et al.  ETS family transcriptional regulators drive chromatin dynamics <mark>and</mark> malignancy in squamous cell carcinomas. <span><span class="ref-journal">eLife. </span>2015;<span class="ref-vol">4</span>:e10870.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4739765/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/26590320" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Morris DR, Geballe AP. Upstream open reading frames as regulators of mRNA translation. <span><span class="ref-journal">Mol Cell Biol. </span>2000;<span class="ref-vol">20</span>:8635–8642.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC86464/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/11073965" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Luo B, et al.  Highly parallel identification of essential genes in cancer cells. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2008;<span class="ref-vol">105</span>:20380–20385.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2629277/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19091943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Hinnebusch AG. The scanning mechanism of eukaryotic translation initiation. <span><span class="ref-journal">Annu Rev Biochem. </span>2014;<span class="ref-vol">83</span>:779–812.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/24499181" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Harding HP, et al.  Regulated translation initiation controls stress-induced gene expression in mammalian cells. <span><span class="ref-journal">Mol Cell. </span>2000;<span class="ref-vol">6</span>:1099–1108.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/11106749" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Skabkin MA, et al.  Activities of Ligatin <mark>and</mark> MCT-1/DENR in eukaryotic translation initiation <mark>and</mark> ribosomal recycling. <span><span class="ref-journal">Genes Dev. </span>2010;<span class="ref-vol">24</span>:1787–1801.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2922506/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/20713520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Dmitriev SE, et al.  GTP-independent tRNA delivery to the ribosomal P-site by a novel eukaryotic translation factor. <span><span class="ref-journal">J Biol Chem. </span>2010;<span class="ref-vol">285</span>:26779–26787.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2930676/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/20566627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Zoll WL, Horton LE, Komar AA, Hensold JO, Merrick WC. Characterization of mammalian eIF2A <mark>and</mark> identification of the yeast homolog. <span><span class="ref-journal">J Biol Chem. </span>2002;<span class="ref-vol">277</span>:37079–37087.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12133843" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Terenin IM, Dmitriev SE, Andreev DE, Shatsky IN. Eukaryotic translation initiation machinery can operate in a bacterial-like mode without eIF2. <span><span class="ref-journal">Nat Struct Mol Biol. </span>2008;<span class="ref-vol">15</span>:836–841.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/18604219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Starck SR, et al.  Leucine-tRNA initiates at CUG start codons for protein synthesis <mark>and</mark> presentation by MHC class I. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">336</span>:1719–1723.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22745432" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Starck SR, et al.  Translation from the 5′ untranslated region shapes the integrated stress response. <span><span class="ref-journal">Science. </span>2016;<span class="ref-vol">351</span>:aad3867.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4882168/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/26823435" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Cancer Genome Atlas Network. Comprehensive genomic characterization of head <mark>and</mark> neck squamous cell carcinomas. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">517</span>:576–582.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4311405/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/25631445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Liu B, Qian SB. Translational reprogramming in cellular stress response. <span><span class="ref-journal">WIREs RNA. </span>2014;<span class="ref-vol">5</span>:301–315.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3991730/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/24375939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Koromilas AE. Roles of the translation initiation factor eIF2α serine 51 phosphorylation in cancer formation <mark>and</mark> treatment. <span><span class="ref-journal">Biochim Biophys Acta. </span>2015;<span class="ref-vol">1849</span>:871–880.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/25497381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Komar AA, et al.  Novel characteristics of the biological properties of the yeast <em>Saccharomyces cerevisiae</em> eukaryotic initiation factor 2A. <span><span class="ref-journal">J Biol Chem. </span>2005;<span class="ref-vol">280</span>:15601–15611.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/15718232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Reineke LC, Cao Y, Baus D, Hossain NM, Merrick WC. Insights into the role of yeast eIF2A in IRES-mediated translation. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>:e24492.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3168509/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/21915340" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Holcik M. Could the eIF2α-independent translation be the achilles heel of cancer? <span><span class="ref-journal">Front Oncol. </span>2015;<span class="ref-vol">5</span>:264.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4659918/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/26636041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Liang H, et al.  PTENα, a PTEN isoform translated through alternative initiation, regulates mitochondrial function <mark>and</mark> energy metabolism. <span><span class="ref-journal">Cell Metab. </span>2014;<span class="ref-vol">19</span>:836–848.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4097321/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/24768297" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Love MI, Huber W, Anders S. Moderated estimation of fold change <mark>and</mark> dispersion for RNA-seq data <mark>with</mark> DESeq2. <span><span class="ref-journal">Genome Biol. </span>2014;<span class="ref-vol">15</span>:550.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4302049/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/25516281" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Gerashchenko MV, Gladyshev VN. Translation inhibitors cause abnormalities in ribosome profiling experiments. <span><span class="ref-journal">Nucleic Acids Res. </span>2014;<span class="ref-vol">42</span>:e134.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4176156/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/25056308" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Ingolia NT, Brar GA, Rouskin S, McGeachy AM, Weissman JS. The ribosome profiling strategy for monitoring translation <em>in vivo</em> by deep sequencing of ribosome-protected mRNA fragments. <span><span class="ref-journal">Nat Protocols. </span>2012;<span class="ref-vol">7</span>:1534–1550.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3535016/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22836135" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Langmead B, Salzberg SL. Fast gapped-read alignment <mark>with</mark> Bowtie 2. <span><span class="ref-journal">Nat Methods. </span>2012;<span class="ref-vol">9</span>:357–359.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3322381/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22388286" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Kim D, et al.  TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions <mark>and</mark> gene fusions. <span><span class="ref-journal">Genome Biol. </span>2013;<span class="ref-vol">14</span>:R36.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4053844/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23618408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Dunn JG, Weissman JS. Plastid: nucleotide-resolution analysis of next-generation sequencing <mark>and</mark> genomics data. <span><span class="ref-journal">BMC Genomics. </span>2016;<span class="ref-vol">17</span>:958.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC5120557/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/27875984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Beronja S, Fuchs E. RNAi-mediated gene function analysis in skin. <span><span class="ref-journal">Methods Mol Biol. </span>2013;<span class="ref-vol">961</span>:351–361.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4160826/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23325656" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Platt RJ, et al.  CRISPR-Cas9 knockin mice for genome editing <mark>and</mark> cancer modeling. <span><span class="ref-journal">Cell. </span>2014;<span class="ref-vol">159</span>:440–455.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4265475/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/25263330" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Faul F, Erdfelder E, Buchner A, Lang AG. Statistical power analyses using G*Power 3.1: tests for correlation <mark>and</mark> regression analyses. <span><span class="ref-journal">Behav Res Methods. </span>2009;<span class="ref-vol">41</span>:1149–1160.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19897823" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Hu Y, Smyth GK. ELDA: extreme limiting dilution analysis for comparing depleted <mark>and</mark> enriched populations in stem cell <mark>and</mark> other assays. <span><span class="ref-journal">J Immunol Methods. </span>2009;<span class="ref-vol">347</span>:70–78.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19567251" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Blanpain C, Lowry WE, Geoghegan A, Polak L, Fuchs E. Self-renewal, multipotency, <mark>and</mark> the existence of two cell populations within an epithelial stem cell niche. <span><span class="ref-journal">Cell. </span>2004;<span class="ref-vol">118</span>:635–648.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/15339667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Sanjana NE, Shalem O, Zhang F. Improved vectors <mark>and</mark> genome-wide libraries for CRISPR screening. <span><span class="ref-journal">Nat Methods. </span>2014;<span class="ref-vol">11</span>:783–784.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC4486245/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/25075903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Ishihama Y, Rappsilber J, Mann M. Modular stop <mark>and</mark> go extraction tips <mark>with</mark> stacked disks for parallel <mark>and</mark> multidimensional peptide fractionation in proteomics. <span><span class="ref-journal">J Proteome Res. </span>2006;<span class="ref-vol">5</span>:988–994.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/16602707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Kleifeld O, et al.  Identifying <mark>and</mark> quantifying proteolytic events <mark>and</mark> the natural N terminome by terminal amine isotopic labeling of substrates. <span><span class="ref-journal">Nat Protocols. </span>2011;<span class="ref-vol">6</span>:1578–1611.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/21959240" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Hofacker IL. RNA secondary structure analysis using the Vienna RNA package. <span><span class="ref-journal">Curr Protoc Bioinformatics. </span>2009;<span class="ref-vol">Chapter 12</span>(Unit12.2)</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19496057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div></div></div><div style="width: 200px; top: -100px; left: -100px; display: none; " aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#__abstractidm139773288589168title">Abstract</a></li><li><a href="#S1title">Translational landscapes</a></li><li><a href="#S2title">Tumour-induced shifts in translation initiation</a></li><li><a href="#S3title">In vivo RNAi screen exposes an eIF2 regulatory node</a></li><li><a href="#S4title">eIF2A is an essential factor for tumour initiation</a></li><li><a href="#S5title">eIF2A, protein synthesis <mark>and</mark> SCC prognosis</a></li><li><a href="#S6title">Discussion</a></li><li><a href="#S7title">METHODS</a></li><li><a href="#S29title">Extended Data</a></li><li><a href="#S30title">Supplementary Material</a></li><li><a href="#S31title">Acknowledgments</a></li><li><a href="#__fn-groupidm139773289937872title">Footnotes</a></li><li><a href="#__ref-listidm139773289930048title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5287289/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5287289/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5287289/pdf/nihms845657.pdf">PDF (3.7M)</a></li> | <li><a href="#" data-citationid="PMC5287289" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5287289%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5287289%2F&amp;text=Translation%20from%20unconventional%205%E2%80%B2%20start%20sites%20drives%20tumour%20initiation" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5287289%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28077873" data-aiid="5287289" data-aid="5287289" data-iid="286576" data-domainid="319" data-domain="nihpa" data-accid="PMC5287289" data-md5="bde71858f32ec68d400b63c62084a892">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-4" role="button" aria-expanded="true" style="position: absolute; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; "></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-4" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input type="hidden" name="absid" id="absid" value="5287289"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="width: 200px; top: -100px; left: -100px; display: none; " aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block; "><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block; "><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input type="submit" class="button_apply" name="addtocollection" value="1"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; "></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/27313038" ref="reftype=relart&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Translational control by 5'-untranslated regions of eukaryotic mRNAs.</a><span class="source">[Science. 2016]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-6" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Hinnebusch AG, Ivanov IP, Sonenberg N. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Science. 2016 Jun 17; 352(6292):1413-6. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/16024603" ref="reftype=relart&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Epidermal expression of the translation inhibitor programmed cell death 4 suppresses tumorigenesis.</a><span class="source">[Cancer Res. 2005]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Jansen AP, Camalier CE, Colburn NH. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Cancer Res. 2005 Jul 15; 65(14):6034-41. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/24909994" ref="reftype=relart&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">SOX2 controls tumour initiation <mark>and</mark> cancer stem-cell functions in squamous-cell carcinoma.</a><span class="source">[Nature. 2014]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Boumahdi S, Driessens G, Lapouge G, Rorive S, Nassar D, Le Mercier M, Delatte B, Caauwe A, Lenglez S, Nkusi E, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature. 2014 Jul 10; 511(7508):246-50. Epub 2014 Jun 8.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/26823435" ref="reftype=relart&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Translation from the 5' untranslated region shapes the integrated stress response.</a><span class="source">[Science. 2016]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Starck SR, Tsai JC, Chen K, Shodiya M, Wang L, Yahiro K, Martins-Green M, Shastri N, Walter P. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Science. 2016 Jan 29; 351(6272):aad3867. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/27358398" ref="reftype=relart&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Upstream Open Reading Frames Differentially Regulate Gene-specific Translation in the Integrated Stress Response.</a><span class="source">[J Biol Chem. 2016]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Young SK, Wek RC. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">J Biol Chem. 2016 Aug 12; 291(33):16927-35. Epub 2016 Jun 29.</em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=28077873&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=28077873&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-11" role="button" aria-expanded="true" style="position: absolute; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; "></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-11" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=bioproject&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_bioproject&amp;IdsFromResult=5287289" ref="reftype=BioProject&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">BioProject</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-12" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">BioProject<div class="brieflinkpopdesc">BioProject links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=nucest&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_nucest&amp;IdsFromResult=5287289" ref="reftype=EST&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">EST</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">EST<div class="brieflinkpopdesc">Published EST sequences</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gds&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gds&amp;IdsFromResult=5287289" ref="reftype=GEO%20DataSets&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|GEO%20DataSets&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">GEO DataSets</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">GEO DataSets<div class="brieflinkpopdesc">GEO DataSet links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/28077873/" ref="reftype=PubMed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=sra&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_sra&amp;IdsFromResult=5287289" ref="reftype=SRA&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">SRA</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">SRA<div class="brieflinkpopdesc">Links to SRA</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=5287289" ref="reftype=Taxonomy&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-17" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-18" role="button" aria-expanded="true" style="position: absolute; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; "></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-18" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity"><li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=59a5eda27f478fb30109cf69">Translation from unconventional 5′ start sites drives tumour initiation</a><div class="ralinkpop offscreen_noflow">Translation from unconventional 5′ start sites drives tumour initiation<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2017 Jan 26; 541(7638)494</div>
</div>
<div class="tertiary"></div>
</li></ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--P2" name="crb--P2" rid="P2" style="position: absolute; visibility: visible; width: 241px; top: 1024px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-19"><ul>
<li class="two_line" reference_id="R1">
<a href="/pubmed/21593866/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Global quantification of mammalian gene expression control.</a><span class="one_line_source">[Nature.  2011]</span><div class="alt-note">
<div class="authors">Schwanhäusser B, Busse D, Li N, Dittmar G, Schuchhardt J, Wolf J, Chen W, Selbach M</div>
<div class="citation">Nature. 2011 May 19; 473(7347):337-42.</div>
</div>
</li>
<li class="two_line" reference_id="R2">
<a href="/pubmed/25043054/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Proteogenomic characterization of human colon <mark>and</mark> rectal cancer.</a><span class="one_line_source">[Nature.  2014]</span><div class="alt-note">
<div class="authors">Zhang B, Wang J, Wang X, Zhu J, Liu Q, Shi Z, Chambers MC, Zimmerman LJ, Shaddox KF, Kim S, Davies SR, Wang S, Wang P, Kinsinger CR, Rivers RC, Rodriguez H, Townsend RR, Ellis MJ, Carr SA, Tabb DL, Coffey RJ, Slebos RJ, Liebler DC, NCI CPTAC.</div>
<div class="citation">Nature. 2014 Sep 18; 513(7518):382-7.</div>
</div>
</li>
<li class="two_line" reference_id="R3">
<a href="/pubmed/17041626/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> mTOR, translation initiation <mark>and</mark> cancer.</a><span class="one_line_source">[Oncogene.  2006]</span><div class="alt-note">
<div class="authors">Mamane Y, Petroulakis E, LeBacquer O, Sonenberg N</div>
<div class="citation">Oncogene. 2006 Oct 16; 25(48):6416-22.</div>
</div>
</li>
<li class="two_line" reference_id="R6">
<a href="/pubmed/22367541/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The translational landscape of mTOR signalling steers cancer initiation <mark>and</mark> metastasis.</a><span class="one_line_source">[Nature.  2012]</span><div class="alt-note">
<div class="authors">Hsieh AC, Liu Y, Edlind MP, Ingolia NT, Janes MR, Sher A, Shi EY, Stumpf CR, Christensen C, Bonham MJ, Wang S, Ren P, Martin M, Jessen K, Feldman ME, Weissman JS, Shokat KM, Rommel C, Ruggero D</div>
<div class="citation">Nature. 2012 Feb 22; 485(7396):55-61.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P3" name="crb--P3" rid="P3" style="position: absolute; visibility: visible; width: 241px; top: 1280px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-20"><ul>
<li class="two_line" reference_id="R7">
<a href="/pubmed/25183851/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Modeling cutaneous squamous carcinoma development in the mouse.</a><span class="one_line_source">[Cold Spring Harb Perspect Med.  2014]</span><div class="alt-note">
<div class="authors">Huang PY, Balmain A</div>
<div class="citation">Cold Spring Harb Perspect Med. 2014 Sep 2; 4(9):a013623.</div>
</div>
</li>
<li class="two_line" reference_id="R8">
<a href="/pubmed/17015430/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Sox2 is required for development of taste bud sensory cells.</a><span class="one_line_source">[Genes Dev.  2006]</span><div class="alt-note">
<div class="authors">Okubo T, Pevny LH, Hogan BL</div>
<div class="citation">Genes Dev. 2006 Oct 1; 20(19):2654-9.</div>
</div>
</li>
<li class="two_line" reference_id="R11">
<a href="/pubmed/22941189/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Comprehensive genomic analysis identifies SOX2 as a frequently amplified gene in small-cell lung cancer.</a><span class="one_line_source">[Nat Genet.  2012]</span><div class="alt-note">
<div class="authors">Rudin CM, Durinck S, Stawiski EW, Poirier JT, Modrusan Z, Shames DS, Bergbower EA, Guan Y, Shin J, Guillory J, Rivers CS, Foo CK, Bhatt D, Stinson J, Gnad F, Haverty PM, Gentleman R, Chaudhuri S, Janakiraman V, Jaiswal BS, Parikh C, Yuan W, Zhang Z, Koeppen H, Wu TD, Stern HM, Yauch RL, Huffman KE, Paskulin DD, Illei PB, Varella-Garcia M, Gazdar AF, de Sauvage FJ, Bourgon R, Minna JD, Brock MV, Seshagiri S</div>
<div class="citation">Nat Genet. 2012 Oct; 44(10):1111-6.</div>
</div>
</li>
<li class="two_line" reference_id="R12">
<a href="/pubmed/23550210/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Integrative analysis of complex cancer genomics <mark>and</mark> clinical profiles using the cBioPortal.</a><span class="one_line_source">[Sci Signal.  2013]</span><div class="alt-note">
<div class="authors">Gao J, Aksoy BA, Dogrusoz U, Dresdner G, Gross B, Sumer SO, Sun Y, Jacobsen A, Sinha R, Larsson E, Cerami E, Sander C, Schultz N</div>
<div class="citation">Sci Signal. 2013 Apr 2; 6(269):pl1.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P4" name="crb--P4" rid="P4" style="position: absolute; visibility: visible; width: 241px; top: 1646px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-21"><ul>
<li class="two_line" reference_id="R13">
<a href="/pubmed/23945586/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">RNAi screens in mice identify physiological regulators of oncogenic growth.</a><span class="one_line_source">[Nature.  2013]</span><div class="alt-note">
<div class="authors">Beronja S, Janki P, Heller E, Lien WH, Keyes BE, Oshimori N, Fuchs E</div>
<div class="citation">Nature. 2013 Sep 12; 501(7466):185-90.</div>
</div>
</li>
<li class="two_line" reference_id="R14">
<a href="/pubmed/20526348/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Rapid functional dissection of genetic networks via tissue-specific transduction <mark>and</mark> RNAi in mouse embryos.</a><span class="one_line_source">[Nat Med.  2010]</span><div class="alt-note">
<div class="authors">Beronja S, Livshits G, Williams S, Fuchs E</div>
<div class="citation">Nat Med. 2010 Jul; 16(7):821-7.</div>
</div>
</li>
<li class="two_line" reference_id="R15">
<a href="/pubmed/23472872/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Sox2 cooperates <mark>with</mark> inflammation-mediated Stat3 activation in the malignant transformation of foregut basal progenitor cells.</a><span class="one_line_source">[Cell Stem Cell.  2013]</span><div class="alt-note">
<div class="authors">Liu K, Jiang M, Lu Y, Chen H, Sun J, Wu S, Ku WY, Nakagawa H, Kita Y, Natsugoe S, Peters JH, Rustgi A, Onaitis MW, Kiernan A, Chen X, Que J</div>
<div class="citation">Cell Stem Cell. 2013 Mar 7; 12(3):304-15.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P5" name="crb--P5" rid="P5" style="position: absolute; visibility: visible; width: 241px; top: 1982px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-22"><ul>
<li class="two_line" reference_id="R16">
<a href="/pubmed/19213877/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide analysis in vivo of translation <mark>with</mark> nucleotide resolution using ribosome profiling.</a><span class="one_line_source">[Science.  2009]</span><div class="alt-note">
<div class="authors">Ingolia NT, Ghaemmaghami S, Newman JR, Weissman JS</div>
<div class="citation">Science. 2009 Apr 10; 324(5924):218-23.</div>
</div>
</li>
<li class="two_line" reference_id="R18">
<a href="/pubmed/22056041/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Ribosome profiling of mouse embryonic stem cells reveals the complexity <mark>and</mark> dynamics of mammalian proteomes.</a><span class="one_line_source">[Cell.  2011]</span><div class="alt-note">
<div class="authors">Ingolia NT, Lareau LF, Weissman JS</div>
<div class="citation">Cell. 2011 Nov 11; 147(4):789-802.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P6" name="crb--P6" rid="P6" style="position: absolute; visibility: visible; width: 241px; top: 2327px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-23"><ul><li class="two_line" reference_id="R19">
<a href="/pubmed/2430980/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Evidence for posttranscriptional regulation of the keratins expressed during hyperproliferation <mark>and</mark> malignant transformation in human epidermis.</a><span class="one_line_source">[J Cell Biol.  1986]</span><div class="alt-note">
<div class="authors">Tyner AL, Fuchs E</div>
<div class="citation">J Cell Biol. 1986 Nov; 103(5):1945-55.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P7" name="crb--P7" rid="P7" style="position: absolute; visibility: visible; width: 241px; top: 2604px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-24"><ul>
<li class="two_line" reference_id="R20">
<a href="/pubmed/24670665/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Haematopoietic stem cells require a highly regulated protein synthesis rate.</a><span class="one_line_source">[Nature.  2014]</span><div class="alt-note">
<div class="authors">Signer RA, Magee JA, Salic A, Morrison SJ</div>
<div class="citation">Nature. 2014 May 1; 509(7498):49-54.</div>
</div>
</li>
<li class="two_line" reference_id="R21">
<a href="/pubmed/27306184/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Stem cell function <mark>and</mark> stress response are controlled by protein synthesis.</a><span class="one_line_source">[Nature.  2016]</span><div class="alt-note">
<div class="authors">Blanco S, Bandiera R, Popis M, Hussain S, Lombard P, Aleksic J, Sajini A, Tanna H, Cortés-Garrido R, Gkatza N, Dietmann S, Frye M</div>
<div class="citation">Nature. 2016 Jun 16; 534(7607):335-40.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P8" name="crb--P8" rid="P8" style="position: absolute; visibility: visible; width: 241px; top: 2982px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-25"><ul><li class="two_line" reference_id="R21">
<a href="/pubmed/27306184/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Stem cell function <mark>and</mark> stress response are controlled by protein synthesis.</a><span class="one_line_source">[Nature.  2016]</span><div class="alt-note">
<div class="authors">Blanco S, Bandiera R, Popis M, Hussain S, Lombard P, Aleksic J, Sajini A, Tanna H, Cortés-Garrido R, Gkatza N, Dietmann S, Frye M</div>
<div class="citation">Nature. 2016 Jun 16; 534(7607):335-40.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P10" name="crb--P10" rid="P10" style="position: absolute; visibility: visible; width: 241px; top: 3619px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-26"><ul><li class="two_line" reference_id="R22">
<a href="/pubmed/26657557/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Detecting actively translated open reading frames in ribosome profiling data.</a><span class="one_line_source">[Nat Methods.  2016]</span><div class="alt-note">
<div class="authors">Calviello L, Mukherjee N, Wyler E, Zauber H, Hirsekorn A, Selbach M, Landthaler M, Obermayer B, Ohler U</div>
<div class="citation">Nat Methods. 2016 Feb; 13(2):165-70.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P11" name="crb--P11" rid="P11" style="position: absolute; visibility: visible; width: 241px; top: 4017px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-27"><ul><li class="two_line" reference_id="R23">
<a href="/pubmed/23160002/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Peptidomic discovery of short open reading frame-encoded peptides in human cells.</a><span class="one_line_source">[Nat Chem Biol.  2013]</span><div class="alt-note">
<div class="authors">Slavoff SA, Mitchell AJ, Schwaid AG, Cabili MN, Ma J, Levin JZ, Karger AD, Budnik BA, Rinn JL, Saghatelian A</div>
<div class="citation">Nat Chem Biol. 2013 Jan; 9(1):59-64.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P13" name="crb--P13" rid="P13" style="position: absolute; visibility: visible; width: 241px; top: 4437px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul><li class="two_line" reference_id="R24">
<a href="/pubmed/17488663/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Translocations <mark>and</mark> mutations involving the nucleophosmin (NPM1) gene in lymphomas <mark>and</mark> leukemias.</a><span class="one_line_source">[Haematologica.  2007]</span><div class="alt-note">
<div class="authors">Falini B, Nicoletti I, Bolli N, Martelli MP, Liso A, Gorello P, Mandelli F, Mecucci C, Martelli MF</div>
<div class="citation">Haematologica. 2007 Apr; 92(4):519-32.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P14" name="crb--P14" rid="P14" style="position: absolute; visibility: visible; width: 241px; top: 4758px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul><li class="two_line" reference_id="R25">
<a href="/pubmed/26590320/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">ETS family transcriptional regulators drive chromatin dynamics <mark>and</mark> malignancy in squamous cell carcinomas.</a><span class="one_line_source">[Elife.  2015]</span><div class="alt-note">
<div class="authors">Yang H, Schramek D, Adam RC, Keyes BE, Wang P, Zheng D, Fuchs E</div>
<div class="citation">Elife. 2015 Nov 21; 4():e10870.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P15" name="crb--P15" rid="P15" style="position: absolute; visibility: visible; width: 241px; top: 5229px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul><li class="two_line" reference_id="R26">
<a href="/pubmed/11073965/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Upstream open reading frames as regulators of mRNA translation.</a><span class="one_line_source">[Mol Cell Biol.  2000]</span><div class="alt-note">
<div class="authors">Morris DR, Geballe AP</div>
<div class="citation">Mol Cell Biol. 2000 Dec; 20(23):8635-42.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P16" name="crb--P16" rid="P16" style="position: absolute; visibility: visible; width: 241px; top: 5373px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul><li class="two_line" reference_id="R14">
<a href="/pubmed/20526348/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Rapid functional dissection of genetic networks via tissue-specific transduction <mark>and</mark> RNAi in mouse embryos.</a><span class="one_line_source">[Nat Med.  2010]</span><div class="alt-note">
<div class="authors">Beronja S, Livshits G, Williams S, Fuchs E</div>
<div class="citation">Nat Med. 2010 Jul; 16(7):821-7.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P17" name="crb--P17" rid="P17" style="position: absolute; visibility: visible; width: 241px; top: 5792px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-32"><ul><li class="two_line" reference_id="R27">
<a href="/pubmed/19091943/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Highly parallel identification of essential genes in cancer cells.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2008]</span><div class="alt-note">
<div class="authors">Luo B, Cheung HW, Subramanian A, Sharifnia T, Okamoto M, Yang X, Hinkle G, Boehm JS, Beroukhim R, Weir BA, Mermel C, Barbie DA, Awad T, Zhou X, Nguyen T, Piqani B, Li C, Golub TR, Meyerson M, Hacohen N, Hahn WC, Lander ES, Sabatini DM, Root DE</div>
<div class="citation">Proc Natl Acad Sci U S A. 2008 Dec 23; 105(51):20380-5.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P18" name="crb--P18" rid="P18" style="position: absolute; visibility: visible; width: 241px; top: 5981px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-33"><ul>
<li class="two_line" reference_id="R5">
<a href="/pubmed/19239892/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Regulation of translation initiation in eukaryotes: mechanisms <mark>and</mark> biological targets.</a><span class="one_line_source">[Cell.  2009]</span><div class="alt-note">
<div class="authors">Sonenberg N, Hinnebusch AG</div>
<div class="citation">Cell. 2009 Feb 20; 136(4):731-45.</div>
</div>
</li>
<li class="two_line" reference_id="R28">
<a href="/pubmed/24499181/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> The scanning mechanism of eukaryotic translation initiation.</a><span class="one_line_source">[Annu Rev Biochem.  2014]</span><div class="alt-note">
<div class="authors">Hinnebusch AG</div>
<div class="citation">Annu Rev Biochem. 2014; 83():779-812.</div>
</div>
</li>
<li class="two_line" reference_id="R29">
<a href="/pubmed/11106749/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Regulated translation initiation controls stress-induced gene expression in mammalian cells.</a><span class="one_line_source">[Mol Cell.  2000]</span><div class="alt-note">
<div class="authors">Harding HP, Novoa I, Zhang Y, Zeng H, Wek R, Schapira M, Ron D</div>
<div class="citation">Mol Cell. 2000 Nov; 6(5):1099-108.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P19" name="crb--P19" rid="P19" style="position: absolute; visibility: visible; width: 241px; top: 6298px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-34"><ul>
<li class="two_line" reference_id="R30">
<a href="/pubmed/20713520/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Activities of Ligatin <mark>and</mark> MCT-1/DENR in eukaryotic translation initiation <mark>and</mark> ribosomal recycling.</a><span class="one_line_source">[Genes Dev.  2010]</span><div class="alt-note">
<div class="authors">Skabkin MA, Skabkina OV, Dhote V, Komar AA, Hellen CU, Pestova TV</div>
<div class="citation">Genes Dev. 2010 Aug 15; 24(16):1787-801.</div>
</div>
</li>
<li class="two_line" reference_id="R34">
<a href="/pubmed/22745432/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Leucine-tRNA initiates at CUG start codons for protein synthesis <mark>and</mark> presentation by MHC class I.</a><span class="one_line_source">[Science.  2012]</span><div class="alt-note">
<div class="authors">Starck SR, Jiang V, Pavon-Eternod M, Prasad S, McCarthy B, Pan T, Shastri N</div>
<div class="citation">Science. 2012 Jun 29; 336(6089):1719-23.</div>
</div>
</li>
<li class="two_line" reference_id="R35">
<a href="/pubmed/26823435/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Translation from the 5' untranslated region shapes the integrated stress response.</a><span class="one_line_source">[Science.  2016]</span><div class="alt-note">
<div class="authors">Starck SR, Tsai JC, Chen K, Shodiya M, Wang L, Yahiro K, Martins-Green M, Shastri N, Walter P</div>
<div class="citation">Science. 2016 Jan 29; 351(6272):aad3867.</div>
</div>
</li>
<li class="two_line" reference_id="R18">
<a href="/pubmed/22056041/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Ribosome profiling of mouse embryonic stem cells reveals the complexity <mark>and</mark> dynamics of mammalian proteomes.</a><span class="one_line_source">[Cell.  2011]</span><div class="alt-note">
<div class="authors">Ingolia NT, Lareau LF, Weissman JS</div>
<div class="citation">Cell. 2011 Nov 11; 147(4):789-802.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P21" name="crb--P21" rid="P21" style="position: absolute; visibility: visible; width: 241px; top: 6790px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-35"><ul><li class="two_line" reference_id="R25">
<a href="/pubmed/26590320/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">ETS family transcriptional regulators drive chromatin dynamics <mark>and</mark> malignancy in squamous cell carcinomas.</a><span class="one_line_source">[Elife.  2015]</span><div class="alt-note">
<div class="authors">Yang H, Schramek D, Adam RC, Keyes BE, Wang P, Zheng D, Fuchs E</div>
<div class="citation">Elife. 2015 Nov 21; 4():e10870.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P25" name="crb--P25" rid="P25" style="position: absolute; visibility: visible; width: 241px; top: 8161px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-36"><ul><li class="two_line" reference_id="R36">
<a href="/pubmed/25631445/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Comprehensive genomic characterization of head <mark>and</mark> neck squamous cell carcinomas.</a><span class="one_line_source">[Nature.  2015]</span><div class="alt-note">
<div class="authors">Cancer Genome Atlas Network.</div>
<div class="citation">Nature. 2015 Jan 29; 517(7536):576-82.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P26" name="crb--P26" rid="P26" style="position: absolute; visibility: visible; width: 241px; top: 8368px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-37"><ul>
<li class="two_line" reference_id="R37">
<a href="/pubmed/24375939/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Translational reprogramming in cellular stress response.</a><span class="one_line_source">[Wiley Interdiscip Rev RNA.  2014]</span><div class="alt-note">
<div class="authors">Liu B, Qian SB</div>
<div class="citation">Wiley Interdiscip Rev RNA. 2014 May-Jun; 5(3):301-15.</div>
</div>
</li>
<li class="two_line" reference_id="R38">
<a href="/pubmed/25497381/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Roles of the translation initiation factor eIF2α serine 51 phosphorylation in cancer formation <mark>and</mark> treatment.</a><span class="one_line_source">[Biochim Biophys Acta.  2015]</span><div class="alt-note">
<div class="authors">Koromilas AE</div>
<div class="citation">Biochim Biophys Acta. 2015 Jul; 1849(7):871-80.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P27" name="crb--P27" rid="P27" style="position: absolute; visibility: visible; width: 241px; top: 8557px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-38"><ul>
<li class="two_line" reference_id="R32">
<a href="/pubmed/12133843/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Characterization of mammalian eIF2A <mark>and</mark> identification of the yeast homolog.</a><span class="one_line_source">[J Biol Chem.  2002]</span><div class="alt-note">
<div class="authors">Zoll WL, Horton LE, Komar AA, Hensold JO, Merrick WC</div>
<div class="citation">J Biol Chem. 2002 Oct 4; 277(40):37079-87.</div>
</div>
</li>
<li class="two_line" reference_id="R39">
<a href="/pubmed/15718232/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Novel characteristics of the biological properties of the yeast Saccharomyces cerevisiae eukaryotic initiation factor 2A.</a><span class="one_line_source">[J Biol Chem.  2005]</span><div class="alt-note">
<div class="authors">Komar AA, Gross SR, Barth-Baus D, Strachan R, Hensold JO, Goss Kinzy T, Merrick WC</div>
<div class="citation">J Biol Chem. 2005 Apr 22; 280(16):15601-11.</div>
</div>
</li>
<li class="two_line" reference_id="R40">
<a href="/pubmed/21915340/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Insights into the role of yeast eIF2A in IRES-mediated translation.</a><span class="one_line_source">[PLoS One.  2011]</span><div class="alt-note">
<div class="authors">Reineke LC, Cao Y, Baus D, Hossain NM, Merrick WC</div>
<div class="citation">PLoS One. 2011; 6(9):e24492.</div>
</div>
</li>
<li class="two_line" reference_id="R38">
<a href="/pubmed/25497381/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Roles of the translation initiation factor eIF2α serine 51 phosphorylation in cancer formation <mark>and</mark> treatment.</a><span class="one_line_source">[Biochim Biophys Acta.  2015]</span><div class="alt-note">
<div class="authors">Koromilas AE</div>
<div class="citation">Biochim Biophys Acta. 2015 Jul; 1849(7):871-80.</div>
</div>
</li>
<li class="two_line" reference_id="R41">
<a href="/pubmed/26636041/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Could the eIF2α-Independent Translation Be the Achilles Heel of Cancer?</a><span class="one_line_source">[Front Oncol.  2015]</span><div class="alt-note">
<div class="authors">Holcik M</div>
<div class="citation">Front Oncol. 2015; 5():264.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P28" name="crb--P28" rid="P28" style="position: absolute; visibility: visible; width: 241px; top: 8812px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-39"><ul>
<li class="two_line" reference_id="R42">
<a href="/pubmed/24768297/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PTENα, a PTEN isoform translated through alternative initiation, regulates mitochondrial function <mark>and</mark> energy metabolism.</a><span class="one_line_source">[Cell Metab.  2014]</span><div class="alt-note">
<div class="authors">Liang H, He S, Yang J, Jia X, Wang P, Chen X, Zhang Z, Zou X, McNutt MA, Shen WH, Yin Y</div>
<div class="citation">Cell Metab. 2014 May 6; 19(5):836-48.</div>
</div>
</li>
<li class="two_line" reference_id="R20">
<a href="/pubmed/24670665/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Haematopoietic stem cells require a highly regulated protein synthesis rate.</a><span class="one_line_source">[Nature.  2014]</span><div class="alt-note">
<div class="authors">Signer RA, Magee JA, Salic A, Morrison SJ</div>
<div class="citation">Nature. 2014 May 1; 509(7498):49-54.</div>
</div>
</li>
<li class="two_line" reference_id="R21">
<a href="/pubmed/27306184/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Stem cell function <mark>and</mark> stress response are controlled by protein synthesis.</a><span class="one_line_source">[Nature.  2016]</span><div class="alt-note">
<div class="authors">Blanco S, Bandiera R, Popis M, Hussain S, Lombard P, Aleksic J, Sajini A, Tanna H, Cortés-Garrido R, Gkatza N, Dietmann S, Frye M</div>
<div class="citation">Nature. 2016 Jun 16; 534(7607):335-40.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P30" name="crb--P30" rid="P30" style="position: absolute; visibility: visible; width: 241px; top: 9467px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-40"><ul>
<li class="two_line" reference_id="R44">
<a href="/pubmed/25056308/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Translation inhibitors cause abnormalities in ribosome profiling experiments.</a><span class="one_line_source">[Nucleic Acids Res.  2014]</span><div class="alt-note">
<div class="authors">Gerashchenko MV, Gladyshev VN</div>
<div class="citation">Nucleic Acids Res. 2014; 42(17):e134.</div>
</div>
</li>
<li class="two_line" reference_id="R45">
<a href="/pubmed/22836135/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The ribosome profiling strategy for monitoring translation in vivo by deep sequencing of ribosome-protected mRNA fragments.</a><span class="one_line_source">[Nat Protoc.  2012]</span><div class="alt-note">
<div class="authors">Ingolia NT, Brar GA, Rouskin S, McGeachy AM, Weissman JS</div>
<div class="citation">Nat Protoc. 2012 Jul 26; 7(8):1534-50.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P33" name="crb--P33" rid="P33" style="position: absolute; visibility: visible; width: 241px; top: 10280px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-41"><ul><li class="two_line" reference_id="R45">
<a href="/pubmed/22836135/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The ribosome profiling strategy for monitoring translation in vivo by deep sequencing of ribosome-protected mRNA fragments.</a><span class="one_line_source">[Nat Protoc.  2012]</span><div class="alt-note">
<div class="authors">Ingolia NT, Brar GA, Rouskin S, McGeachy AM, Weissman JS</div>
<div class="citation">Nat Protoc. 2012 Jul 26; 7(8):1534-50.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P35" name="crb--P35" rid="P35" style="position: absolute; visibility: visible; width: 241px; top: 10871px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-42"><ul>
<li class="two_line" reference_id="R46">
<a href="/pubmed/22388286/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Fast gapped-read alignment <mark>with</mark> Bowtie 2.</a><span class="one_line_source">[Nat Methods.  2012]</span><div class="alt-note">
<div class="authors">Langmead B, Salzberg SL</div>
<div class="citation">Nat Methods. 2012 Mar 4; 9(4):357-9.</div>
</div>
</li>
<li class="two_line" reference_id="R47">
<a href="/pubmed/23618408/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions <mark>and</mark> gene fusions.</a><span class="one_line_source">[Genome Biol.  2013]</span><div class="alt-note">
<div class="authors">Kim D, Pertea G, Trapnell C, Pimentel H, Kelley R, Salzberg SL</div>
<div class="citation">Genome Biol. 2013 Apr 25; 14(4):R36.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P37" name="crb--P37" rid="P37">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-43"><ul><li class="expanded" reference_id="R48">
<a href="/pubmed/27875984/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Plastid: nucleotide-resolution analysis of next-generation sequencing <mark>and</mark> genomics data.</a><span class="one_line_source">[BMC Genomics.  2016]</span><div class="alt-note">
<div class="authors">Dunn JG, Weissman JS</div>
<div class="citation">BMC Genomics. 2016 Nov 22; 17(1):958.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P38" name="crb--P38" rid="P38">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-44"><ul><li class="expanded" reference_id="R43">
<a href="/pubmed/25516281/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Moderated estimation of fold change <mark>and</mark>  dispersion for RNA-seq data <mark>with</mark> DESeq2.</a><span class="one_line_source">[Genome Biol.  2014]</span><div class="alt-note">
<div class="authors">Love MI, Huber W, Anders S</div>
<div class="citation">Genome Biol. 2014; 15(12):550.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P48" name="crb--P48" rid="P48">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-45"><ul><li class="expanded" reference_id="R12">
<a href="/pubmed/23550210/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Integrative analysis of complex cancer genomics <mark>and</mark> clinical profiles using the cBioPortal.</a><span class="one_line_source">[Sci Signal.  2013]</span><div class="alt-note">
<div class="authors">Gao J, Aksoy BA, Dogrusoz U, Dresdner G, Gross B, Sumer SO, Sun Y, Jacobsen A, Sinha R, Larsson E, Cerami E, Sander C, Schultz N</div>
<div class="citation">Sci Signal. 2013 Apr 2; 6(269):pl1.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P50" name="crb--P50" rid="P50">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-46"><ul><li class="expanded" reference_id="R49">
<a href="/pubmed/23325656/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">RNAi-mediated gene function analysis in skin.</a><span class="one_line_source">[Methods Mol Biol.  2013]</span><div class="alt-note">
<div class="authors">Beronja S, Fuchs E</div>
<div class="citation">Methods Mol Biol. 2013; 961():351-61.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P51" name="crb--P51" rid="P51">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-47"><ul><li class="expanded" reference_id="R14">
<a href="/pubmed/20526348/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Rapid functional dissection of genetic networks via tissue-specific transduction <mark>and</mark> RNAi in mouse embryos.</a><span class="one_line_source">[Nat Med.  2010]</span><div class="alt-note">
<div class="authors">Beronja S, Livshits G, Williams S, Fuchs E</div>
<div class="citation">Nat Med. 2010 Jul; 16(7):821-7.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P53" name="crb--P53" rid="P53">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-48"><ul><li class="expanded" reference_id="R27">
<a href="/pubmed/19091943/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Highly parallel identification of essential genes in cancer cells.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2008]</span><div class="alt-note">
<div class="authors">Luo B, Cheung HW, Subramanian A, Sharifnia T, Okamoto M, Yang X, Hinkle G, Boehm JS, Beroukhim R, Weir BA, Mermel C, Barbie DA, Awad T, Zhou X, Nguyen T, Piqani B, Li C, Golub TR, Meyerson M, Hacohen N, Hahn WC, Lander ES, Sabatini DM, Root DE</div>
<div class="citation">Proc Natl Acad Sci U S A. 2008 Dec 23; 105(51):20380-5.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P54" name="crb--P54" rid="P54">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-49"><ul><li class="expanded" reference_id="R50">
<a href="/pubmed/25263330/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">CRISPR-Cas9 knockin mice for genome editing <mark>and</mark> cancer modeling.</a><span class="one_line_source">[Cell.  2014]</span><div class="alt-note">
<div class="authors">Platt RJ, Chen S, Zhou Y, Yim MJ, Swiech L, Kempton HR, Dahlman JE, Parnas O, Eisenhaure TM, Jovanovic M, Graham DB, Jhunjhunwala S, Heidenreich M, Xavier RJ, Langer R, Anderson DG, Hacohen N, Regev A, Feng G, Sharp PA, Zhang F</div>
<div class="citation">Cell. 2014 Oct 9; 159(2):440-55.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P56" name="crb--P56" rid="P56">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-50"><ul><li class="expanded" reference_id="R51">
<a href="/pubmed/19897823/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Statistical power analyses using G*Power 3.1: tests for correlation <mark>and</mark> regression analyses.</a><span class="one_line_source">[Behav Res Methods.  2009]</span><div class="alt-note">
<div class="authors">Faul F, Erdfelder E, Buchner A, Lang AG</div>
<div class="citation">Behav Res Methods. 2009 Nov; 41(4):1149-60.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P57" name="crb--P57" rid="P57">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-51"><ul><li class="expanded" reference_id="R52">
<a href="/pubmed/19567251/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">ELDA: extreme limiting dilution analysis for comparing depleted <mark>and</mark> enriched populations in stem cell <mark>and</mark> other assays.</a><span class="one_line_source">[J Immunol Methods.  2009]</span><div class="alt-note">
<div class="authors">Hu Y, Smyth GK</div>
<div class="citation">J Immunol Methods. 2009 Aug 15; 347(1-2):70-8.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P58" name="crb--P58" rid="P58">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-52"><ul>
<li class="expanded" reference_id="R53">
<a href="/pubmed/15339667/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Self-renewal, multipotency, <mark>and</mark> the existence of two cell populations within an epithelial stem cell niche.</a><span class="one_line_source">[Cell.  2004]</span><div class="alt-note">
<div class="authors">Blanpain C, Lowry WE, Geoghegan A, Polak L, Fuchs E</div>
<div class="citation">Cell. 2004 Sep 3; 118(5):635-48.</div>
</div>
</li>
<li class="expanded" reference_id="R25">
<a href="/pubmed/26590320/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">ETS family transcriptional regulators drive chromatin dynamics <mark>and</mark> malignancy in squamous cell carcinomas.</a><span class="one_line_source">[Elife.  2015]</span><div class="alt-note">
<div class="authors">Yang H, Schramek D, Adam RC, Keyes BE, Wang P, Zheng D, Fuchs E</div>
<div class="citation">Elife. 2015 Nov 21; 4():e10870.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P59" name="crb--P59" rid="P59">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-53"><ul><li class="expanded" reference_id="R54">
<a href="/pubmed/25075903/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Improved vectors <mark>and</mark> genome-wide libraries for CRISPR screening.</a><span class="one_line_source">[Nat Methods.  2014]</span><div class="alt-note">
<div class="authors">Sanjana NE, Shalem O, Zhang F</div>
<div class="citation">Nat Methods. 2014 Aug; 11(8):783-784.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P62" name="crb--P62" rid="P62">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-54"><ul><li class="expanded" reference_id="R55">
<a href="/pubmed/16602707/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Modular stop <mark>and</mark> go extraction tips <mark>with</mark> stacked disks for parallel <mark>and</mark> multidimensional Peptide fractionation in proteomics.</a><span class="one_line_source">[J Proteome Res.  2006]</span><div class="alt-note">
<div class="authors">Ishihama Y, Rappsilber J, Mann M</div>
<div class="citation">J Proteome Res. 2006 Apr; 5(4):988-94.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P67" name="crb--P67" rid="P67">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-55"><ul><li class="expanded" reference_id="R56">
<a href="/pubmed/21959240/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Identifying <mark>and</mark> quantifying proteolytic events <mark>and</mark> the natural N terminome by terminal amine isotopic labeling of substrates.</a><span class="one_line_source">[Nat Protoc.  2011]</span><div class="alt-note">
<div class="authors">Kleifeld O, Doucet A, Prudova A, auf dem Keller U, Gioia M, Kizhakkedathu JN, Overall CM</div>
<div class="citation">Nat Protoc. 2011 Sep 22; 6(10):1578-611.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P69" name="crb--P69" rid="P69">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-56"><ul><li class="expanded" reference_id="R57">
<a href="/pubmed/19496057/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">RNA secondary structure analysis using the Vienna RNA package.</a><span class="one_line_source">[Curr Protoc Bioinformatics.  2009]</span><div class="alt-note">
<div class="authors">Hofacker IL</div>
<div class="citation">Curr Protoc Bioinformatics. 2009 Jun; Chapter 12():Unit12.2.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P70" name="crb--P70" rid="P70">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-57"><ul><li class="expanded" reference_id="R22">
<a href="/pubmed/26657557/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Detecting actively translated open reading frames in ribosome profiling data.</a><span class="one_line_source">[Nat Methods.  2016]</span><div class="alt-note">
<div class="authors">Calviello L, Mukherjee N, Wyler E, Zauber H, Hirsekorn A, Selbach M, Landthaler M, Obermayer B, Ohler U</div>
<div class="citation">Nat Methods. 2016 Feb; 13(2):165-70.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P72" name="crb--P72" rid="P72">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-58"><ul><li class="expanded" reference_id="R18">
<a href="/pubmed/22056041/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Ribosome profiling of mouse embryonic stem cells reveals the complexity <mark>and</mark> dynamics of mammalian proteomes.</a><span class="one_line_source">[Cell.  2011]</span><div class="alt-note">
<div class="authors">Ingolia NT, Lareau LF, Weissman JS</div>
<div class="citation">Cell. 2011 Nov 11; 147(4):789-802.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P75" name="crb--P75" rid="P75">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-59"><ul><li class="expanded" reference_id="R18">
<a href="/pubmed/22056041/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Ribosome profiling of mouse embryonic stem cells reveals the complexity <mark>and</mark> dynamics of mammalian proteomes.</a><span class="one_line_source">[Cell.  2011]</span><div class="alt-note">
<div class="authors">Ingolia NT, Lareau LF, Weissman JS</div>
<div class="citation">Cell. 2011 Nov 11; 147(4):789-802.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P76" name="crb--P76" rid="P76">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-60"><ul><li class="expanded" reference_id="R22">
<a href="/pubmed/26657557/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Detecting actively translated open reading frames in ribosome profiling data.</a><span class="one_line_source">[Nat Methods.  2016]</span><div class="alt-note">
<div class="authors">Calviello L, Mukherjee N, Wyler E, Zauber H, Hirsekorn A, Selbach M, Landthaler M, Obermayer B, Ohler U</div>
<div class="citation">Nat Methods. 2016 Feb; 13(2):165-70.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P78" name="crb--P78" rid="P78">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-61"><ul><li class="expanded" reference_id="R27">
<a href="/pubmed/19091943/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Highly parallel identification of essential genes in cancer cells.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2008]</span><div class="alt-note">
<div class="authors">Luo B, Cheung HW, Subramanian A, Sharifnia T, Okamoto M, Yang X, Hinkle G, Boehm JS, Beroukhim R, Weir BA, Mermel C, Barbie DA, Awad T, Zhou X, Nguyen T, Piqani B, Li C, Golub TR, Meyerson M, Hacohen N, Hahn WC, Lander ES, Sabatini DM, Root DE</div>
<div class="citation">Proc Natl Acad Sci U S A. 2008 Dec 23; 105(51):20380-5.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P80" name="crb--P80" rid="P80">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-62"><ul><li class="expanded" reference_id="R12">
<a href="/pubmed/23550210/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Integrative analysis of complex cancer genomics <mark>and</mark> clinical profiles using the cBioPortal.</a><span class="one_line_source">[Sci Signal.  2013]</span><div class="alt-note">
<div class="authors">Gao J, Aksoy BA, Dogrusoz U, Dresdner G, Gross B, Sumer SO, Sun Y, Jacobsen A, Sinha R, Larsson E, Cerami E, Sander C, Schultz N</div>
<div class="citation">Sci Signal. 2013 Apr 2; 6(269):pl1.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p7" name="crb--__p7" rid="__p7">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-63"><ul><li class="expanded" reference_id="R43">
<a href="/pubmed/25516281/" ref="reftype=pubmed&amp;article-id=5287289&amp;issue-id=286576&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Moderated estimation of fold change <mark>and</mark>  dispersion for RNA-seq data <mark>with</mark> DESeq2.</a><span class="one_line_source">[Genome Biol.  2014]</span><div class="alt-note">
<div class="authors">Love MI, Huber W, Anders S</div>
<div class="citation">Genome Biol. 2014; 15(12):550.</div>
</div>
</li></ul></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="">Support Center</a>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2017-08-29T18:41:38-04:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.45&amp;Host=ptpmc202&amp;ncbi_phid=F4FC036F9A5ED16100000000004E0040&amp;ncbi_session=F4FC79929A5EDA21_0156SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5287289%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies <mark>and</mark> Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 66px; left: 299px; width: 421px; z-index: 1001; "><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block; "><a class="ui-ncbiautocomplete-link-pref" style="display: none; ">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block; ">Turn off</a></div></div><div style="top: -100px; left: -100px; display: none; " aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div><script async="1" src="https://www.google-analytics.com/analytics.js"></script>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript>&lt;img alt="statistics" src="/stat?jsdisabled=true&amp;amp;ncbi_db=pmc&amp;amp;ncbi_pdid=article&amp;amp;ncbi_acc=&amp;amp;ncbi_domain=nihpa&amp;amp;ncbi_report=record&amp;amp;ncbi_type=fulltext&amp;amp;ncbi_objectid=&amp;amp;ncbi_pcid=/articles/PMC5287289/&amp;amp;ncbi_app=pmc" /&gt;</noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- F4FC79929A5EDA21_0156SID /projects/PMC/PMCViewer@4.45 ptpmc202 v4.1.r526566 Fri, Feb 03 2017 14:27:04 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/js/3879255/4121861/3818874/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none; " tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block; " class="ui-dialog-content ui-widget-content">  <p>We are experimenting <mark>with</mark> display styles <mark>that</mark> make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems <mark>with</mark> the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><ul class="print-log"><li></li></ul></body></html>