<!--?xml version="1.0" encoding="utf-8"?--><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4266106/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A Comparative Encyclopedia of DNA Elements in the Mouse Genome</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4266106/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature"><meta name="citation_title" content="A Comparative Encyclopedia of DNA Elements in the Mouse Genome"><meta name="citation_authors" content="Feng Yue, Yong Cheng, Alessandra Breschi, Jeff Vierstra, Weisheng Wu, Tyrone Ryba, Richard Sandstrom, Zhihai Ma, Carrie Davis, Benjamin D. Pope, Yin Shen, Dmitri D. Pervouchine, Sarah Djebali, Bob Thurman, Rajinder Kaul, Eric Rynes, Anthony Kirilusha, Georgi K. Marinov, Brian A. Williams, Diane Trout, Henry Amrhein, Katherine Fisher-Aylor, Igor Antoshechkin, Gilberto DeSalvo, Lei-Hoon See, Meagan Fastuca, Jorg Drenkow, Chris Zaleski, Alex Dobin, Pablo Prieto, Julien Lagarde, Giovanni Bussotti, Andrea Tanzer, Olgert Denas, Kanwei Li, M. A. Bender, Miaohua Zhang, Rachel Byron, Mark T. Groudine, David McCleary, Long Pham, Zhen Ye, Samantha Kuan, Lee Edsall, Yi-Chieh Wu, Matthew D. Rasmussen, Mukul S. Bansal, Cheryl A. Keller, Christapher S. Morrissey, Tejaswini Mishra, Deepti Jain, Nergiz Dogan, Robert S. Harris, Philip Cayting, Trupti Kawli, Alan P. Boyle, Ghia Euskirchen, Anshul Kundaje, Shin Lin, Yiing Lin, Camden Jansen, Venkat S. Malladi, Melissa S. Cline, Drew T. Erickson, Vanessa M Kirkup, Katrina Learned, Cricket A. Sloan, Kate R. Rosenbloom, Beatriz Lacerda de Sousa, Kathryn Beal, Miguel Pignatelli, Paul Flicek, Jin Lian, Tamer Kahveci, Dongwon Lee, W. James Kent, Miguel Ramalho Santos, Javier Herrero, Cedric Notredame, Audra Johnson, Shinny Vong, Kristen Lee, Daniel Bates, Fidencio Neri, Morgan Diegel, Theresa Canfield, Peter J. Sabo, Matthew S. Wilken, Thomas A. Reh, Erika Giste, Anthony Shafer, Tanya Kutyavin, Eric Haugen, Douglas Dunn, Alex P. Reynolds, Shane Neph, Richard Humbert, R. Scott Hansen, Marella De Bruijn, Licia Selleri, Alexander Rudensky, Steven Josefowicz, Robert Samstein, Evan E. Eichler, Stuart H. Orkin, Dana Levasseur, Thalia Papayannopoulou, Kai-Hsin Chang, Arthur Skoultchi, Srikanta Gosh, Christine Disteche, Piper Treuting, Yanli Wang, Mitchell J. Weiss, Gerd A. Blobel, Peter J. Good, Rebecca F. Lowdon, Leslie B. Adams, Xiao-Qiao Zhou, Michael J. Pazin, Elise A. Feingold, Barbara Wold, James Taylor, Manolis Kellis, Ali Mortazavi, Sherman M. Weissman, John Stamatoyannopoulos, Michael P. Snyder, Roderic Guigo, Thomas R. Gingeras, David M. Gilbert, Ross C. Hardison, Michael A. Beer, Bing Ren"><meta name="citation_date" content="20 November 2014"><meta name="citation_issue" content="7527"><meta name="citation_volume" content="515"><meta name="citation_firstpage" content="355"><meta name="citation_doi" content="10.1038/nature13992"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4266106/?report=abstract"><meta name="citation_pmid" content="25409824"><meta name="DC.Title" content="A Comparative Encyclopedia of DNA Elements in the Mouse Genome"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Feng Yue"><meta name="DC.Contributor" content="Yong Cheng"><meta name="DC.Contributor" content="Alessandra Breschi"><meta name="DC.Contributor" content="Jeff Vierstra"><meta name="DC.Contributor" content="Weisheng Wu"><meta name="DC.Contributor" content="Tyrone Ryba"><meta name="DC.Contributor" content="Richard Sandstrom"><meta name="DC.Contributor" content="Zhihai Ma"><meta name="DC.Contributor" content="Carrie Davis"><meta name="DC.Contributor" content="Benjamin D. Pope"><meta name="DC.Contributor" content="Yin Shen"><meta name="DC.Contributor" content="Dmitri D. Pervouchine"><meta name="DC.Contributor" content="Sarah Djebali"><meta name="DC.Contributor" content="Bob Thurman"><meta name="DC.Contributor" content="Rajinder Kaul"><meta name="DC.Contributor" content="Eric Rynes"><meta name="DC.Contributor" content="Anthony Kirilusha"><meta name="DC.Contributor" content="Georgi K. Marinov"><meta name="DC.Contributor" content="Brian A. Williams"><meta name="DC.Contributor" content="Diane Trout"><meta name="DC.Contributor" content="Henry Amrhein"><meta name="DC.Contributor" content="Katherine Fisher-Aylor"><meta name="DC.Contributor" content="Igor Antoshechkin"><meta name="DC.Contributor" content="Gilberto DeSalvo"><meta name="DC.Contributor" content="Lei-Hoon See"><meta name="DC.Contributor" content="Meagan Fastuca"><meta name="DC.Contributor" content="Jorg Drenkow"><meta name="DC.Contributor" content="Chris Zaleski"><meta name="DC.Contributor" content="Alex Dobin"><meta name="DC.Contributor" content="Pablo Prieto"><meta name="DC.Contributor" content="Julien Lagarde"><meta name="DC.Contributor" content="Giovanni Bussotti"><meta name="DC.Contributor" content="Andrea Tanzer"><meta name="DC.Contributor" content="Olgert Denas"><meta name="DC.Contributor" content="Kanwei Li"><meta name="DC.Contributor" content="M. A. Bender"><meta name="DC.Contributor" content="Miaohua Zhang"><meta name="DC.Contributor" content="Rachel Byron"><meta name="DC.Contributor" content="Mark T. Groudine"><meta name="DC.Contributor" content="David McCleary"><meta name="DC.Contributor" content="Long Pham"><meta name="DC.Contributor" content="Zhen Ye"><meta name="DC.Contributor" content="Samantha Kuan"><meta name="DC.Contributor" content="Lee Edsall"><meta name="DC.Contributor" content="Yi-Chieh Wu"><meta name="DC.Contributor" content="Matthew D. Rasmussen"><meta name="DC.Contributor" content="Mukul S. Bansal"><meta name="DC.Contributor" content="Cheryl A. Keller"><meta name="DC.Contributor" content="Christapher S. Morrissey"><meta name="DC.Contributor" content="Tejaswini Mishra"><meta name="DC.Contributor" content="Deepti Jain"><meta name="DC.Contributor" content="Nergiz Dogan"><meta name="DC.Contributor" content="Robert S. Harris"><meta name="DC.Contributor" content="Philip Cayting"><meta name="DC.Contributor" content="Trupti Kawli"><meta name="DC.Contributor" content="Alan P. Boyle"><meta name="DC.Contributor" content="Ghia Euskirchen"><meta name="DC.Contributor" content="Anshul Kundaje"><meta name="DC.Contributor" content="Shin Lin"><meta name="DC.Contributor" content="Yiing Lin"><meta name="DC.Contributor" content="Camden Jansen"><meta name="DC.Contributor" content="Venkat S. Malladi"><meta name="DC.Contributor" content="Melissa S. Cline"><meta name="DC.Contributor" content="Drew T. Erickson"><meta name="DC.Contributor" content="Vanessa M Kirkup"><meta name="DC.Contributor" content="Katrina Learned"><meta name="DC.Contributor" content="Cricket A. Sloan"><meta name="DC.Contributor" content="Kate R. Rosenbloom"><meta name="DC.Contributor" content="Beatriz Lacerda de Sousa"><meta name="DC.Contributor" content="Kathryn Beal"><meta name="DC.Contributor" content="Miguel Pignatelli"><meta name="DC.Contributor" content="Paul Flicek"><meta name="DC.Contributor" content="Jin Lian"><meta name="DC.Contributor" content="Tamer Kahveci"><meta name="DC.Contributor" content="Dongwon Lee"><meta name="DC.Contributor" content="W. James Kent"><meta name="DC.Contributor" content="Miguel Ramalho Santos"><meta name="DC.Contributor" content="Javier Herrero"><meta name="DC.Contributor" content="Cedric Notredame"><meta name="DC.Contributor" content="Audra Johnson"><meta name="DC.Contributor" content="Shinny Vong"><meta name="DC.Contributor" content="Kristen Lee"><meta name="DC.Contributor" content="Daniel Bates"><meta name="DC.Contributor" content="Fidencio Neri"><meta name="DC.Contributor" content="Morgan Diegel"><meta name="DC.Contributor" content="Theresa Canfield"><meta name="DC.Contributor" content="Peter J. Sabo"><meta name="DC.Contributor" content="Matthew S. Wilken"><meta name="DC.Contributor" content="Thomas A. Reh"><meta name="DC.Contributor" content="Erika Giste"><meta name="DC.Contributor" content="Anthony Shafer"><meta name="DC.Contributor" content="Tanya Kutyavin"><meta name="DC.Contributor" content="Eric Haugen"><meta name="DC.Contributor" content="Douglas Dunn"><meta name="DC.Contributor" content="Alex P. Reynolds"><meta name="DC.Contributor" content="Shane Neph"><meta name="DC.Contributor" content="Richard Humbert"><meta name="DC.Contributor" content="R. Scott Hansen"><meta name="DC.Contributor" content="Marella De Bruijn"><meta name="DC.Contributor" content="Licia Selleri"><meta name="DC.Contributor" content="Alexander Rudensky"><meta name="DC.Contributor" content="Steven Josefowicz"><meta name="DC.Contributor" content="Robert Samstein"><meta name="DC.Contributor" content="Evan E. Eichler"><meta name="DC.Contributor" content="Stuart H. Orkin"><meta name="DC.Contributor" content="Dana Levasseur"><meta name="DC.Contributor" content="Thalia Papayannopoulou"><meta name="DC.Contributor" content="Kai-Hsin Chang"><meta name="DC.Contributor" content="Arthur Skoultchi"><meta name="DC.Contributor" content="Srikanta Gosh"><meta name="DC.Contributor" content="Christine Disteche"><meta name="DC.Contributor" content="Piper Treuting"><meta name="DC.Contributor" content="Yanli Wang"><meta name="DC.Contributor" content="Mitchell J. Weiss"><meta name="DC.Contributor" content="Gerd A. Blobel"><meta name="DC.Contributor" content="Peter J. Good"><meta name="DC.Contributor" content="Rebecca F. Lowdon"><meta name="DC.Contributor" content="Leslie B. Adams"><meta name="DC.Contributor" content="Xiao-Qiao Zhou"><meta name="DC.Contributor" content="Michael J. Pazin"><meta name="DC.Contributor" content="Elise A. Feingold"><meta name="DC.Contributor" content="Barbara Wold"><meta name="DC.Contributor" content="James Taylor"><meta name="DC.Contributor" content="Manolis Kellis"><meta name="DC.Contributor" content="Ali Mortazavi"><meta name="DC.Contributor" content="Sherman M. Weissman"><meta name="DC.Contributor" content="John Stamatoyannopoulos"><meta name="DC.Contributor" content="Michael P. Snyder"><meta name="DC.Contributor" content="Roderic Guigo"><meta name="DC.Contributor" content="Thomas R. Gingeras"><meta name="DC.Contributor" content="David M. Gilbert"><meta name="DC.Contributor" content="Ross C. Hardison"><meta name="DC.Contributor" content="Michael A. Beer"><meta name="DC.Contributor" content="Bing Ren"><meta name="DC.Date" content="2014 Nov 20"><meta name="DC.Identifier" content="10.1038/nature13992"><meta name="DC.Language" content="en"><meta property="og:title" content="A Comparative Encyclopedia of DNA Elements in the Mouse Genome"><meta property="og:type" content="article"><meta property="og:description" content="As the premier model organism in biomedical research, the laboratory mouse shares the majority of protein-coding genes <mark>with</mark> humans, yet the two mammals differ in significant ways. To gain greater insights into both shared <mark>and</mark> species-specific transcriptional ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4266106/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.7/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.7/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'/pmc/utils/ctxp/'"><script type="text/javascript" src="/corehtml/pmc/ctxp/jquery.citationexporter.js">//</script><link rel="stylesheet" href="/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4266106/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4266106/pdf/nihms638072.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="8A1A484999CA71610000000000F300CC">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/css/3411343/3882866/4157116.css" media="print"><style type="text/css" media="print">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }.print-log li { list-style-image: url('https://www.ncbi.nlm.nih.gov/stat?jsevent=print&ncbi_app=pmc&ncbi_db=pmc&ncbi_pcid=%2Farticles%2FPMC4266106%2F&ncbi_pdid=article&ncbi_phid=8A1A484999CA71610000000000F300CC'); }</style><style type="text/css">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none; "></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	&lt;p class="nojs"&gt;
	&lt;strong&gt;Warning:&lt;/strong&gt;
	The NCBI web site requires JavaScript to function. 
	&lt;a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank"&gt;more...&lt;/a&gt;
	&lt;/p&gt;
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="https://www.ncbi.nlm.nih.gov/account/?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266106%2F" id="sign_in">Sign in to NCBI</a><a href="https://www.ncbi.nlm.nih.gov/account/signout/?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266106%2F" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="toolkitbookgh">ToolKitBookgh</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input type="text" name="term" id="term" title="Search PMC. Use up <mark>and</mark> down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="OFF" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="#" style="visibility: hidden; "><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4266106</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="small"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nature</a></span>. Author manuscript; available in PMC 2015 May 20.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25409824" target="pmc_ext" ref="reftype=publisher&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal"><span class="cit">Nature. 2014 Nov 20; 515(7527): 355–364. </span></a></div> <span class="doi">doi:&nbsp; <a href="//dx.doi.org/10.1038%2Fnature13992" target="pmc_ext" ref="reftype=other&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">10.1038/nature13992</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4266106</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS638072</span></div></div></div></div></div><h1 class="content-title">A Comparative Encyclopedia of DNA Elements in the Mouse Genome</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Yue%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Feng Yue</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Cheng%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Yong Cheng</a>,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Breschi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Alessandra Breschi</a>,<sup>3,</sup><sup>*</sup> <a href="/pubmed/?term=Vierstra%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Jeff Vierstra</a>,<sup>4,</sup><sup>*</sup> <a href="/pubmed/?term=Wu%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Weisheng Wu</a>,<sup>5,</sup><sup>*</sup> <a href="/pubmed/?term=Ryba%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Tyrone Ryba</a>,<sup>6,</sup><sup>*</sup> <a href="/pubmed/?term=Sandstrom%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Richard Sandstrom</a>,<sup>4,</sup><sup>*</sup> <a href="/pubmed/?term=Ma%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Zhihai Ma</a>,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Davis%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Carrie Davis</a>,<sup>7,</sup><sup>*</sup> <a href="/pubmed/?term=Pope%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Benjamin D. Pope</a>,<sup>6,</sup><sup>*</sup> <a href="/pubmed/?term=Shen%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Yin Shen</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Pervouchine%20DD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Dmitri D. Pervouchine</a>,<sup>3</sup> <a href="/pubmed/?term=Djebali%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Sarah Djebali</a>,<sup>3</sup> <a href="/pubmed/?term=Thurman%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Bob Thurman</a>,<sup>4</sup> <a href="/pubmed/?term=Kaul%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Rajinder Kaul</a>,<sup>4</sup> <a href="/pubmed/?term=Rynes%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Eric Rynes</a>,<sup>4</sup> <a href="/pubmed/?term=Kirilusha%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Anthony Kirilusha</a>,<sup>8</sup> <a href="/pubmed/?term=Marinov%20GK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Georgi K. Marinov</a>,<sup>8</sup> <a href="/pubmed/?term=Williams%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Brian A. Williams</a>,<sup>8</sup> <a href="/pubmed/?term=Trout%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Diane Trout</a>,<sup>8</sup> <a href="/pubmed/?term=Amrhein%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Henry Amrhein</a>,<sup>8</sup> <a href="/pubmed/?term=Fisher-Aylor%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Katherine Fisher-Aylor</a>,<sup>8</sup> <a href="/pubmed/?term=Antoshechkin%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Igor Antoshechkin</a>,<sup>8</sup> <a href="/pubmed/?term=DeSalvo%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Gilberto DeSalvo</a>,<sup>8</sup> <a href="/pubmed/?term=See%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Lei-Hoon See</a>,<sup>7</sup> <a href="/pubmed/?term=Fastuca%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Meagan Fastuca</a>,<sup>7</sup> <a href="/pubmed/?term=Drenkow%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Jorg Drenkow</a>,<sup>7</sup> <a href="/pubmed/?term=Zaleski%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Chris Zaleski</a>,<sup>7</sup> <a href="/pubmed/?term=Dobin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Alex Dobin</a>,<sup>7</sup> <a href="/pubmed/?term=Prieto%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Pablo Prieto</a>,<sup>3</sup> <a href="/pubmed/?term=Lagarde%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Julien Lagarde</a>,<sup>3</sup> <a href="/pubmed/?term=Bussotti%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Giovanni Bussotti</a>,<sup>3</sup> <a href="/pubmed/?term=Tanzer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Andrea Tanzer</a>,<sup>3</sup> <a href="/pubmed/?term=Denas%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Olgert Denas</a>,<sup>9</sup> <a href="/pubmed/?term=Li%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Kanwei Li</a>,<sup>9</sup> <a href="/pubmed/?term=Bender%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">M. A. Bender</a>,<sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Zhang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Miaohua Zhang</a>,<sup>12</sup> <a href="/pubmed/?term=Byron%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Rachel Byron</a>,<sup>12</sup> <a href="/pubmed/?term=Groudine%20MT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Mark T. Groudine</a>,<sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=McCleary%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">David McCleary</a>,<sup>1</sup> <a href="/pubmed/?term=Pham%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Long Pham</a>,<sup>1</sup> <a href="/pubmed/?term=Ye%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Zhen Ye</a>,<sup>1</sup> <a href="/pubmed/?term=Kuan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Samantha Kuan</a>,<sup>1</sup> <a href="/pubmed/?term=Edsall%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Lee Edsall</a>,<sup>1</sup> <a href="/pubmed/?term=Wu%20YC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Yi-Chieh Wu</a>,<sup>14</sup> <a href="/pubmed/?term=Rasmussen%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Matthew D. Rasmussen</a>,<sup>14</sup> <a href="/pubmed/?term=Bansal%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Mukul S. Bansal</a>,<sup>14</sup> <a href="/pubmed/?term=Keller%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Cheryl A. Keller</a>,<sup>5</sup> <a href="/pubmed/?term=Morrissey%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Christapher S. Morrissey</a>,<sup>5</sup> <a href="/pubmed/?term=Mishra%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Tejaswini Mishra</a>,<sup>5</sup> <a href="/pubmed/?term=Jain%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Deepti Jain</a>,<sup>5</sup> <a href="/pubmed/?term=Dogan%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Nergiz Dogan</a>,<sup>5</sup> <a href="/pubmed/?term=Harris%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Robert S. Harris</a>,<sup>5</sup> <a href="/pubmed/?term=Cayting%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Philip Cayting</a>,<sup>2</sup> <a href="/pubmed/?term=Kawli%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Trupti Kawli</a>,<sup>2</sup> <a href="/pubmed/?term=Boyle%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Alan P. Boyle</a>,<sup>2</sup> <a href="/pubmed/?term=Euskirchen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Ghia Euskirchen</a>,<sup>2</sup> <a href="/pubmed/?term=Kundaje%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Anshul Kundaje</a>,<sup>2</sup> <a href="/pubmed/?term=Lin%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Shin Lin</a>,<sup>2</sup> <a href="/pubmed/?term=Lin%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Yiing Lin</a>,<sup>2</sup> <a href="/pubmed/?term=Jansen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Camden Jansen</a>,<sup>16</sup> <a href="/pubmed/?term=Malladi%20VS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Venkat S. Malladi</a>,<sup>2</sup> <a href="/pubmed/?term=Cline%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Melissa S. Cline</a>,<sup>17</sup> <a href="/pubmed/?term=Erickson%20DT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Drew T. Erickson</a>,<sup>2</sup> <a href="/pubmed/?term=Kirkup%20VM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Vanessa M Kirkup</a>,<sup>17</sup> <a href="/pubmed/?term=Learned%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Katrina Learned</a>,<sup>17</sup> <a href="/pubmed/?term=Sloan%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Cricket A. Sloan</a>,<sup>2</sup> <a href="/pubmed/?term=Rosenbloom%20KR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Kate R. Rosenbloom</a>,<sup>17</sup> <a href="/pubmed/?term=de%20Sousa%20BL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Beatriz Lacerda de Sousa</a>,<sup>18</sup> <a href="/pubmed/?term=Beal%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Kathryn Beal</a>,<sup>19</sup> <a href="/pubmed/?term=Pignatelli%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Miguel Pignatelli</a>,<sup>19</sup> <a href="/pubmed/?term=Flicek%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Paul Flicek</a>,<sup>19</sup> <a href="/pubmed/?term=Lian%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Jin Lian</a>,<sup>20</sup> <a href="/pubmed/?term=Kahveci%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Tamer Kahveci</a>,<sup>21</sup> <a href="/pubmed/?term=Lee%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Dongwon Lee</a>,<sup>22</sup> <a href="/pubmed/?term=Kent%20WJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">W. James Kent</a>,<sup>17</sup> <a href="/pubmed/?term=Santos%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Miguel Ramalho Santos</a>,<sup>18</sup> <a href="/pubmed/?term=Herrero%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Javier Herrero</a>,<sup>19,</sup><sup>23</sup> <a href="/pubmed/?term=Notredame%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Cedric Notredame</a>,<sup>3</sup> <a href="/pubmed/?term=Johnson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Audra Johnson</a>,<sup>4</sup> <a href="/pubmed/?term=Vong%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Shinny Vong</a>,<sup>4</sup> <a href="/pubmed/?term=Lee%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Kristen Lee</a>,<sup>4</sup> <a href="/pubmed/?term=Bates%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Daniel Bates</a>,<sup>4</sup> <a href="/pubmed/?term=Neri%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Fidencio Neri</a>,<sup>4</sup> <a href="/pubmed/?term=Diegel%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Morgan Diegel</a>,<sup>4</sup> <a href="/pubmed/?term=Canfield%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Theresa Canfield</a>,<sup>4</sup> <a href="/pubmed/?term=Sabo%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Peter J. Sabo</a>,<sup>4</sup> <a href="/pubmed/?term=Wilken%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Matthew S. Wilken</a>,<sup>24</sup> <a href="/pubmed/?term=Reh%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Thomas A. Reh</a>,<sup>24</sup> <a href="/pubmed/?term=Giste%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Erika Giste</a>,<sup>4</sup> <a href="/pubmed/?term=Shafer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Anthony Shafer</a>,<sup>4</sup> <a href="/pubmed/?term=Kutyavin%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Tanya Kutyavin</a>,<sup>4</sup> <a href="/pubmed/?term=Haugen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Eric Haugen</a>,<sup>4</sup> <a href="/pubmed/?term=Dunn%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Douglas Dunn</a>,<sup>4</sup> <a href="/pubmed/?term=Reynolds%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Alex P. Reynolds</a>,<sup>4</sup> <a href="/pubmed/?term=Neph%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Shane Neph</a>,<sup>4</sup> <a href="/pubmed/?term=Humbert%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Richard Humbert</a>,<sup>4</sup> <a href="/pubmed/?term=Hansen%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">R. Scott Hansen</a>,<sup>4</sup> <a href="/pubmed/?term=De%20Bruijn%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Marella De Bruijn</a>,<sup>25</sup> <a href="/pubmed/?term=Selleri%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Licia Selleri</a>,<sup>26</sup> <a href="/pubmed/?term=Rudensky%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Alexander Rudensky</a>,<sup>27</sup> <a href="/pubmed/?term=Josefowicz%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Steven Josefowicz</a>,<sup>27</sup> <a href="/pubmed/?term=Samstein%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Robert Samstein</a>,<sup>27</sup> <a href="/pubmed/?term=Eichler%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Evan E. Eichler</a>,<sup>4</sup> <a href="/pubmed/?term=Orkin%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Stuart H. Orkin</a>,<sup>28</sup> <a href="/pubmed/?term=Levasseur%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Dana Levasseur</a>,<sup>29</sup> <a href="/pubmed/?term=Papayannopoulou%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Thalia Papayannopoulou</a>,<sup>30</sup> <a href="/pubmed/?term=Chang%20KH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Kai-Hsin Chang</a>,<sup>30</sup> <a href="/pubmed/?term=Skoultchi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Arthur Skoultchi</a>,<sup>31</sup> <a href="/pubmed/?term=Gosh%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Srikanta Gosh</a>,<sup>31</sup> <a href="/pubmed/?term=Disteche%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Christine Disteche</a>,<sup>32</sup> <a href="/pubmed/?term=Treuting%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Piper Treuting</a>,<sup>33</sup> <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Yanli Wang</a>,<sup>34</sup> <a href="/pubmed/?term=Weiss%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Mitchell J. Weiss</a>,<sup>35,</sup><sup>36</sup> <a href="/pubmed/?term=Blobel%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Gerd A. Blobel</a>,<sup>35,</sup><sup>36</sup> <a href="/pubmed/?term=Good%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Peter J. Good</a>,<sup>37</sup> <a href="/pubmed/?term=Lowdon%20RF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Rebecca F. Lowdon</a>,<sup>37</sup> <a href="/pubmed/?term=Adams%20LB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Leslie B. Adams</a>,<sup>37</sup> <a href="/pubmed/?term=Zhou%20XQ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Xiao-Qiao Zhou</a>,<sup>37</sup> <a href="/pubmed/?term=Pazin%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Michael J. Pazin</a>,<sup>37</sup> <a href="/pubmed/?term=Feingold%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Elise A. Feingold</a>,<sup>37</sup> <a href="/pubmed/?term=Wold%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Barbara Wold</a>,<sup>8</sup> <a href="/pubmed/?term=Taylor%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">James Taylor</a>,<sup>9</sup> <a href="/pubmed/?term=Kellis%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Manolis Kellis</a>,<sup>14,</sup><sup>15</sup> <a href="/pubmed/?term=Mortazavi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Ali Mortazavi</a>,<sup>16</sup> <a href="/pubmed/?term=Weissman%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Sherman M. Weissman</a>,<sup>20</sup> <a href="/pubmed/?term=Stamatoyannopoulos%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">John Stamatoyannopoulos</a>,<sup>4,</sup><sup>#</sup> <a href="/pubmed/?term=Snyder%20MP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Michael P. Snyder</a>,<sup>2,</sup><sup>#</sup> <a href="/pubmed/?term=Guigo%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Roderic Guigo</a>,<sup>3,</sup><sup>#</sup> <a href="/pubmed/?term=Gingeras%20TR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Thomas R. Gingeras</a>,<sup>7,</sup><sup>#</sup> <a href="/pubmed/?term=Gilbert%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">David M. Gilbert</a>,<sup>6,</sup><sup>#</sup> <a href="/pubmed/?term=Hardison%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Ross C. Hardison</a>,<sup>5,</sup><sup>#</sup> <a href="/pubmed/?term=Beer%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Michael A. Beer</a>,<sup>22,</sup><sup>#</sup><sup>*</sup> <mark>and</mark>  <a href="/pubmed/?term=Ren%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25409824">Bing Ren</a><sup>1,</sup><sup>#</sup>, The mouse ENCODE Consortium</div></div><div class="fm-panel small half_rhythm"><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140163195858288_ai" style="display:none"><div class="fm-affl" lang="en"><sup>1</sup>Ludwig Institute for Cancer Research <mark>and</mark> University of California, San Diego School of Medicine, 9500 Gilman Drive, La Jolla, CA 92093, USA</div><div class="fm-affl" lang="en"><sup>2</sup>Department of Genetics, Stanford University, 300 Pasteur Drive, MC-5477 Stanford, CA 94305, USA</div><div class="fm-affl" lang="en"><sup>3</sup>Bioinformatics <mark>and</mark> Genomics, Centre for Genomic Regulation (CRG) <mark>and</mark> UPF, Doctor Aiguader, 88, Barcelona 08003, Catalonia, Spain</div><div class="fm-affl" lang="en"><sup>4</sup>Department of Genome Sciences, University of Washington, Seattle, Washington 98195, USA</div><div class="fm-affl" lang="en"><sup>5</sup>Center for Comparative Genomics <mark>and</mark> Bioinformatics, Huck Institutes of the Life Sciences, The Pennsylvania State University, University Park, PA 16802, USA</div><div class="fm-affl" lang="en"><sup>6</sup>Department of Biological Science, 319 Stadium Drive, Florida State University, Tallahassee, FL 32306-4295, USA</div><div class="fm-affl" lang="en"><sup>7</sup>Functional Genomics, Cold Spring Harbor Laboratory, Bungtown Road, Cold Spring Harbor, New York 11724, USA</div><div class="fm-affl" lang="en"><sup>8</sup>Division of Biology, California Institute of Technology, Pasadena, CA 91125</div><div class="fm-affl" lang="en"><sup>9</sup>Departments of Biology <mark>and</mark> Mathematics <mark>and</mark> Computer Science, Emory University, O. Wayne Rollins Research Center, 1510 Clifton Road NE, Atlanta, Georgia 30322, USA</div><div class="fm-affl" lang="en"><sup>10</sup>Departments of Pediatrics, University of Washington, Seattle, Washington 98195, USA</div><div class="fm-affl" lang="en"><sup>11</sup>Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</div><div class="fm-affl" lang="en"><sup>12</sup>Basic Science Division, Fred Hutchinson Cancer Research Center, Seattle, WA</div><div class="fm-affl" lang="en"><sup>13</sup>Departments of Radiation Oncology, University of Washington, Seattle, Washington 98195, USA</div><div class="fm-affl" lang="en"><sup>14</sup>Computer Science <mark>and</mark> Artificial Intelligence Laboratory, Massachusetts Institute of Technology (MIT), Cambridge, MA, 02139, USA</div><div class="fm-affl" lang="en"><sup>15</sup>Broad Institute of MIT <mark>and</mark> Harvard, Cambridge</div><div class="fm-affl" lang="en"><sup>16</sup>Department of Developmental <mark>and</mark> Cell Biology, University of California, Irvine, Irvine, CA 92697, USA</div><div class="fm-affl" lang="en"><sup>17</sup>Center for Biomolecular Science <mark>and</mark> Engineering, School of Engineering, University of California Santa Cruz (UCSC), Santa Cruz, CA 95064, USA</div><div class="fm-affl" lang="en"><sup>18</sup>Departments of Ob/Gyn <mark>and</mark> Pathology, <mark>and</mark> Center for Reproductive Sciences, University of California San Francisco, San Francisco, CA, USA</div><div class="fm-affl" lang="en"><sup>19</sup>European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton Cambridge CB10 1SD, UK</div><div class="fm-affl" lang="en"><sup>20</sup>Yale University, Department of Genetics, PO Box 208005, 333 Cedar Street, New Haven, CT 06520-8005</div><div class="fm-affl" lang="en"><sup>21</sup>Computer &amp; Information Sciences &amp; Engineering, University of Florida, Gainesville, FL 32611, USA</div><div class="fm-affl" lang="en"><sup>22</sup>McKusick-Nathans Institute of Genetic Medicine <mark>and</mark> Department of Biomedical Engineering, Johns Hopkins University, 733 N. Broadway, BRB 573 Baltimore, Maryland 21205, USA</div><div class="fm-affl" lang="en"><sup>23</sup>Bill Lyons Informatics Centre, UCL Cancer Institute, University College London, London WC1E 6DD, UK</div><div class="fm-affl" lang="en"><sup>24</sup>Department of Biological Structure, University of Washington, HSB I-516, 1959 NE Pacific Street, Seattle, Washington 98195, USA</div><div class="fm-affl" lang="en"><sup>25</sup>MRC Molecular Haemotology Unit, University of Oxford, Oxford, UK</div><div class="fm-affl" lang="en"><sup>26</sup>Department of Cell <mark>and</mark> Developmental Biology, Weill Cornell Medical College, New York, NY 10065, USA</div><div class="fm-affl" lang="en"><sup>27</sup>HHMI <mark>and</mark> Ludwig Center at Memorial Sloan Kettering Cancer Center, Immunology Program, Memorial Sloan Kettering Cancer Canter, New York, NY 10065, USA</div><div class="fm-affl" lang="en"><sup>28</sup>Dana Farber Cancer Institute, Harvard Medical School, Cambridge MA 02138, USA</div><div class="fm-affl" lang="en"><sup>29</sup>University of Iowa Carver College of Medicine, Department of Internal Medicine, Iowa City, Iowa, IA 52242, USA</div><div class="fm-affl" lang="en"><sup>30</sup>Division of Hematology, Department of Medicine, University of Washington, Seattle, WA 98195, USA</div><div class="fm-affl" lang="en"><sup>31</sup>Department of Cell Biology, Albert Einstein College of Medicine, Bronx, NY 10461, USA</div><div class="fm-affl" lang="en"><sup>32</sup>Department of Pathology, University of Washington, Seattle, WA 98195, USA</div><div class="fm-affl" lang="en"><sup>33</sup>Department of Comparative Medicine, University of Washington, Seattle, WA 98195, USA</div><div class="fm-affl" lang="en"><sup>34</sup>Bioinformatics <mark>and</mark> Genomics program, The Pennsylvania State University, The Pennsylvania State University, University Park, PA 16802, USA</div><div class="fm-affl" lang="en"><sup>35</sup>Division of Hematology, The Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</div><div class="fm-affl" lang="en"><sup>36</sup>Perelman School of Medicine at the University of Pennsylvania, Philadelphia, PA 19104, USA</div><div class="fm-affl" lang="en"><sup>37</sup>NHGRI, National Institutes of Health, 5635 Fishers Lane, Bethesda, Maryland 20892-9307, USA</div><div id="FN1"><sup>#</sup>To whom correspondence should be addressed: <a href="mailto:dev@null" data-email="ude.dscu@nerib" class="oemail">ude.dscu@nerib</a> (B.R.); <a href="mailto:dev@null" data-email="ude.uhj@reebm" class="oemail">ude.uhj@reebm</a> (M.B.); <a href="mailto:dev@null" data-email="ude.usp.xb@ssor" class="oemail">ude.usp.xb@ssor</a> (R.C.H.); <a href="mailto:dev@null" data-email="ude.usf.oib@treblig" class="oemail">ude.usf.oib@treblig</a> (D.M.G.); <a href="mailto:dev@null" data-email="ude.lhsc@saregnig" class="oemail">ude.lhsc@saregnig</a> (T.R.G.); <a href="mailto:dev@null" data-email="tac.grc@ogiug.ciredor" class="oemail">tac.grc@ogiug.ciredor</a> (R.G.); <a href="mailto:dev@null" data-email="ude.drofnats@redynspm" class="oemail">ude.drofnats@redynspm</a> (M.S.); <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@matsj" class="oemail">ude.notgnihsaw.u@matsj</a> (J.S.)</div><div id="FN2"><sup>*</sup>These authors contributed equally to the work</div><div id="FN3">Current address for Feng Yue: Department of Biochemistry <mark>and</mark> Molecular Biology, School of Medicine, The Pennsylvania State University, Hershey, PA 17033, USA.<p id="__p3" class="p">Current address for Rebecca Lowdon: Washington University in St. Louis, St. Louis, MO 63108</p><p id="__p4" class="p">Current address for Leslie Adams: University of North Carolina Gillings School of Global Public Health, Chapel Hill, NC 27599, USA.</p><p id="__p5" class="p p-last">Current address for Weisheng Wu: Bioinformatics Core, Department of Computational Medicine <mark>and</mark> Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA.</p></div></div><div class="togglers"><a href="#" class="pmctoggle" rid="idm140163195858288_ai">Author information <span>►</span></a> <a href="#" class="pmctoggle" rid="idm140163195858288_cpl">Copyright <mark>and</mark> License information <span>►</span></a></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="fm-cpl-info fm-panel hide half_rhythm" id="idm140163195858288_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a>  <mark>and</mark> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-copyright half_rhythm">Users may view, print, copy, <mark>and</mark> download text <mark>and</mark> data-mine the content in such documents, for the purposes of academic research, subject always to the full Conditions of use:<a href="http://www.nature.com/authors/editorial_policies/license.html#terms" ref="reftype=extlink&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://www.nature.com/authors/editorial_policies/license.html#terms</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="small"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25409824" target="pmc_ext" ref="reftype=publisher&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal">Nature</a></div><div>See other articles in PMC <mark>that</mark> <a href="/pmc/articles/PMC4266106/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="__abstractidm140163161313152" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__abstractidm140163161313152title">Summary</h2><!--article-meta--><div><p id="P2" class="p p-first-last">As the premier model organism in biomedical research, the laboratory mouse shares the majority of protein-coding genes <mark>with</mark> humans, yet the two mammals differ in significant ways. To gain greater insights into both shared <mark>and</mark> species-specific transcriptional <mark>and</mark> cellular regulatory programs in the mouse, the Mouse ENCODE Consortium has mapped transcription, DNase I hypersensitivity, transcription factor binding, chromatin modifications, <mark>and</mark> replication domains throughout the mouse genome in diverse cell <mark>and</mark> tissue types. By comparing <mark>with</mark> the human genome, we not only confirm substantial conservation in the newly annotated potential functional sequences, but also find a large degree of divergence of other sequences involved in transcriptional regulation, chromatin state <mark>and</mark> higher order chromatin organization. Our results illuminate the wide range of evolutionary forces acting on genes <mark>and</mark> their regulatory regions, <mark>and</mark> provide a general resource for research into mammalian biology <mark>and</mark> mechanisms of human diseases.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P3" class="p p-first">Despite the widespread use of mouse models in biomedical research<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830736">1</a></sup>, the genetic <mark>and</mark> genomic differences between mice <mark>and</mark> humans remain to be fully characterized. At the sequence level, the two species have diverged substantially: approximately one half of human genomic DNA can be aligned to mouse genomic DNA, <mark>and</mark> only a small fraction (3–8%) is estimated to be under purifying selection across mammals <sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830761">2</a></sup>. At the cellular level, a systematic comparison is still lacking. Recent studies have revealed divergent DNA binding patterns for a limited number of transcription factors across multiple related mammals <sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830794">3</a>–<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830787">6</a>, <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830778">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830733">8</a></sup>, suggesting potentially wide-ranging differences in cellular functions <mark>and</mark> regulatory mechanisms<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830744">9</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830758">10</a></sup>. To fully understand how DNA sequences contribute to the unique molecular <mark>and</mark> cellular traits in mouse, it is crucial to have a comprehensive catalog of the genes <mark>and</mark> non-coding functional sequences in the mouse genome.</p><p id="P4">Advances in DNA sequencing technologies have led to the development of RNA-seq, DNase-seq, ChIP-seq, <mark>and</mark> other methods <mark>that</mark> allow rapid <mark>and</mark> genome-wide analysis of transcription, replication, chromatin accessibility, chromatin modifications, <mark>and</mark> transcription factor binding in cells <sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830782">11</a></sup>. Using these large-scale approaches, the ENCODE consortium has produced a catalog of potential functional elements in the human genome <sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830795">12</a></sup>. Notably, 62% of the human genome is transcribed in one or more cell types <sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830772">13</a></sup>, <mark>and</mark> 20% of human DNA is associated <mark>with</mark> biochemical signatures typical of functional elements including transcription factor binding, chromatin modification, <mark>and</mark> DNase hypersensitivity. The results support the notion <mark>that</mark> nucleotides outside the mammalian-conserved genomic regions could contribute to species-specific traits <sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830798">6</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830765">12</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830751">14</a></sup>.</p><p id="P5" class="p">We have applied the same high throughput approaches to over 100 mouse cell types <mark>and</mark> tissues <sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830726">15</a></sup>, producing a coordinated group of datasets for annotating the mouse genome. Integrative analyses of these datasets uncovered widespread transcriptional activities, dynamic gene expression <mark>and</mark> chromatin modification patterns, abundant <em>cis</em> regulatory elements, <mark>and</mark> remarkably stable chromosome domains in the mouse genome. The generation of these datasets also allowed an unprecedented level of comparison of genomic features of mouse <mark>and</mark> human. Described in the current manuscript <mark>and</mark> companion works, these comparisons revealed both conserved sequence features <mark>and</mark> widespread divergence in transcription <mark>and</mark> regulation. Some of the key findings are:</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
--><li><div>Although much conservation exists, the expression profiles of many mouse genes involved in distinct biological pathways show considerable divergence from their human orthologs.</div></li><li><div>A large portion of the <em>cis</em>-regulatory landscape has diverged between mouse <mark>and</mark> human, though the magnitude of regulatory DNA divergence varies widely between different classes of elements active in different tissue contexts.</div></li><li><div>Mouse <mark>and</mark> human transcription factor networks are substantially more conserved than <em>cis</em>-regulatory DNA.</div></li><li><div>Species-specific candidate regulatory sequences are significantly enriched for particular classes of repetitive DNA elements.</div></li><li><div>Chromatin state landscape in a cell lineage is relatively stable in both human <mark>and</mark> mouse.</div></li><li><div>Chromatin domains, interrogated through genome-wide analysis of DNA replication timing, are developmentally stable <mark>and</mark> evolutionarily conserved.</div></li></ul></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Results</h2><div id="S3" class="sec sec-first"><h3>Overview of data production <mark>and</mark> initial processing</h3><p id="P12" class="p p-first">To annotate potential functional sequences in the mouse genome, we used ChIP-seq, RNA-seq <mark>and</mark> DNase-seq to profile transcription factor binding, chromatin modification, transcriptome <mark>and</mark> chromatin accessibility in a collection of 123 mouse cell types <mark>and</mark> primary tissues (<a href="/pmc/articles/PMC4266106/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1–3</a>). Additionally, to interrogate large-scale chromatin organization across different cell types, we also utilized a microarray-based technique to generate replication-timing profiles in 18 mouse tissues <mark>and</mark> cell types (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>) <sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830796">16</a></sup>. Altogether, we produced over 1000 datasets. The list of the datasets <mark>and</mark> all the <a href="#SD1" rid="SD1" class=" supplementary-material">supporting material</a> for this manuscript are also available at website <a href="http://mouseencode.org" ref="reftype=extlink&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://mouseencode.org</a>. Below we briefly outline the experimental approach <mark>and</mark> initial data processing for each class of sequence features.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a class="icnblk_img figpopup" href="/pmc/articles/PMC4266106/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><img src="/pmc/articles/PMC4266106/bin/nihms638072f1.gif" class="small-thumb" alt="Figure 1" title="Figure 1" src-large="/pmc/articles/PMC4266106/bin/nihms638072f1.jpg"></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4266106/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a4--><div><span><!--caption a8--><strong>Overview of the mouse ENCODE data sets</strong></span></div></div></div><div id="S4" class="sec"><p></p><h4 class="inline">RNA transcriptome </h4><p id="P13" class="p p-first-last">To comprehensively identify the genic regions <mark>that</mark> produce transcripts in the mouse genome, we performed RNA-seq experiments in 69 different mouse tissues <mark>and</mark> cell types <mark>with</mark> two biological replicates each (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3, Supplemental Materials</a>) <mark>and</mark> uncovered 436,410 contigs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Confirming previous reports <sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830769">13</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830731">17</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830734">18</a></sup> <mark>and</mark> similar to the human genome, the mouse genome is pervasively transcribed (<a href="/pmc/articles/PMC4266106/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a>), <mark>with</mark> 46% capable of producing polyadenylated messenger RNAs (mRNA). By comparison, 39% of the human genome is devoted to making mRNAs. In both species, the vast majority (87–93%) of exonic nucleotides were detected as transcribed, confirming the sensitivity of the approach. However, a higher percentage of intronic sequences were detected as transcribed in the mouse, <mark>and</mark> this might be due to a greater sequencing depth <mark>and</mark> broader spectrum of biological samples analyzed in mouse (<a href="/pmc/articles/PMC4266106/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a>).</p></div><div id="S5" class="sec"><p></p><h4 class="inline">Candidate cis regulatory sequences </h4><p id="P14" class="p p-first">To identify potential <em>cis</em> regulatory regions in the mouse genome, we utilized three complementary approaches <mark>that</mark> involved mapping of chromatin accessibility, specific transcription factor (TF) occupancy sites <mark>and</mark> histone modification patterns. All of these approaches have previously been shown to uncover <em>cis</em> regulatory elements <mark>with</mark> high accuracy <mark>and</mark> sensitivity <sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830732">19</a>,<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830774">20</a></sup>.</p><p id="P15">By mapping DNase I hypersensitive sites (DHSs) in 55 mouse cell <mark>and</mark> tissues types <sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830760">21</a></sup>, we identified a combined total of ~1.5 million distinct DHSs at a false discovery rate (FDR) of 1% (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>) (<em>Vierstra et al</em>. in press). Genomic footprinting analysis in a subset (25) of these cell types further delineated 8.9 million distinct TF footprints. De novo derivation of a <em>cis</em>-regulatory lexicon from mouse TF footprints revealed a recognition repertoire nearly identical <mark>with</mark> <mark>that</mark> of the human, including both known <mark>and</mark> novel recognition motifs.</p><p id="P16">We used ChIP-seq to determine the binding sites for a total of 37 TFs in various subsets of 33 cell/tissue types. Of these 37 TFs, 24 were also extensively mapped in the murine <mark>and</mark> human erythroid cell models (MEL <mark>and</mark> K562) <mark>and</mark> B-lymphoid cell lines (CH12 <mark>and</mark> GM12878) (Cite: <em>Cheng et al</em>. in press). In total we defined 2,107,950 discrete ChIP-seq peaks, representing differential cell/tissue occupancy patterns of 280,396 distinct TF binding sites (<a href="#SD1" rid="SD1" class=" supplementary-material">supplemental methods</a>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>).</p><p id="P17">We also performed ChIP-seq for as many as nine histone H3 modifications (H3K4me1, H3K4me2, H3K4me3, H3K9me3, H3K27ac, H3K27me3, H3K36me3, H3K79me2, <mark>and</mark> H3K79me3) in up to 23 mouse tissues <mark>and</mark> cell types per mark. We applied a supervised machine learning technique, Random Forest based Enhancer prediction from Chromatin State (RFECS), to three histone modifications (H3K4me1, H3K4me3 <mark>and</mark> H3K27ac) <sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830749">22</a></sup>, identifying a total of 291,200 candidate enhancers <mark>and</mark> 82,853 candidate promoters in the mouse genome (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7 <mark>and</mark> 8</a>). To functionally validate the predictions, we randomly selected 76 candidate promoter elements (avg. size 1000bp) <mark>and</mark> 183 candidate enhancer elements (avg. size 1000 bp), cloned these previously unannotated sequences into reporter constructs, <mark>and</mark> performed luciferase reporter assays via transient transfection in pertinent mouse cell lines (See <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental methods</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9 <mark>and</mark> 10</a>). Overall, 66/76 (87%) candidate promoters <mark>and</mark> 129/183 (70.5%) candidate enhancers showed significant activity in these assays compared to 2/30 randomly selected negative controls (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1c</a>).</p><p id="P18" class="p p-last">Collectively, our studies assigned potential regulatory function to 12.6% of the mouse genome (<a href="/pmc/articles/PMC4266106/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1c</span></a>).</p></div><div id="S6" class="sec"><p></p><h4 class="inline">Transcription factor networks </h4><p id="P19" class="p p-first-last">We explored the TF networks <mark>and</mark> combinatorial TF binding patterns in the mouse samples in two companion papers, <mark>and</mark> compared these networks to regulatory circuitry models generated for the human genome (<em>Cheng et al.</em> <mark>and</mark> <em>Stergachis et al</em>. in press). From genomic footprints, we constructed TF-to-TF cross-regulatory network in each of 25 cell/tissue types for a total of ~500 TFs <mark>with</mark> known recognition sequences. Analyses of these networks revealed regulatory relationships between transcription factor genes <mark>that</mark> are strongly preserved in human <mark>and</mark> mouse, in spite of the extensive plasticity of the <em>cis</em>-regulatory landscape (detailed below). Whereas only 22% of TF footprints are conserved, nearly 50% of cross-regulatory connections between mouse TFs are conserved in human through the innovation of novel binding sites. Moreover, analysis of network motifs shows <mark>that</mark> larger-scale architectural features of mouse <mark>and</mark> human TF networks are strikingly similar (<em>Stergachis et al</em>. in press).</p></div><div id="S7" class="sec"><p></p><h4 class="inline">Chromatin states </h4><p id="P20" class="p p-first-last">We produced integrative maps of chromatin states in 15 mouse tissue <mark>and</mark> cell types <mark>and</mark> six human cell lines (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 11</a>) using a hidden Markov model (chromHMM) <sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830728">23</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830757">24</a></sup>, which allowed us to segment the genome in each cell type into seven distinct combination of chromatin modification marks (or chromatin states). One state is characterized by the absence of any chromatin marks, while every other state features either predominantly one modification or a combination of two to three modifications (<a href="/pmc/articles/PMC4266106/table/T1/" target="true" class="fig-table-link table figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Extended Data Table 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary materials</a>). The portion of the genome in each chromatin state varied <mark>with</mark> cell type (<a href="/pmc/articles/PMC4266106/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1d</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). Similar proportions of the genome are found in the active states in each cell type, for both mouse <mark>and</mark> human. Interestingly, the fraction of each genome <mark>that</mark> is in the H3K27me3-dominated, transcriptional repressed state is most variable, suggesting a profound role of transcriptional repression in shaping the <em>cis</em> regulatory landscape during mammalian development.</p></div><div id="S8" class="sec sec-last"><p></p><h4 class="inline">Replication domains (RDs) </h4><p id="P21" class="p p-first-last">Replication-timing, the temporal order in which megabase-sized genomic regions replicate during S-phase, is linked to the spatial organization of chromatin in the nucleus <sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830755">25</a>–<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830735">28</a></sup>, serving as a useful proxy for tracking differences in genome architecture between cell types <sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830783">29</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830768">30</a></sup>. Since different types of chromatin are assembled at different times during the S phase <sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830789">31</a></sup>, changes in replication timing during differentiation could elicit changes in chromatin structure across large domains. We obtained 36 mouse <mark>and</mark> 31 human replication-timing profiles covering 11 <mark>and</mark> 9 distinct stages of development, respectively (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>). We defined “replication boundaries” as the sites where replication profiles change slope from synchronously replicating segments (discussed later). A total of 64,535 <mark>and</mark> 50,194 boundaries identified across all mouse <mark>and</mark> human datasets, respectively, were mapped to 4,322 <mark>and</mark> 4,675 positions, <mark>with</mark> each cell type displaying replication-timing transitions at 50–80% of these positions (<a href="/pmc/articles/PMC4266106/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1e</span></a>).</p></div></div><div id="S9" class="sec"><h3>Annotation of orthologous protein coding <mark>and</mark> non-coding genes in the human <mark>and</mark> mouse genomes</h3><p id="P22" class="p p-first-last">To facilitate a systematic comparison of the transcriptome, <em>cis</em> regulatory elements <mark>and</mark> chromatin landscape between the human <mark>and</mark> mouse genomes, we built a high-quality set of human-mouse orthologs of protein coding <mark>and</mark> non-coding genes <sup><a href="#R32" rid="R32" class=" bibr popnode">32</a></sup>. The list of protein-coding orthologs, based on phylogenetic reconstruction, contains a total of 15,736 1:1 <mark>and</mark> a smaller set of 1:many <mark>and</mark> many:many ortholog pairs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 13, 14, 15</a>). We also inferred orthologous relationships among short ncRNA genes using a similar phylogenetic approach. We established 1:1 human-mouse orthologues for 151,257 internal exon pairs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 16</a>) <mark>and</mark> 204,887 intron pairs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 17</a>), <mark>and</mark> predicted 2,717 (3,446) novel human (respectively, mouse) exons (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 18</a>). Additionally, we mapped the 17,547 human long non-coding RNA (lncRNA) transcripts annotated in Gencode v10 onto the mouse genome. We found 2,327 (13.26%) human lncRNA transcripts (corresponding to 1,679, or 15.48% of the lncRNA genes) homologous to 5,067 putative mouse transcripts (corresponding to 3,887 putative genes) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 19</a>). Consistent <mark>with</mark> previous observations, only a small fraction of lncRNAs are constrained at the primary sequence level, <mark>with</mark> rapid evolutionary turnover<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830725">33</a></sup>. Other comparisons of human <mark>and</mark> mouse transcriptomes covering areas including pre-mRNA splicing, antisense, <mark>and</mark> intergenic RNA transcription are detailed in a companion paper (Pervouchine et al., submitted).</p></div><div id="S10" class="sec"><h3>Divergent <mark>and</mark> conserved gene expression patterns in human <mark>and</mark> mouse cells</h3><p id="P23" class="p p-first">Previous studies have revealed dramatic examples of species-specific gene expression patterns <mark>that</mark> underlie phenotypic changes during evolution <sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830747">34</a>–<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830746">38</a></sup>. In these cases changes in expression of a single gene between closely related species led to adaptive changes. However, it is not clear how extensive the changes in expression patterns are between more distantly related species, such as mouse <mark>and</mark> human, <mark>with</mark> some studies emphasizing similarities in transcriptome patterns of orthologous tissues <sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830771">39</a>–<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830737">41</a></sup> <mark>and</mark> others emphasizing substantial interspecies differences <sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830766">42</a></sup>. Our initial analyses revealed <mark>that</mark> gene expression patterns tended to cluster more by species rather than by tissue (<a href="/pmc/articles/PMC4266106/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>). To resolve the sets of genes contributing to different components in the clustering, we employed variance decomposition (see Methods) to estimate, for each orthologous human-mouse gene pair, the proportion of the variance in expression <mark>that</mark> is contributed by tissue <mark>and</mark> by species (<a href="/pmc/articles/PMC4266106/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). This analysis revealed the sets of genes whose expression varies more across tissues than between species, <mark>and</mark> those whose expression varies more between species than across tissues. As expected, the clustering of the RNA-seq samples is dominated either by species or tissues, depending on the gene set employed (<a href="/pmc/articles/PMC4266106/figure/F8/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 1a, 1b</span></a>). Furthermore, removal of the ~4800 genes <mark>that</mark> drive the species-specific clustering (see Shin et al., in press, <a href="#SD1" rid="SD1" class=" supplementary-material">Fig. S1D</a> therein) or normalization methods <mark>that</mark> reduce the species effects reveals tissue specific patterns of expression in the same samples (<a href="/pmc/articles/PMC4266106/figure/F8/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 1c</span></a>). Categorizing orthologous gene pairs into these groups should enable more informative translation of research results between mouse <mark>and</mark> human. In particular, for gene pairs whose variance in expression is largest between tissues (<mark>and</mark> less between species), mouse should be a particularly informative model for human biology. In contrast, interpretation of studies involving genes whose variance in expression is larger between species needs to take into account the species variation. The relative contributions of species-specific <mark>and</mark> tissue-specific factors to each gene’s expression are further explored in two companion papers (Lin et al., in press, Pervouchine et al. submitted).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a class="icnblk_img figpopup" href="/pmc/articles/PMC4266106/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><img src="/pmc/articles/PMC4266106/bin/nihms638072f2.gif" class="small-thumb" alt="Figure 2" title="Figure 2" src-large="/pmc/articles/PMC4266106/bin/nihms638072f2.jpg"></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4266106/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a4--><div><span><!--caption a8--><strong>Comparative analysis of the gene expression programs in human <mark>and</mark> mouse samples</strong></span></div></div></div><p id="P24">To further identify genes <mark>with</mark> conserved expression patterns <mark>and</mark> those <mark>that</mark> have diverged between humans <mark>and</mark> mice, we developed a novel method, referred to as Neighborhood Analysis of Conserved Co-expression (NACC), to compare the transcriptional programs of orthologous genes in a way <mark>that</mark> did not require precisely matched cell lines, tissues, or developmental stages, as long as a sufficiently diverse panel of samples is used in each species (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods</a>). Observing <mark>that</mark> the orthologs of most sets of co-expressed genes in one species remained significantly correlated across samples in the other species, we use the mean of these small correlated sets of orthologous genes as a reference expression pattern in the other species. We compute Euclidean distance to the reference pattern in the multi-dimensional tissue/gene expression space as a relative measure of conservation of expression of each gene. Specifically, for each human gene (the test gene), we defined the most similarly expressed set of genes (N=20) across all the human samples as <mark>that</mark> gene’s co-expression neighborhood. We then quantify the average distance between the transcript levels of the mouse ortholog of the test gene <mark>and</mark> the transcript levels of each mouse ortholog of the neighborhood genes across the mouse samples. We then invert the analysis, <mark>and</mark> choose a mouse test gene <mark>and</mark> define a similar gene co-expression neighborhood in the mouse samples, <mark>and</mark> calculate the average distance between the expression of orthologs of the test gene <mark>and</mark> expression of neighborhood genes across the human samples. The average change in the human to mouse <mark>and</mark> mouse to human distances, referred herein as a NACC score, is a symmetric measure of the degree of conservation of co-expression for each gene. The distribution of this quantity for each gene is shown in <a href="/pmc/articles/PMC4266106/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2c</span></a>, showing <mark>that</mark> genes in one species show a strong tendency to be co-expressed <mark>with</mark> orthologs of similarly expressed genes in the other species compared to random genes (also see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Materials</a>). We quantify the degree to which a specific biological process diverges between human <mark>and</mark> mouse as the average NACC scores of genes in each GO category by calculating a z-score using random sampling of equal size sets of genes. <a href="/pmc/articles/PMC4266106/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2d</span></a> shows <mark>that</mark> genes coding for proteins in the nuclear <mark>and</mark> intracellular organelle compartments, <mark>and</mark> involved in RNA processing, nucleic acid metabolic processes, chromatin organization <mark>and</mark> other intracellular metabolic processes tend to exhibit more similar gene expression patterns between human <mark>and</mark> mouse, <mark>and</mark> genes involved in extracellular matrix, cellular adhesion, signaling receptors, immune responses <mark>and</mark> other cell membrane related processes are more diverged (for a complete list of all GO categories <mark>and</mark> conservation analysis, see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 21</a>). As a control, when we applied the NACC analysis to two different replicates of RNA-seq datasets from the same species, no difference in biological processes can be detected (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>).</p><p id="P25" class="p p-last">Several lines of evidence indicate <mark>that</mark> NACC is a sensitive <mark>and</mark> robust method to detect conserved as well as diverged gene expression patterns from a panel of imperfectly matched tissue samples. First, when we applied NACC to a set of simulated datasets, we found <mark>that</mark> NACC is robust for the diversity <mark>and</mark> conservation of the mouse-human sample panel (in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a>). Second, we randomly sampled subsets of the full panel of samples <mark>and</mark> demonstrated <mark>that</mark> the categories of human-mouse divergence shown in <a href="/pmc/articles/PMC4266106/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig 2d</span></a> are robust to the particular sets of samples we selected (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 7</a>). Third, when we repeated NACC on a limited collection of more closely matched tissues <mark>and</mark> primary cell types (see <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental methods</a>), the biological processes detected as conserved <mark>and</mark> species specific in the larger panel of mis-matched human/mouse samples are largely recapitulated, although some pathways are detected <mark>with</mark> somewhat less significance, likely due to the smaller number of datasets used (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 8</a>). In summary, the NACC results support <mark>and</mark> extend the principal component analysis, showing <mark>that</mark> while large differences between mouse <mark>and</mark> human transcriptome profiles can be observed (revealed in PC1), genes involved in distinct cellular pathways or functional groups exhibit different degrees of conservation of expression patterns between human <mark>and</mark> mouse, <mark>with</mark> some strongly preserved <mark>and</mark> others changing dramatically.</p></div><div id="S11" class="sec"><h3>Prevalent species-specific regulatory sequences along <mark>with</mark> a core of conserved regulatory sequences</h3><p id="P26" class="p p-first">To better understand how divergence of <em>cis</em> regulatory sequences is linked to the range of conservation patterns detected in comparisons of gene expression programs between species, we examined evolutionary patterns in our predicted regulatory sequences. Previous studies have identified a wide range of evolutionary patterns <mark>and</mark> rates for <em>cis</em> regulatory regions in mammals<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830799">8</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830792">5</a></sup>, but there are still questions regarding the overall degree of similarity <mark>and</mark> divergence between the <em>cis</em> regulatory landscapes in the mouse <mark>and</mark> human. The variety of assays <mark>and</mark> breadth of tissue <mark>and</mark> cell type coverage in the mouse ENCODE data therefore provide an opportunity to address this problem more comprehensively.</p><p id="P27">We first determined sequence homology of the predicted <em>cis</em> elements in the mouse <mark>and</mark> human genomes. We established one-to-one <mark>and</mark> one-to-many mapping of human <mark>and</mark> mouse bases derived from reciprocal chained blastz alignments <sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830791">43</a></sup> <mark>and</mark> identified conserved <em>cis</em> regulatory sequences (<em>Denas et al., submitted</em>). This analysis showed <mark>that</mark> 79.3% of chromatin-based enhancer predictions, 79.6% of chromatin-based promoter predictions, 67.1% of the DHS, <mark>and</mark> 66.7% of the TF binding sites in the mouse genome have homologs in the human genome <mark>with</mark> at least 10% overlapping nucleotides, while by random chance one expects 51.2%, 52.3%, 44.3%, <mark>and</mark> 39.3% respectively (<a href="/pmc/articles/PMC4266106/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3a</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary materials</a> for details). <mark>With</mark> a more stringent cutoff <mark>that</mark> requires 50% alignment of nucleotides, we found <mark>that</mark> 56.4% of the enhancer predictions, 62.4% of promoter predictions, 61.5% of DHS, <mark>and</mark> 53.3% of the TF binding sites have homologs, compared <mark>with</mark> an expected frequency of 34%, 33.8%, 33.6% <mark>and</mark> 33.7% by random chance (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 9</a>). The candidate mouse regulatory regions <mark>with</mark> human homologs are listed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 22–25</a>. Thus, between half <mark>and</mark> two-thirds of candidate regulatory regions demonstrate a significant enrichment in sequence conservation between human <mark>and</mark> mouse. The remaining half to one-third have no identifiable orthologous sequence.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a class="icnblk_img figpopup" href="/pmc/articles/PMC4266106/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><img src="/pmc/articles/PMC4266106/bin/nihms638072f3.gif" class="small-thumb" alt="Figure 3" title="Figure 3" src-large="/pmc/articles/PMC4266106/bin/nihms638072f3.jpg"></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4266106/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a4--><div><span><!--caption a8--><strong>Comparative analysis of the cis elements predicted in the human <mark>and</mark> mouse genome</strong></span></div></div></div><p id="P28">The candidate regulatory regions in mouse <mark>with</mark> no ortholog in human could arise either because they were generated by lineage-specific events, such as transposition, or because the ortholog in the other species was lost. Species-specific <em>cis</em> regulatory sequences have been reported before <sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830745">3</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830781">14</a></sup>, but the fraction of regulatory sequences in this category remains debatable <mark>and</mark> may vary <mark>with</mark> different roles in regulation. We find <mark>that</mark> 15% (12,387 out of 82,853) of candidate mouse promoters <mark>and</mark> 16.6% (48,245 out of 291,200) of candidate enhancers (both predicted by patterns of histone modifications) have no sequence ortholog in humans (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 26, 28</a>, for details please refer to <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary method</a> section). However, the question remains as to whether these species-specific elements are truly functional elements or simply correspond to false positive predictions due to measurement errors or biological noise. Supporting the function of mouse-specific <em>cis</em> elements, 18 out of 20 randomly selected candidate mouse-specific promoters tested positive using reporter assays in mouse ES cells, where they were initially identified (<a href="/pmc/articles/PMC4266106/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3b</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 27</a>). Further, when these 18 mouse-specific promoters were tested using reporter assays in the human ES cells, all of them also exhibited significant promoter activities (<a href="/pmc/articles/PMC4266106/figure/F9/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Extended Data Fig. 2a</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 27</a>), indicating <mark>that</mark> the majority of candidate mouse-specific promoters are indeed functional sequences, which are either gained in the mouse lineage or lost in the human lineage. Similarly, a majority of the candidate mouse-specific enhancers discovered in ES cells are also likely bona fide <em>cis</em> elements, as 70.2% (26 out of 37) candidate enhancers randomly selected from this group were found to exhibit enhancer activities in reporter assays (<a href="/pmc/articles/PMC4266106/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3b</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 29</a>). Like the candidate mouse-specific promoters, 61.5% (16 out of 26) of the candidate mouse-specific enhancers also show enhancer activities in human ES cells (<a href="/pmc/articles/PMC4266106/figure/F9/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Extended Data Fig. 2a</span></a>).</p><p id="P29">We next test whether the rapidly diverged <em>cis</em> regulatory elements would correspond to the same cellular pathways shown to be less conserved by the NACC analysis of gene expression programs. Indeed, GO analysis revealed <mark>that</mark> the mouse-specific regulatory elements are significantly enriched near genes involved in immune function (<a href="/pmc/articles/PMC4266106/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3c</span></a>), in agreement <mark>with</mark> the divergent transcription patterns for these genes reported above <mark>and</mark> an earlier report based on a smaller number of primate-specific candidate regulatory regions <sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830730">44</a></sup>. This suggests <mark>that</mark> regulation of genes involved in immune function tends to be species-specific <sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830784">44</a></sup>, just as the protein-coding sequences coding for immunity, pheromones <mark>and</mark> other environmental genes are frequent targets for adaptive selection in each species <sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830788">2</a>,<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830770">45</a></sup>. The target genes for mouse specific TF binding sites (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 30</a>) are enriched in molecular functions such as histone acetyltransferase activity, high-density lipoprotein particle receptor activity, in addition to immune function (IgG binding).</p><p id="P30">We next investigated the mechanisms generating mouse-specific <em>cis</em> regulatory sequences: loss in human, gain in mouse, or both. 89% (42,947 out of 48,245) of mouse-specific enhancers <mark>and</mark> 85% (10535 out of 12387) of mouse-specific promoters overlap <mark>with</mark> at least one class of repeat elements (compared to 78% by random chance). Confirming earlier reports <sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830742">46</a>–<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830764">48</a></sup>, we found <mark>that</mark> mouse-specific candidate promoters <mark>and</mark> enhancers are significantly enriched for repetitive DNA sequences, <mark>with</mark> several classes of repeat DNA highly represented (<a href="/pmc/articles/PMC4266106/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3d</span></a> <mark>and</mark> <a href="/pmc/articles/PMC4266106/figure/F9/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Extended Data Fig. 2b</span></a>). Furthermore, mouse specific TF binding sites are highly enriched in mobile elements like SINE <mark>and</mark> LTR (<em>Sundaram et al.</em>, in press, Genome Research).</p><p id="P31">The 50% to 60% of candidate regulatory regions <mark>with</mark> sequences conserved between mouse <mark>and</mark> human are a mixture of: (i) sequences whose function has been preserved via strong constraint since these species diverged, (ii) sequences <mark>that</mark> have been co-opted (or exapted) to perform different functions in the other species, <mark>and</mark> (iii) sequences whose ortholog in the other species no longer has a discernable function, but divergence by evolutionary drift has not been sufficient to prevent sequence alignment between mouse <mark>and</mark> human. Several companion papers delve deeply into these issues (<em>Denas et al., Cheng et al., Vierstra et al</em>.). In particular, Cheng et al. show <mark>that</mark> the conservation of TF binding at orthologous positions (falling in category i) is associated <mark>with</mark> pleiotropic roles of enhancers, as evidenced by activity in multiple tissues. <em>Denas et al</em>. <mark>and</mark> <em>Vierstra et al.</em> describe the exaptation of conserved regulatory sequences for other functions.</p><p id="P32">We surveyed the conservation of function in the subset of mouse candidate <em>cis</em> elements <mark>that</mark> have sequence counterparts in the human genome. Of the 51,661 chromatin based promoter predictions <mark>that</mark> have human orthologs, 44% (22,655) of them are still predicted as a promoter in human based on the same analysis of histone modifications (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 31</a>, see <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary methods</a> for details). Of the 164,428 chromatin based enhancer predictions <mark>that</mark> have human orthologs, 40% (64,962) of them are predicted as an enhancer in human (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 32</a>). The remaining 56%–60% of candidate mouse regulatory regions <mark>with</mark> a human ortholog fall into category ii or iii (above), i.e. the orthologous sequence in human either performs a different function or does not maintain a detectable function.</p><p id="P33">One caveat of the above observation is <mark>that</mark> the tissues or cell samples used in the survey were not perfectly matched. To better examine the conservation of biochemical activities among these predicted <em>cis</em> regulatory elements <mark>with</mark> orthologs between mouse <mark>and</mark> human, we analyzed the chromatin modifications at the promoter or enhancer predictions in a broad set of 23 mouse tissue <mark>and</mark> cell types <mark>with</mark> the neighborhood co-expression association analysis (NACC) method described above. Instead of gene expression levels, we selected the histone modification H3K27ac as an indicator of promoter or enhancer activity as previously reported <sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830753">49</a></sup>. As shown in <a href="/pmc/articles/PMC4266106/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4a</span></a>, the promoter predictions (blue) show a significantly higher correlation in the level of H3K27ac in human <mark>and</mark> mouse than the random controls (red). Similarly, most chromatin based enhancer predictions in the mouse genome exhibit conserved chromatin modification patterns in the human, albeit to a lesser degree than the promoters (<a href="/pmc/articles/PMC4266106/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4b</span></a>). NACC analysis on DNase-Seq signal resulted in very similar distributions of conserved chromatin accessibility patterns at promoters (<a href="/pmc/articles/PMC4266106/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4c</span></a>) <mark>and</mark> enhancers (<a href="/pmc/articles/PMC4266106/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4d</span></a>). Thus many sequence-conserved candidate <em>cis</em> regulatory elements appeared to have conserved patterns of activities in mice <mark>and</mark> humans.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a class="icnblk_img figpopup" href="/pmc/articles/PMC4266106/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><img src="/pmc/articles/PMC4266106/bin/nihms638072f4.gif" class="small-thumb" alt="Figure 4" title="Figure 4" src-large="/pmc/articles/PMC4266106/bin/nihms638072f4.jpg"></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC4266106/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a4--><div><span><!--caption a8--><strong>Analysis of conservation in biochemical activities at the predicted mouse <em>cis</em> regulatory sequences <mark>with</mark> human orthologs</strong></span></div></div></div><p id="P34" class="p p-last">Taken together, these analyses show <mark>that</mark> the mammalian <em>cis</em>-regulatory landscapes in the human <mark>and</mark> mouse genomes are substantially different, driven primarily by gain or loss of sequence elements during evolution. These species-specific candidate regulatory elements are enriched near genes involved in stress response, immunity <mark>and</mark> certain metabolic processes, <mark>and</mark> contain elevated levels of repeated DNA elements. On the other hand, a core set of candidate regulatory sequences are conserved <mark>and</mark> display similar activity profiles in humans <mark>and</mark> mice.</p></div><div id="S12" class="sec"><h3>Chromatin state landscape reflects tissue <mark>and</mark> cell identities in both human <mark>and</mark> mouse</h3><p id="P35" class="p p-first">We examined gene-centered chromatin state maps in the mouse <mark>and</mark> human cell types (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>) (<a href="/pmc/articles/PMC4266106/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5a</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 10</a>). In all cell types, the low-expressed genes were almost uniformly in chromatin states <mark>with</mark> the repressive H3K27me3 mark or in the state unmarked by these histone modifications. In contrast, expressed genes showed the canonical pattern of H3K4me3 at the TSS surrounded by H3K4me1, followed by H3K36me3-dominated states in the remainder of the transcription unit. A similar pattern was seen for all the active genes, regardless of the level of expression; the only exception was a tendency for the H3K4me3 to spread further into the transcription unit for the most highly expressed genes. The same binary relationship between chromatin state maps <mark>and</mark> expression levels of genes was observed in mouse <mark>and</mark> human cell types (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 10</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a class="icnblk_img figpopup" href="/pmc/articles/PMC4266106/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><img src="/pmc/articles/PMC4266106/bin/nihms638072f5.gif" class="small-thumb" alt="Figure 5" title="Figure 5" src-large="/pmc/articles/PMC4266106/bin/nihms638072f5.jpg"></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC4266106/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a4--><div><span><!--caption a8--><strong>Chromatin landscape is stable within individual cell lineages</strong></span></div></div></div><p id="P36">For both mouse <mark>and</mark> human cells, the majority of the genome was in the unmarked state in each cell type, consistent <mark>with</mark> previous observations in <em>Drosophila</em>
<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830775">50</a></sup> <mark>and</mark> human cell lines <sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830740">12</a></sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). About 55% of the mouse genome was in an unmarked state in all the 15 cell types examined, while 65% is unmarked in all six human cell types. For genes <mark>that</mark> were in the unmarked state in mouse, their orthologs in human also tended to be in the unmarked state, <mark>and</mark> vice versa, leading to a positive correlation for the amount of gene neighborhoods in unmarked states (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 11</a>). Strong correlations were also observed in profiles of other chromatin marks averaged over cell lines <mark>and</mark> tissues (Pervouchine et al., submitted). The genes in the unmarked zones were depleted of transcribed nucleotides relative to the number expected based on fraction of the genome included, <mark>and</mark> the levels of the transcripts mapped there were lower than those seen in the active chromatin states (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 12</a>).</p><p id="P37" class="p p-last">Previous studies revealed limited changes of the chromatin states in lineage-restricted cells as they undergo large-scale changes in gene expression during maturation <sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830754">51</a>–<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830741">53</a></sup>. The chromatin state maps recapitulated this result, showing very similar patterns of chromatin modification in a cell line model for proliferating erythroid progenitor cells (G1E) <mark>and</mark> in maturing erythroblasts (G1E-ER4 cells treated <mark>with</mark> estradiol) across genes whose expression level changed significantly during maturation (<a href="/pmc/articles/PMC4266106/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5b</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 10b</a>). This limited change raised the possibility <mark>that</mark> the chromatin landscape, once established during lineage commitment, dictates a permissive (or restrictive) environment for the gene regulatory programs in each cell lineage <sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830739">53</a></sup>, <mark>and</mark> the chromatin states may differ between cell lineages. We tested this by examining the chromatin state maps for genes <mark>that</mark> were differentially expressed between hematopoietic cell lineages (erythroblasts versus megakaryocytes), <mark>and</mark> we found dramatic differences between the two cell types (<a href="/pmc/articles/PMC4266106/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5c</span></a> <mark>and</mark> <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 10b</a>). Genes expressed at a higher level in megakaryocytes than in erythroblasts were all in active chromatin states in megakaryocytes, but many were in inactive chromatin states in erythroblasts (<a href="/pmc/articles/PMC4266106/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5c</span></a>). In the converse situation, genes expressed at a higher level in erythroblasts than in megakaryocytes showed more inactive states in the cells in which they were repressed (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 10b</a>). These greater differences in chromatin states correlating <mark>with</mark> differential expression of genes between but not within cell lineages support the model <mark>that</mark> chromatin states are established during the process of lineage commitment. The clustering of cell types together by lineage based on chromatin state maps (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 10c</a>) also supports the model <mark>that</mark> the landscape of active <mark>and</mark> repressed chromatin is established no later than lineage commitment, <mark>and</mark> <mark>that</mark> this landscape is a defining feature of each cell type. Greater differences in chromatin states correlating <mark>with</mark> differences in gene expression were also observed when comparing average chromatin profiles in human <mark>and</mark> mouse (Pervouchine et al., submitted).</p></div><div id="S13" class="sec"><h3>Mouse chromatin states inform interpretation of human disease associated sequence variants</h3><p id="P38" class="p p-first-last">In order to investigate whether the mouse chromatin states were informative on sequence variants linked to human diseases by genome-wide association studies (GWAS), we combined the chromatin state segmentations of the fifteen mouse samples into a refined segmentation, which we used to train a self-organizing map (SOM) <sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830785">54</a></sup> on four histone modification ChIP-seq datasets (H3K4me3, H3K4me1, H3K36me3, <mark>and</mark> H3K27me3) for each mouse sample. We mapped 4,265 SNPs from the human GWAS studies uniquely onto the mouse genome <mark>and</mark> scored these Single Nucleotide Polymorphisms (SNPs) onto the trained SOM to determine whether SNP subsets were enriched in specific areas of the map. As shown in <a href="/pmc/articles/PMC4266106/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6a</span></a>, the highest enriched H3K4me1 unit in the kidney contains five GWAS hits (<em>p</em>-value&lt;3.95e-14) on different chromosomes related to blood characteristics such as platelet counts (<a href="/pmc/articles/PMC4266106/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6a</span></a>, <a href="/pmc/articles/PMC4266106/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Extended Data Table 2a</span></a>). Similarly, the second highest enriched unit in liver H3K36me3 contained six GWAS hits (<em>p</em>-value&lt;7.54e-31) related to cholesterol <mark>and</mark> alcohol dependence out of twelve in <mark>that</mark> unit (<a href="/pmc/articles/PMC4266106/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6b</span></a>, <a href="/pmc/articles/PMC4266106/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Extended Data Table 2b</span></a>). In contrast, one of the highest units in brain H3K27me3 has five GWAS hits (<em>p</em>-value&lt;4.93e-33) on different chromosomes associated <mark>with</mark> brain disorders/response to addictive substances (<a href="/pmc/articles/PMC4266106/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6c</span></a>, <a href="/pmc/articles/PMC4266106/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Extended Data Table 2c</span></a>). This unit is different from the other examples in <mark>that</mark> it is enriched for H3K27me3 signal in multiple tissues, <mark>with</mark> brain being the highest. 801 out of the 1350 units of the map showed statistical enrichment of SNPs of 0.05 after Holm-Bonferroni correction for multiple hypothesis testing, 55% of which (accounting for 1750 GWAS hits) had signal for at least one histone mark <mark>that</mark> ranked within the top 100 units on the map (<a href="/pmc/articles/PMC4266106/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6d</span></a>). The best histone marks for enriched GWAS units were primarily H3K4me1 (23%), H3K36me3 (18%), <mark>and</mark> H3K27me3 (12%), <mark>with</mark> H3K4me3 accounting for a less than 2% of the remainder. Together these results suggest <mark>that</mark> the chromatin state maps can be used to identify potential sites for functional characterization in mouse for human GWAS hits. Indeed, Cheng et al. (in press) show <mark>that</mark> conserved DNA segments bound by orthologous TFs in human <mark>and</mark> mouse are enriched for trait-associated SNPs mapped by GWAS.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F6" co-legend-rid="lgnd_F6"><a class="icnblk_img figpopup" href="/pmc/articles/PMC4266106/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><img src="/pmc/articles/PMC4266106/bin/nihms638072f6.gif" class="small-thumb" alt="Figure 6" title="Figure 6" src-large="/pmc/articles/PMC4266106/bin/nihms638072f6.jpg"></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC4266106/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6</a></div><!--caption a4--><div><span><!--caption a8--><strong>Human GWAS hits when mapped onto mouse genome are associated <mark>with</mark> specific chromatin states</strong></span></div></div></div></div><div id="S14" class="sec sec-last"><h3>Large-scale chromatin domains are developmentally stable <mark>and</mark> evolutionarily conserved</h3><p id="P39" class="p p-first">We mapped the positions of early <mark>and</mark> late replication timing boundaries in each of 36 mouse <mark>and</mark> 31 human profiles (<a href="/pmc/articles/PMC4266106/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7a</span></a>). Significantly clustered boundary positions (above the 95th percentile of re-sampled positions) were identified <mark>and</mark> peaks in boundary density were aligned between cell types using a common heuristic (<a href="/pmc/articles/PMC4266106/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 3a,b</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 13</a>). After alignment, consensus boundaries were further classified by orientation <mark>and</mark> amount of replication timing separation, resulting in a more stringent filtering of boundaries (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 14, 15</a>). Overall, we found <mark>that</mark> 88% of boundary positions (vs. 20% expected for random alignment; Fisher exact test p&lt;2e-16) aligned position <mark>and</mark> orientation between two or more cell types in both mouse <mark>and</mark> human (i.e. 12% were cell-type specific, <a href="/pmc/articles/PMC4266106/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig 7b</span></a>, <a href="/pmc/articles/PMC4266106/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 3</span></a>). Pair-wise comparisons of boundaries were consistent <mark>with</mark> developmental similarity between cell types (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 16</a>). The earliest <mark>and</mark> latest replicating boundaries were most well preserved between cell types, while those of mid-S replicating boundaries were highly variable (<a href="/pmc/articles/PMC4266106/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 3e, f</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F7" co-legend-rid="lgnd_F7"><a class="icnblk_img figpopup" href="/pmc/articles/PMC4266106/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><img src="/pmc/articles/PMC4266106/bin/nihms638072f7.gif" class="small-thumb" alt="Figure 7" title="Figure 7" src-large="/pmc/articles/PMC4266106/bin/nihms638072f7.jpg"></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC4266106/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 7</a></div><!--caption a4--><div><span><!--caption a8--><strong>RT boundaries preserved among tissues are conserved in mice <mark>and</mark> humans</strong></span></div></div></div><p id="P40">Interestingly, the greatest number of boundaries was detected in embryonic stem cells (ESCs) in both species, <mark>with</mark> significant reduction in boundary numbers during differentiation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 16</a>) consistent <mark>with</mark> consolidation of domains <mark>and</mark> by proxy large-scale chromatin organization into larger “Constant Timing Regions” CTRs during differentiation <sup><a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830759">55</a></sup>. Given <mark>that</mark> over half of the mouse <mark>and</mark> human genomes exhibit significant replication timing changes during development <sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830743">16</a>,<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830752">56</a></sup>, these observations support the model <mark>that</mark> developmental plasticity in replication timing is derived from differential regulation of replication timing within CTRs whose boundaries are preserved during development.</p><p id="P41">Although conservation of replication timing between mouse <mark>and</mark> human has been reported <sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830763">26</a>,<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830729">27</a></sup>, the conservation of replicating timing boundaries has not been examined. We converted boundary coordinates ±100kb across boundary positions between species, revealing significant overlap (<a href="/pmc/articles/PMC4266106/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7c–d</span></a>; p&lt;2.2e-16 by Fisher’s exact test relative to a randomized boundary list). The level of conservation of the positions of boundaries improved from a median of 27% for cell type specific boundaries to 70% for boundaries preserved in 9 or more cell types (<a href="/pmc/articles/PMC4266106/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7c</span></a>), demonstrating <mark>that</mark> boundaries most highly preserved during development were the most conserved across species. This was consistent <mark>with</mark> results for transcription (<a href="/pmc/articles/PMC4266106/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>), as well as the previous observation <mark>that</mark> suggest <mark>that</mark> an increased plasticity of replication timing during development is associated <mark>with</mark> increased plasticity of replication timing during evolution<sup><a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830750">57</a></sup>. Together, these findings identify evolutionarily labile vs. constrained domains of the mammalian genome at the megabase scale.</p><p id="P42" class="p p-last">Given the link between replication <mark>and</mark> chromatin assembly, we compared replication timing <mark>and</mark> levels of other chromatin properties in 200kb windows across the genome (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 17</a>). Features associated <mark>with</mark> active enhancers (H3K4me1, H3K27ac, DNase I sensitivity) were more closely correlated to replication timing than features associated <mark>with</mark> active transcription (RNA pol II, H3K4me3, H3K36me3, H3K79me2). By contrast, the correlation of replication timing to repressive features, such as H3K9me3, was poor <mark>and</mark> cell-type-specific, consistent <mark>with</mark> prior results. A more stringent comparison of differences in chromatin to differences in replication timing between cell types (<a href="/pmc/articles/PMC4266106/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 3c <mark>and</mark> 3g</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 17</a>) again revealed <mark>that</mark> marks of enhancers, including p300, H3K4me1 <mark>and</mark> H3K27ac, <mark>and</mark> DNase I sensitivity were more strongly correlated to replication timing than marks of active transcription.</p></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Conclusion</h2><p id="P43" class="p p-first">By comparing the transcriptional activities, chromatin accessibilities, transcription factor binding, chromatin landscapes <mark>and</mark> replication timing throughout the mouse genome in a wide spectrum of tissues <mark>and</mark> cell types, we have made significant progress toward a comprehensive catalog of potential functional elements in the mouse genome. The catalog described in the current study should provide a valuable reference to guide researchers to formulate new hypotheses <mark>and</mark> develop new mouse models, in the same way as the recent human ENCODE studies have impacted the research community <sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830779">12</a></sup>.</p><p id="P44">We provide multiple lines of evidence <mark>that</mark> gene expression <mark>and</mark> their underlying regulatory programs have substantially diverged between the human <mark>and</mark> mouse lineages while a subset of core regulatory programs are largely conserved. The divergence of regulatory programs between mouse <mark>and</mark> human is manifested not only in the gain or loss of <em>cis</em> regulatory sequences in the mouse genome, but also in the lack of conservation in regulatory activities across different tissues <mark>and</mark> cell types. This finding is in line <mark>with</mark> previous observations of rapidly evolving transcription factor binding in mammals, flies <mark>and</mark> yeasts, <mark>and</mark> highlights the dynamic nature of gene regulatory programs in different species <sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830738">3</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830776">4</a>,<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830773">7</a>,<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830797">58</a></sup>. Furthermore, by comprehensively delineating the potential <em>cis</em> regulatory elements we demonstrated <mark>that</mark> specific groups of genes <mark>and</mark> regulatory elements have undergone more rapid evolution than others. Of particular interest is the finding <mark>that</mark> <em>cis</em> regulatory sequences next to immune system related genes are more divergent. The finding of species-specific <em>cis</em>-elements near genes involved in immune function suggests rapid evolution of regulatory mechanisms related to the immune system. Indeed, previous studies have uncovered extensive differences in the immune systems among different mouse strains <mark>and</mark> between humans <mark>and</mark> mice <sup><a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830756">59</a></sup>, ranging from relative makeup of the innate immune <mark>and</mark> adaptive immune cells <sup><a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830790">59</a></sup>, to gene expression patterns in various immune cell types <sup><a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830777">60</a></sup>, <mark>and</mark> transcriptional responses to acute inflammatory insults <sup><a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830786">61</a>,<a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830762">62</a></sup>. At least some of these differences may be attributed to distinct regulatory mechanisms <sup><a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830727">60</a></sup>, <mark>and</mark> our finding <mark>that</mark> many predicted mouse <em>cis</em> elements near genes <mark>with</mark> immune function lack sequence conservation supports the model <mark>that</mark> evolution of <em>ci</em>s regulatory sequences contributes to differences in the immune systems between humans <mark>and</mark> mice. More generally, our findings are consistent <mark>with</mark> the view <mark>that</mark> changes in transcriptional regulatory sequences are a source for phenotypic differences in species evolution.</p><p id="P45">How can species-specific gains or loss of <em>cis</em> regulatory elements during evolution be compatible <mark>with</mark> their putative regulatory function? The finding of different rates of divergence associated <mark>with</mark> regulatory programs of distinct biological pathways suggests complex forces driving the evolution of the <em>ci</em>s regulatory landscape in mammals. We discovered <mark>that</mark> specific classes of endogenous retroviral elements are enriched at the species-specific putative <em>cis</em> regulatory elements, implicating transposition of DNA as a potential mechanism leading to divergence of gene regulatory programs during evolution. Previous studies have shown <mark>that</mark> endogenous retroviral elements can be transcribed in tissue-specific manner <sup><a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830748">63</a>,<a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830767">64</a></sup>, <mark>with</mark> a fraction of them derived from enhancers <mark>and</mark> necessary for transcription of genes involved in pluripotency <sup><a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830793">65</a>,<a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_505830780">66</a></sup>. Future studies will be necessary to determine whether retroviral elements at or near enhancers is generally involved in driving tissue-specific gene expression programs in different mammalian species.</p><p id="P46" class="p p-last">Despite the divergence of the regulatory landscape between mouse <mark>and</mark> human, the pattern of chromatin states (defined by histone modifications) <mark>and</mark> the large-scale chromatin domains are highly similar between the two species. Half of the genome is well conserved in replication timing (<mark>and</mark> by proxy, chromatin interaction compartment) <mark>with</mark> the other half highly plastic both between cell types <mark>and</mark> between species. It will be interesting to investigate the significance of these conserved <mark>and</mark> divergent classes of DNA elements at different scales, both <mark>with</mark> regard to the forces driving evolution <mark>and</mark> for implications of the use of the laboratory mouse as a model for human disease.</p></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Extended Data</h2><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F8"><h3>Extended Data Figure 1</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4266106_nihms638072f8.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms638072f8.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4266106/bin/nihms638072f8.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Clustering analysis of human <mark>and</mark> mouse tissue samples</strong><p id="P47"><strong>a</strong>, RNA-seq data from Ilumina Body Map (Adipose, Adrenal, Brain, Colon, Heart, Kidney, Liver, Lung, Ovary <mark>and</mark> Testis) were analyzed together <mark>with</mark> <mark>that</mark> from the matched mouse samples using clustering analysis. Genes <mark>with</mark> high variance across tissues were used, resulting in cell samples clustering by tissues, not by species. <strong>b,</strong> Clustering employing genes <mark>with</mark> high variance between species shows clustering by species instead of tissues. <strong>c,</strong> Principal Component Analysis (PCA) was performed for RNA-seq data for 10 human <mark>and</mark> mouse matching tissues. The expression values are normalized within each species <mark>and</mark> we observed the clustering of samples by tissue types.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F9"><h3>Extended Data Figure 2</h3><div class="figure"><img class="fig-image" alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms638072f9.jpg" title="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms638072f9.jpg" src="/pmc/articles/PMC4266106/bin/nihms638072f9.jpg"></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Comparative analysis of sequence conservation in the cis elements predicted in the human <mark>and</mark> mouse genome</strong><p id="P48"><strong>a</strong>, The predicted mouse-specific promoters <mark>and</mark> enhancers can function in human ES cells (hESCs). Percentages of predicted enhancers or promoters <mark>that</mark> test positive are shown in a bar chart. <strong>b</strong>, A bar chart shows the percentage of the predicted mouse-specific promoters containing various subclasses of LTR <mark>and</mark> SINE elements. As control, the predicted mouse cis elements <mark>with</mark> homologous sequences in the human genome or random genomic regions are included.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F10"><h3>Extended Data Figure 3</h3><div class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4266106_nihms638072f10.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file <mark>that</mark> holds a picture, illustration, etc.
Object name is nihms638072f10.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4266106/bin/nihms638072f10.jpg"></a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Replication timing boundaries preserved among tissues are conserved during evolution</strong><p id="P49"><strong>a</strong>, Heatmap of TTR overlap <mark>with</mark> positive (yellow) or negative (blue) slope. Replication timing (RT) boundaries were identified as clustered TTR endpoints (gray) above the 95th percentile (dashed line) of randomly resampled positions (black). <strong>b</strong>, Examples of constitutive boundaries (blue regions) <mark>and</mark> regulated boundaries (gray regions) highlighted. <strong>c</strong>, Spearman correlations between differences in chromatin feature enrichment <mark>and</mark> differences in RT in non-overlapping 200kb windows. <strong>d,</strong> Percentage of boundaries preserved between the indicated number of human cell types. (<strong>e &amp; f</strong>). Distribution of boundary replication timing in mouse (e) <mark>and</mark> human (f) as a function of preservation level between cell types. <strong>g</strong>, Comparison of changes in replication timing vs. various histone marks across a segment of mouse Chr6.</p></div></div><!--table ft1--><!--table-wrap @position="anchor" mode=article t4--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Extended Data Table 1</h3><!--caption a7--><div class="caption"><p id="P50">A seven-state chromHMM model learned from four histone modifications in 15 mouse cell types or lines <mark>and</mark> six human cell lines is shown. The numbers represent the emission probabilities of each histone modification (column) in each chromatin state (row). The enriched histone modifications in each state are summarized in the first column. The fraction of genome assigned in each state was calculated (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). The average <mark>and</mark> variation of these fraction values across all included cell types/tissues are listed in the last two columns.</p></div><div class="large_tbl" style="width: 435px; height: auto; overflow-x: scroll; overflow-y: visible; clear: both; "><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th valign="middle" align="center" rowspan="1" colspan="1">State</th><th valign="middle" align="center" rowspan="1" colspan="1">Feature</th><th valign="middle" align="center" rowspan="1" colspan="1">H3K27m3</th><th valign="middle" align="center" rowspan="1" colspan="1">H3K4m3</th><th valign="middle" align="center" rowspan="1" colspan="1">H3K4m1</th><th valign="middle" align="center" rowspan="1" colspan="1">H3K36m3</th><th valign="middle" align="center" rowspan="1" colspan="1">Average%</th><th valign="middle" align="center" rowspan="1" colspan="1">Variation</th></tr></thead><tbody><tr><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" style="background-color:#ED2024;color:#FFFFFF" rowspan="1" colspan="1">K4m3</td><td align="center" valign="middle" style="background-color:#ECEDEE" rowspan="1" colspan="1">0.07</td><td align="center" valign="middle" style="background-color:#050708;color:#FFFFFF" rowspan="1" colspan="1">0.92</td><td align="center" valign="middle" style="background-color:#F4F5F5" rowspan="1" colspan="1">0.05</td><td align="center" valign="middle" style="background-color:#F9FAFB" rowspan="1" colspan="1">0.03</td><td align="center" valign="middle" rowspan="1" colspan="1">0.75</td><td align="center" valign="middle" rowspan="1" colspan="1">0.07</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" style="background-color:#FEC00F" rowspan="1" colspan="1">K4m1/3</td><td align="center" valign="middle" style="background-color:#D2D2D3" rowspan="1" colspan="1">0.17</td><td align="center" valign="middle" style="background-color:#131313;color:#FFFFFF" rowspan="1" colspan="1">0.85</td><td align="center" valign="middle" style="background-color:#0A0B0C;color:#FFFFFF" rowspan="1" colspan="1">0.88</td><td align="center" valign="middle" style="background-color:#F4F5F5" rowspan="1" colspan="1">0.05</td><td align="center" valign="middle" rowspan="1" colspan="1">0.55</td><td align="center" valign="middle" rowspan="1" colspan="1">0.10</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">3</td><td align="center" valign="middle" style="background-color:#F3EC1A" rowspan="1" colspan="1">K4m1</td><td align="center" valign="middle" rowspan="1" colspan="1">0.01</td><td align="center" valign="middle" rowspan="1" colspan="1">0.01</td><td align="center" valign="middle" style="background-color:#7F7F7F;color:#FFFFFF" rowspan="1" colspan="1">0.47</td><td align="center" valign="middle" style="background-color:#FDFDFE" rowspan="1" colspan="1">0.02</td><td align="center" valign="middle" rowspan="1" colspan="1">3.35</td><td align="center" valign="middle" rowspan="1" colspan="1">0.57</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">4</td><td align="center" valign="middle" style="background-color:#94CA54" rowspan="1" colspan="1">K4m1+K36m3</td><td align="center" valign="middle" rowspan="1" colspan="1">0.01</td><td align="center" valign="middle" style="background-color:#F5F6F7" rowspan="1" colspan="1">0.05</td><td align="center" valign="middle" style="background-color:#5C5C5C;color:#FFFFFF" rowspan="1" colspan="1">0.59</td><td align="center" valign="middle" style="background-color:#3A3B3A;color:#FFFFFF" rowspan="1" colspan="1">0.71</td><td align="center" valign="middle" rowspan="1" colspan="1">0.58</td><td align="center" valign="middle" rowspan="1" colspan="1">0.23</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">5</td><td align="center" valign="middle" style="background-color:#0E8040;color:#FFFFFF" rowspan="1" colspan="1">K36m3</td><td align="center" valign="middle" rowspan="1" colspan="1">0.00</td><td align="center" valign="middle" rowspan="1" colspan="1">0.00</td><td align="center" valign="middle" rowspan="1" colspan="1">0.01</td><td align="center" valign="middle" style="background-color:#8C8C8B;color:#FFFFFF" rowspan="1" colspan="1">0.42</td><td align="center" valign="middle" rowspan="1" colspan="1">6.31</td><td align="center" valign="middle" rowspan="1" colspan="1">1.54</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">6</td><td align="center" valign="middle" style="background-color:#C0BFBF" rowspan="1" colspan="1">Unmarked</td><td align="center" valign="middle" rowspan="1" colspan="1">0.01</td><td align="center" valign="middle" rowspan="1" colspan="1">0.00</td><td align="center" valign="middle" rowspan="1" colspan="1">0.00</td><td align="center" valign="middle" rowspan="1" colspan="1">0.00</td><td align="center" valign="middle" rowspan="1" colspan="1">85.45</td><td align="center" valign="middle" rowspan="1" colspan="1">9.20</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">7</td><td align="center" valign="middle" style="background-color:#3A51A3;color:#FFFFFF" rowspan="1" colspan="1">K27m3</td><td align="center" valign="middle" style="background-color:#B0B1B1" rowspan="1" colspan="1">0.29</td><td align="center" valign="middle" rowspan="1" colspan="1">0.00</td><td align="center" valign="middle" style="background-color:#FCFCFC" rowspan="1" colspan="1">0.02</td><td align="center" valign="middle" rowspan="1" colspan="1">0.00</td><td align="center" valign="middle" rowspan="1" colspan="1">3.01</td><td align="center" valign="middle" rowspan="1" colspan="1">3.87</td></tr></tbody></table></div><div class="large-table-link align_right small" style="display: block; "><a target="object" href="/pmc/articles/PMC4266106/table/T1/?report=objectonly">View it in a separate window</a></div></div><!--table ft1--><!--table-wrap @position="anchor" mode=article t4--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Extended Data Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Self-Organizing Map of histone modifications shows enrichment of human GWAS SNPs when mapped onto mouse</strong><p id="P51"><strong>a</strong>, Kidney-specific H3K4me1 <mark>that</mark> shows enrichment of specific GWAS hits associated <mark>with</mark> urate levels <mark>and</mark> metabolites. <strong>b</strong>, Liver-specific H3K36me3 unit shows enrichment in GWAS hits related to cholesterol, alcohol dependence, <mark>and</mark> triglyceride levels. <strong>c</strong>, Brain-specific H3K27me3 signals show enrichment in GWAS SNPs associated <mark>with</mark> neurological disorders.</p></div><div class="large_tbl" style="width: auto; height: auto; overflow-x: visible; overflow-y: visible; "><table frame="void" rules="none" class="rendered small default_table"><tbody><tr><td colspan="2" align="left" valign="top" rowspan="1"><strong>a</strong></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs6900341</td><td align="left" valign="top" rowspan="1" colspan="1">Metabolite</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs1668871</td><td align="left" valign="top" rowspan="1" colspan="1">Platelet counts</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs1063856</td><td align="left" valign="top" rowspan="1" colspan="1">Coagulation factor levels</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs6798928</td><td align="left" valign="top" rowspan="1" colspan="1">Immunoglobulin A</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs2079742</td><td align="left" valign="top" rowspan="1" colspan="1">Urate levels</td></tr><tr><td colspan="2" align="left" valign="top" rowspan="1"><strong>b</strong></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs1789891</td><td align="left" valign="top" rowspan="1" colspan="1">Alcohol dependence</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs3811647</td><td align="left" valign="top" rowspan="1" colspan="1">Hepcidin levels</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs10199768</td><td align="left" valign="top" rowspan="1" colspan="1">Cardiovascular disease risk factors</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs17155315</td><td align="left" valign="top" rowspan="1" colspan="1">QT interval</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs12686004</td><td align="left" valign="top" rowspan="1" colspan="1">HDL cholesterol</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs3890182</td><td align="left" valign="top" rowspan="1" colspan="1">HDL cholesterol</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs7758229</td><td align="left" valign="top" rowspan="1" colspan="1">Colorectal cancer</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs6017342</td><td align="left" valign="top" rowspan="1" colspan="1">Ulcerative colitis</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs603446</td><td align="left" valign="top" rowspan="1" colspan="1">Triglycerides</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs2266788</td><td align="left" valign="top" rowspan="1" colspan="1">HDL Cholesterol - Triglycerides</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs6056</td><td align="left" valign="top" rowspan="1" colspan="1">Fibrinogen</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs641153</td><td align="left" valign="top" rowspan="1" colspan="1">Age-related macular degeneration (CNV)</td></tr><tr><td colspan="2" align="left" valign="top" rowspan="1"><strong>c</strong></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs6952808</td><td align="left" valign="top" rowspan="1" colspan="1">Bipolar disorder <mark>and</mark> schizophrenia</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs2424635</td><td align="left" valign="top" rowspan="1" colspan="1">Bipolar disorder <mark>and</mark> schizophrenia</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs2023454</td><td align="left" valign="top" rowspan="1" colspan="1">Functional MRI</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs17115100</td><td align="left" valign="top" rowspan="1" colspan="1">Parkinson’s disease</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1"> rs9312648</td><td align="left" valign="top" rowspan="1" colspan="1">Response to amphetamines</td></tr></tbody></table></div><div class="large-table-link align_right small" style="display: none"><a target="object" href="/pmc/articles/PMC4266106/table/T2/?report=objectonly">View it in a separate window</a></div></div></div><div id="S16" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supp material</h4><div class="sup-box half_rhythm" id="d37e2648"><a href="/pmc/articles/PMC4266106/bin/NIHMS638072-supplement-supp_material.pdf">Click here to view.</a><sup>(28M, pdf)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="__p6">This work is funded by grants R01HG003991 (B.R.), 1U54HG007004 (T.R.G.), 3RC2HG005602 (M.P.S), GM083337 <mark>and</mark> GM085354 (D.M.G), F31CA165863(BDP), RC2HG005573 <mark>and</mark> R01DK065806 (R.C.H.) from the National Institutes of Health, <mark>and</mark> BIO2011-26205 from the Spanish Plan Nacional <mark>and</mark> ERC 294653 (to R.G.), J.V. is supported by a National Science Foundation Graduate Research Fellowship under grant no. DGE-071824.. KB, MP, JH <mark>and</mark> PF acknowledge the Wellcome Trust (grant number 095908), the NHGRI (grant number U01HG004695) <mark>and</mark> the European Molecular Biology Laboratory. We thank Gary Hon for helping the analysis of high-throughput enhancer validation. L.S. is supported by R01HD043997-09. S.L. was supported by grants F32HL110473 <mark>and</mark> K99HL119617.</p></div></div><div id="__fn-groupidm140163198898800" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__fn-groupidm140163198898800title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="__p7" class="p p-first-last">Note: The role of the NHGRI Project Management Group in the preparation of this paper was limited to coordination <mark>and</mark> scientific management of the mouse ENCODE consortium.</p><p></p><div><p id="__p8" class="p p-first-last"><strong>Author Contributions:</strong> F.Y., Y.C., A.B., J.V., W.W., T.R., M.B., R.C.H., J.S., M.P.S., R. G., T.R.G., D.M.G., <mark>and</mark> B.R. led the data analysis effort, R.S, Z.M., C.D., B.P., Y.S. R.C.H., J.S., M.P.S., R. G., T.R.G., D.M.G., <mark>and</mark> B.R.led the data production. F.Y., M.A.B., L.E., C.Y., P.C., A.B., A.K., S.L., Y.L., J.V., R.S., R.E.T., E.R., E.H., A.P.R., S.N., R.H., W.W., T.M., R.S.H., C.J., A.M., B.D.P., T.R., T.K., D.L., O.D., J.T., C.Z., A.D., D.P., S.D., P.P., J.L., G.B., A.T., K.B., M.P., P.F., <mark>and</mark> J.H. analyzed data. Y.S., D.M. L.P., Z.Y., S.K. Z.M., T.K., G.E., J.L., S.M.W. R.K., M.A.B., S.L., Y.L., M.Z., R.B., M.T.G., A.J., S.V., K.L., D.B., F.N., M.D., T.C., R.S.H., P.J. S., M.S.W., T.A.R., E.G., A.S., T.K., E.H., D.D., M.D.B., L.S., A.R., S.J., R.S., E.E.E., S.H.O., D.L., T.P., K.C., A.S., C.D., P.T., W.W., C.A.K., C.S.M., T.M., D.J., N.D., B.D.P., T.R., C.D., LH.S., M.F., J.D. produced data. F.Y., C.Y., W.W, T.R., B.D.P., S.L., Y.L., C.J., C.D., A.D., A.B., D.P., S.D., C.N., A.M., J.S., M.P.S., R. G., T.R.G., D.M.G., R.C.H., M.B., B.R. wrote the manuscript. The role of the NIH Project Management Group was limited to coordination <mark>and</mark> scientific management of the mouse ENCODE consortium.</p></div></div></div><div id="__ref-listidm140163198894864" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__ref-listidm140163198894864title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Paigen K. One hundred years of mouse genetics: an intellectual history. I. The classical period (1902–1980) <span><span class="ref-journal">Genetics. </span>2003;<span class="ref-vol">163</span>:1–7.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC1462407/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12586691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Mouse Genome Sequencing C et al. Initial sequencing <mark>and</mark> comparative analysis of the mouse genome. <span><span class="ref-journal">Nature. </span>2002;<span class="ref-vol">420</span>:520–562.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12466850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Odom DT, et al.  Tissue-specific transcriptional regulation has diverged significantly between human <mark>and</mark> mouse. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:730–732.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3797512/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17529977" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Schmidt D, et al.  Five-vertebrate ChIP-seq reveals the evolutionary dynamics of transcription factor binding. <span><span class="ref-journal">Science. </span>2010;<span class="ref-vol">328</span>:1036–1040.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3008766/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/20378774" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Stefflova K, et al.  Cooperativity <mark>and</mark> rapid evolution of cobound transcription factors in closely related mammals. <span><span class="ref-journal">Cell. </span>2013;<span class="ref-vol">154</span>:530–540.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3732390/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23911320" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Wilson MD, Odom DT. Evolution of transcriptional control in mammals. <span><span class="ref-journal">Curr Opin Genet Dev. </span>2009;<span class="ref-vol">19</span>:579–585.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19913406" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Borneman AR, et al.  Divergence of transcription factor binding sites across related yeast species. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">317</span>:815–819.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17690298" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Zheng W, Gianoulis TA, Karczewski KJ, Zhao H, Snyder M. Regulatory variation within <mark>and</mark> between species. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2011;<span class="ref-vol">12</span>:327–346.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/21721942" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Wray GA. The evolutionary significance of cis-regulatory mutations. <span><span class="ref-journal">Nat Rev Genet. </span>2007;<span class="ref-vol">8</span>:206–216.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17304246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">King MC, Wilson AC. Evolution at two levels in humans <mark>and</mark> chimpanzees. <span><span class="ref-journal">Science. </span>1975;<span class="ref-vol">188</span>:107–116.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/1090005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Hawkins RD, Hon GC, Ren B. Next-generation genomics: an integrative approach. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>:476–486.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3321268/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/20531367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Consortium EP, et al.  An integrated encyclopedia of DNA elements in the human genome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:57–74.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3439153/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22955616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Djebali S, et al.  Landscape of transcription in human cells. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:101–108.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3684276/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22955620" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Consortium EP, et al.  Identification <mark>and</mark> analysis of functional elements in 1% of the human genome by the ENCODE pilot project. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:799–816.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2212820/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17571346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Stamatoyannopoulos JA, et al.  An encyclopedia of mouse DNA elements (Mouse ENCODE) <span><span class="ref-journal">Genome Biol. </span>2012;<span class="ref-vol">13</span>:418. doi: 10.1186/gb-2012-13-8-418.</span> gb-2012-13-8-418 [pii] <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3491367/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22889292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span> <span class="nowrap ref crossref">[<a href="//dx.doi.org/10.1186%2Fgb-2012-13-8-418" target="pmc_ext" ref="reftype=other&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">Cross Ref</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Hiratani I, et al.  Genome-wide dynamics of replication timing revealed by in vitro models of mouse embryogenesis. <span><span class="ref-journal">Genome Res. </span>2010;<span class="ref-vol">20</span>:155–169.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2813472/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19952138" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Jacquier A. The complex eukaryotic transcriptome: unexpected pervasive transcription <mark>and</mark> novel small RNAs. <span><span class="ref-journal">Nat Rev Genet. </span>2009;<span class="ref-vol">10</span>:833–844.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19920851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Xu Z, et al.  Bidirectional promoters generate pervasive transcription in yeast. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">457</span>:1033–1037.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2766638/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19169243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Maston GA, Landt SG, Snyder M, Green MR. Characterization of enhancer function from genome-wide analyses. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2011;<span class="ref-vol">13</span>:29–57.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22703170" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Hardison RC, Taylor J. Genomic approaches towards finding cis-regulatory modules in animals. <span><span class="ref-journal">Nat Rev Genet. </span>2012;<span class="ref-vol">13</span>:469–483.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3541939/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22705667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Thurman RE, et al.  The accessible chromatin landscape of the human genome. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">489</span>:75–82.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3721348/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22955617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Rajagopal N, et al.  RFECS: a random-forest based algorithm for enhancer identification from chromatin state. <span><span class="ref-journal">PLoS Comput Biol. </span>2013;<span class="ref-vol">9</span>:e1002968.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3597546/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23526891" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Ernst J, Kellis M. ChromHMM: automating chromatin-state discovery <mark>and</mark> characterization. <span><span class="ref-journal">Nat Methods. </span>2012;<span class="ref-vol">9</span>:215–216.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3577932/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22373907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Hoffman MM, et al.  Integrative annotation of chromatin elements from ENCODE data. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>:827–841.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3553955/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23221638" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Dixon JR, et al.  Topological domains in mammalian genomes identified by analysis of chromatin interactions. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">485</span>:376–380.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3356448/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22495300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Ryba T, et al.  Evolutionarily conserved replication timing profiles predict long-range chromatin interactions <mark>and</mark> distinguish closely related cell types. <span><span class="ref-journal">Genome Res. </span>2010;<span class="ref-vol">20</span>:761–770.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2877573/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/20430782" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Yaffe E, et al.  Comparative analysis of DNA replication timing reveals conserved large-scale chromosomal architecture. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1001011.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2895651/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/20617169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Baker A, et al.  Replication fork polarity gradients revealed by megabase-sized U-shaped replication timing domains in human cell lines. <span><span class="ref-journal">PLoS Comput Biol. </span>2012;<span class="ref-vol">8</span>:e1002443.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3320577/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22496629" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Moindrot B, et al.  3D chromatin conformation correlates <mark>with</mark> replication timing <mark>and</mark> is conserved in resting cells. <span><span class="ref-journal">Nucleic Acids Res. </span>2012;<span class="ref-vol">40</span>:9470–9481.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3479194/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22879376" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Takebayashi S-i, Dileep V, Ryba T, Dennis JH, Gilbert DM. Chromatin-interaction compartment switch at developmentally regulated chromosomal domains reveals an unusual principle of chromatin folding. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2012;<span class="ref-vol">109</span>:12574–12579.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3411983/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22807480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Lande-Diner L, Zhang J, Cedar H. Shifts in replication timing actively affect histone acetylation during nucleosome reassembly. <span><span class="ref-journal">Mol Cell. </span>2009;<span class="ref-vol">34</span>:767–774.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2717743/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19560427" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Wu Y-C, Bansa lS, Rasmussen MD, Herrero J, Kellis M. Phylogenetic identification <mark>and</mark> functional validation of orthologous genes across human, mouse, fly, worm, yeast. <span></span> in preperation.</span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Derrien T, et al.  The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, <mark>and</mark> expression. <span><span class="ref-journal">Genome Res. </span>2012;<span class="ref-vol">22</span>:1775–1789.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3431493/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22955988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">McLean CY, et al.  Human-specific loss of regulatory DNA <mark>and</mark> the evolution of human-specific traits. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">471</span>:216–219.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3071156/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/21390129" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Shubin N, Tabin C, Carroll S. Deep homology <mark>and</mark> the origins of evolutionary novelty. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">457</span>:818–823.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19212399" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Jones FC, et al.  The genomic basis of adaptive evolution in threespine sticklebacks. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">484</span>:55–61.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3322419/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22481358" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Grossman SR, et al.  Identifying recent adaptations in large-scale genomic data. <span><span class="ref-journal">Cell. </span>2013;<span class="ref-vol">152</span>:703–713.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3674781/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23415221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Fraser HB. Gene expression drives local adaptation in humans. <span><span class="ref-journal">Genome Res. </span>2013;<span class="ref-vol">23</span>:1089–1096.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3698502/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23539138" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Brawand D, et al.  The evolution of gene expression levels in mammalian organs. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">478</span>:343–348.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22012392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Merkin J, Russell C, Chen P, Burge CB. Evolutionary dynamics of gene <mark>and</mark> isoform regulation in Mammalian tissues. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">338</span>:1593–1599.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3568499/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23258891" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Barbosa-Morais NL, et al.  The evolutionary landscape of alternative splicing in vertebrate species. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">338</span>:1587–1593.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23258890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Sabeti PC, et al.  Positive natural selection in the human lineage. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">312</span>:1614–1620.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/16778047" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Schwartz S, et al.  Human-mouse alignments <mark>with</mark> BLASTZ. <span><span class="ref-journal">Genome Res. </span>2003;<span class="ref-vol">13</span>:103–107.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC430961/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12529312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">King DC, et al.  Finding cis-regulatory elements using comparative genomics: some lessons from ENCODE data. <span><span class="ref-journal">Genome Res. </span>2007;<span class="ref-vol">17</span>:775–786.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC1891337/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17567996" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Ponting CP. The functional repertoires of metazoan genomes. <span><span class="ref-journal">Nat Rev Genet. </span>2008;<span class="ref-vol">9</span>:689–698.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/18663365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Bourque G, et al.  Evolution of the mammalian transcription factor binding repertoire via transposable elements. <span><span class="ref-journal">Genome Res. </span>2008;<span class="ref-vol">18</span>:1752–1762.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2577865/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/18682548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Kunarso G, et al.  Transposable elements have rewired the core regulatory network of human embryonic stem cells. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:631–634.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/20526341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Jacques PE, Jeyakani J, Bourque G. The majority of primate-specific regulatory sequences are derived from transposable elements. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1003504.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3649963/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23675311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Calo E, Wysocka J. Modification of enhancer chromatin: what, how, <mark>and</mark> why? <span><span class="ref-journal">Mol Cell. </span>2013;<span class="ref-vol">49</span>:825–837.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3857148/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23473601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Filion GJ, et al.  Systematic protein location mapping reveals five principal chromatin types in Drosophila cells. <span><span class="ref-journal">Cell. </span>2010;<span class="ref-vol">143</span>:212–224.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3119929/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/20888037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">John S, et al.  Chromatin accessibility pre-determines glucocorticoid receptor binding patterns. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:264–268.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/21258342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Jin F, Li Y, Ren B, Natarajan R. PU.1 <mark>and</mark> C/EBP(alpha) synergistically program distinct response to NF-kappaB activation through establishing monocyte specific enhancers. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2011;<span class="ref-vol">108</span>:5290–5295.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3069155/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/21402921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Wu W, et al.  Dynamics of the epigenetic landscape during erythroid differentiation after GATA1 restoration. <span><span class="ref-journal">Genome Res. </span>2011;<span class="ref-vol">21</span>:1659–1671.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3202283/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/21795386" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Mortazavi A, et al.  Integrating <mark>and</mark> mining the chromatin landscape of cell-type specificity using self-organizing maps. <span><span class="ref-journal">Genome Res. </span>2013;<span class="ref-vol">23</span>:2136–2148.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3847782/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/24170599" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Hiratani I, et al.  Global reorganization of replication domains during embryonic stem cell differentiation. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:e245.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2561079/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/18842067" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Hansen RS, et al.  Sequencing newly replicated DNA reveals widespread plasticity in human replication timing. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2010;<span class="ref-vol">107</span>:139–144.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC2806781/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19966280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Ryba T, et al.  Replication timing: a fingerprint for cell identity <mark>and</mark> pluripotency. <span><span class="ref-journal">PLoS computational biology. </span>2011;<span class="ref-vol">7</span>:e1002225. doi: 10.1371/journal.pcbi.1002225.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3197641/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/22028635" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span> <span class="nowrap ref crossref">[<a href="//dx.doi.org/10.1371%2Fjournal.pcbi.1002225" target="pmc_ext" ref="reftype=other&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">Cross Ref</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Moses AM, et al.  Large-scale turnover of functional transcription factor binding sites in Drosophila. <span><span class="ref-journal">PLoS Comput Biol. </span>2006;<span class="ref-vol">2</span>:e130.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC1599766/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/17040121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Mestas J, Hughes CCW. Of mice <mark>and</mark> not men: differences between mouse <mark>and</mark> human immunology. <span><span class="ref-journal">J Immunol. </span>2004;<span class="ref-vol">172</span>:2731–2738.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/14978070" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Shay T, et al.  Conservation <mark>and</mark> divergence in the transcriptional programs of the human <mark>and</mark> mouse immune systems. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2013;<span class="ref-vol">110</span>:2946–2951.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3581886/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23382184" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="element-citation">Seok J, et al.  Genomic responses in mouse models poorly mimic human inflammatory diseases. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2013;<span class="ref-vol">110</span>:3507–3512.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3587220/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23401516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="element-citation">Wells CA, et al.  Genetic control of the innate immune response. <span><span class="ref-journal">BMC immunology. </span>2003;<span class="ref-vol">4</span>:5. doi: 10.1186/1471-2172-4-5.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC194878/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12826024" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span> <span class="nowrap ref crossref">[<a href="//dx.doi.org/10.1186%2F1471-2172-4-5" target="pmc_ext" ref="reftype=other&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">Cross Ref</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="element-citation">Faulkner GJ, et al.  The regulated retrotransposon transcriptome of mammalian cells. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:563–571. doi: 10.1038/ng.368.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/19377475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span> <span class="nowrap ref crossref">[<a href="//dx.doi.org/10.1038%2Fng.368" target="pmc_ext" ref="reftype=other&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">Cross Ref</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="element-citation">Xie W, et al.  Epigenomic analysis of multilineage differentiation of human embryonic stem cells. <span><span class="ref-journal">Cell. </span>2013;<span class="ref-vol">153</span>:1134–1148.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC3786220/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/23664764" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="element-citation">Lu X, et al.  The retrovirus HERVH is a long noncoding RNA required for human embryonic stem cell identity. <span><span class="ref-journal">Nature structural &amp; molecular biology. </span>2014;<span class="ref-vol">21</span>:423–425. doi: 10.1038/nsmb.2799.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/24681886" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span> <span class="nowrap ref crossref">[<a href="//dx.doi.org/10.1038%2Fnsmb.2799" target="pmc_ext" ref="reftype=other&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">Cross Ref</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">66. <span class="element-citation">Fort A, et al.  Deep transcriptome profiling of mammalian stem cells supports a regulatory role for retrotransposons in pluripotency maintenance. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:558–566. doi: 10.1038/ng.2965.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/24777452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span> <span class="nowrap ref crossref">[<a href="//dx.doi.org/10.1038%2Fng.2965" target="pmc_ext" ref="reftype=other&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">Cross Ref</a>]</span></span></div></div></div><div style="width: 200px; top: -100px; left: -100px; display: none; " aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#__abstractidm140163161313152title">Summary</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Results</a></li><li><a href="#S15title">Conclusion</a></li><li><a href="#S19title">Extended Data</a></li><li><a href="#S16title">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#__fn-groupidm140163198898800title">Footnotes</a></li><li><a href="#__ref-listidm140163198894864title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4266106/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4266106/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4266106/pdf/nihms638072.pdf">PDF (2.1M)</a></li> | <li><a href="#" data-citationid="PMC4266106" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266106%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266106%2F&amp;text=A%20Comparative%20Encyclopedia%20of%20DNA%20Elements%20in%20the%20Mouse%20Genome" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266106%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25409824" data-aiid="4266106" data-aid="4266106" data-iid="246497" data-domainid="319" data-domain="nihpa" data-accid="PMC4266106" data-md5="aa4db686f9c4b761cfc6e59ac9014a26">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-4" role="button" aria-expanded="true" style="position: absolute; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; "></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-4" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input type="hidden" name="absid" id="absid" value="4266106"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="width: 200px; top: -100px; left: -100px; display: none; " aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block; "><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block; "><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input type="submit" class="button_apply" name="addtocollection" value="1"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; "></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/22763441" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A map of the cis-regulatory sequences in the mouse genome.</a><span class="source">[Nature. 2012]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-6" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Shen Y, Yue F, McCleary DF, Ye Z, Edsall L, Kuan S, Wagner U, Dixon J, Lee L, Lobanenkov VV, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature. 2012 Aug 2; 488(7409):116-20. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/22955616" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">An integrated encyclopedia of DNA elements in the human genome.</a><span class="source">[Nature. 2012]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">ENCODE Project Consortium.. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature. 2012 Sep 6; 489(7414):57-74. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/22955614" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genomics: ENCODE explained.</a><span class="source">[Nature. 2012]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Ecker JR, Bickmore WA, Barroso I, Pritchard JK, Gilad Y, Segal E. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature. 2012 Sep 6; 489(7414):52-5. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/23722115" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A brief review on the Human Encyclopedia of DNA Elements (ENCODE) project.</a><span class="source">[Genomics Proteomics Bioinforma...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Qu H, Fang X. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Genomics Proteomics Bioinformatics. 2013 Jun; 11(3):135-41. Epub 2013 May 28.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/26133010" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Lessons from modENCODE.</a><span class="source">[Annu Rev Genomics Hum Genet. 2...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Brown JB, Celniker SE. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Annu Rev Genomics Hum Genet. 2015; 16:31-53. Epub 2015 Jun 26.</em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=25409824&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=25409824&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-11" role="button" aria-expanded="true" style="position: absolute; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; "></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-11" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC5551495/" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Epo reprograms the epigenome of erythroid cells</a><span class="source">[Experimental hematology. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-12" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Perreault AA, Benton ML, Koury MJ, Brandt SJ, Venters BJ. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Experimental hematology. 2017 Apr 12; 51: 47-62</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC5542423/" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Annotation Regression for Genome-Wide Association Studies <mark>with</mark> an Application to Psychiatric Genomic Consortium Data</a><span class="source">[Statistics in biosciences. 2016]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Shin S, Keleş S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Statistics in biosciences. 2016 Aug 12; 9(1): 50-72</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC5534336/" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Similar Ratios of Introns to Intergenic Sequence across Animal Genomes</a><span class="source">[Genome Biology <mark>and</mark> Evolution. ...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Francis WR, Wörheide G. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Genome Biology <mark>and</mark> Evolution. 2017 Jun 13; 9(6): 1582-1598</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC5526747/" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Predicting Enhancer Activity <mark>and</mark> Variant Impact using gkm-SVM</a><span class="source">[Human mutation. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Beer MA. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Human mutation. 2017 Mar 15; 38(9): 1251-1258</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC5518781/" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Induction of specific neuron types by overexpression of single transcription factors</a><span class="source"></span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Teratani-Ota Y, Yamamizu K, Piao Y, Sharova L, Amano M, Yu H, Schlessinger D, Ko MS, Sharov AA. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">In vitro cellular &amp; developmental biology. Animal. 2016 Jun 1; 52(9): 961-973</em>
</div>
</li>
</ul>
<a class="seemore" href="citedby/" ref="reftype=relart&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-17" role="button" aria-expanded="true" style="position: absolute; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; "></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-17" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=bioproject&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_bioproject&amp;IdsFromResult=4266106" ref="reftype=BioProject&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">BioProject</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-18" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">BioProject<div class="brieflinkpopdesc">BioProject links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=pccompound&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_pccompound&amp;IdsFromResult=4266106" ref="reftype=Compound&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|PubChem%20Compound&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Compound</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">Compound<div class="brieflinkpopdesc">PubChem Compound links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gds&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gds&amp;IdsFromResult=4266106" ref="reftype=GEO%20DataSets&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|GEO%20DataSets&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">GEO DataSets</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">GEO DataSets<div class="brieflinkpopdesc">GEO DataSet links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/25409824/" ref="reftype=PubMed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=4266106" ref="reftype=Taxonomy&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-22" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none; ">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-23" role="button" aria-expanded="true" style="position: absolute; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; "></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-23" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity"><li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=599ca85fed354abae407a65d">A Comparative Encyclopedia of DNA Elements in the Mouse Genome</a><div class="ralinkpop offscreen_noflow">A Comparative Encyclopedia of DNA Elements in the Mouse Genome<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2014 Nov 20; 515(7527)355</div>
</div>
<div class="tertiary"></div>
</li></ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--P3" name="crb--P3" rid="P3" style="position: absolute; visibility: visible; width: 241px; top: 1619px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-24"><ul>
<li class="two_line" reference_id="R1">
<a href="/pubmed/12586691/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">One hundred years of mouse genetics: an intellectual history. I. The classical period (1902-1980).</a><span class="one_line_source">[Genetics.  2003]</span><div class="alt-note">
<div class="authors">Paigen K</div>
<div class="citation">Genetics. 2003 Jan; 163(1):1-7.</div>
</div>
</li>
<li class="two_line" reference_id="R2">
<a href="/pubmed/12466850/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Initial sequencing <mark>and</mark> comparative analysis of the mouse genome.</a><span class="one_line_source">[Nature.  2002]</span><div class="alt-note">
<div class="authors">Mouse Genome Sequencing Consortium., Waterston RH, Lindblad-Toh K, Birney E, Rogers J, Abril JF, Agarwal P, Agarwala R, Ainscough R, Alexandersson M, An P, Antonarakis SE, Attwood J, Baertsch R, Bailey J, Barlow K, Beck S, Berry E, Birren B, Bloom T, Bork P, Botcherby M, Bray N, Brent MR, Brown DG, Brown SD, Bult C, Burton J, Butler J, Campbell RD, Carninci P, Cawley S, Chiaromonte F, Chinwalla AT, Church DM, Clamp M, Clee C, Collins FS, Cook LL, Copley RR, Coulson A, Couronne O, Cuff J, Curwen V, Cutts T, Daly M, David R, Davies J, Delehaunty KD, Deri J, Dermitzakis ET, Dewey C, Dickens NJ, Diekhans M, Dodge S, Dubchak I, Dunn DM, Eddy SR, Elnitski L, Emes RD, Eswara P, Eyras E, Felsenfeld A, Fewell GA, Flicek P, Foley K, Frankel WN, Fulton LA, Fulton RS, Furey TS, Gage D, Gibbs RA, Glusman G, Gnerre S, Goldman N, Goodstadt L, Grafham D, Graves TA, Green ED, Gregory S, Guigó R, Guyer M, Hardison RC, Haussler D, Hayashizaki Y, Hillier LW, Hinrichs A, Hlavina W, Holzer T, Hsu F, Hua A, Hubbard T, Hunt A, Jackson I, Jaffe DB, Johnson LS, Jones M, Jones TA, Joy A, Kamal M, Karlsson EK, Karolchik D, Kasprzyk A, Kawai J, Keibler E, Kells C, Kent WJ, Kirby A, Kolbe DL, Korf I, Kucherlapati RS, Kulbokas EJ, Kulp D, Landers T, Leger JP, Leonard S, Letunic I, Levine R, Li J, Li M, Lloyd C, Lucas S, Ma B, Maglott DR, Mardis ER, Matthews L, Mauceli E, Mayer JH, McCarthy M, McCombie WR, McLaren S, McLay K, McPherson JD, Meldrim J, Meredith B, Mesirov JP, Miller W, Miner TL, Mongin E, Montgomery KT, Morgan M, Mott R, Mullikin JC, Muzny DM, Nash WE, Nelson JO, Nhan MN, Nicol R, Ning Z, Nusbaum C, O'Connor MJ, Okazaki Y, Oliver K, Overton-Larty E, Pachter L, Parra G, Pepin KH, Peterson J, Pevzner P, Plumb R, Pohl CS, Poliakov A, Ponce TC, Ponting CP, Potter S, Quail M, Reymond A, Roe BA, Roskin KM, Rubin EM, Rust AG, Santos R, Sapojnikov V, Schultz B, Schultz J, Schwartz MS, Schwartz S, Scott C, Seaman S, Searle S, Sharpe T, Sheridan A, Shownkeen R, Sims S, Singer JB, Slater G, Smit A, Smith DR, Spencer B, Stabenau A, Stange-Thomann N, Sugnet C, Suyama M, Tesler G, Thompson J, Torrents D, Trevaskis E, Tromp J, Ucla C, Ureta-Vidal A, Vinson JP, Von Niederhausern AC, Wade CM, Wall M, Weber RJ, Weiss RB, Wendl MC, West AP, Wetterstrand K, Wheeler R, Whelan S, Wierzbowski J, Willey D, Williams S, Wilson RK, Winter E, Worley KC, Wyman D, Yang S, Yang SP, Zdobnov EM, Zody MC, Lander ES</div>
<div class="citation">Nature. 2002 Dec 5; 420(6915):520-62.</div>
</div>
</li>
<li class="two_line" reference_id="R3">
<a href="/pubmed/17529977/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Tissue-specific transcriptional regulation has diverged significantly between human <mark>and</mark> mouse.</a><span class="one_line_source">[Nat Genet.  2007]</span><div class="alt-note">
<div class="authors">Odom DT, Dowell RD, Jacobsen ES, Gordon W, Danford TW, MacIsaac KD, Rolfe PA, Conboy CM, Gifford DK, Fraenkel E</div>
<div class="citation">Nat Genet. 2007 Jun; 39(6):730-2.</div>
</div>
</li>
<li class="two_line" reference_id="R6">
<a href="/pubmed/19913406/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Evolution of transcriptional control in mammals.</a><span class="one_line_source">[Curr Opin Genet Dev.  2009]</span><div class="alt-note">
<div class="authors">Wilson MD, Odom DT</div>
<div class="citation">Curr Opin Genet Dev. 2009 Dec; 19(6):579-85.</div>
</div>
</li>
<li class="two_line" reference_id="R7">
<a href="/pubmed/17690298/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Divergence of transcription factor binding sites across related yeast species.</a><span class="one_line_source">[Science.  2007]</span><div class="alt-note">
<div class="authors">Borneman AR, Gianoulis TA, Zhang ZD, Yu H, Rozowsky J, Seringhaus MR, Wang LY, Gerstein M, Snyder M</div>
<div class="citation">Science. 2007 Aug 10; 317(5839):815-9.</div>
</div>
</li>
<li class="two_line" reference_id="R8">
<a href="/pubmed/21721942/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Regulatory variation within <mark>and</mark> between species.</a><span class="one_line_source">[Annu Rev Genomics Hum Genet.  2011]</span><div class="alt-note">
<div class="authors">Zheng W, Gianoulis TA, Karczewski KJ, Zhao H, Snyder M</div>
<div class="citation">Annu Rev Genomics Hum Genet. 2011; 12():327-46.</div>
</div>
</li>
<li class="two_line" reference_id="R9">
<a href="/pubmed/17304246/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> The evolutionary significance of cis-regulatory mutations.</a><span class="one_line_source">[Nat Rev Genet.  2007]</span><div class="alt-note">
<div class="authors">Wray GA</div>
<div class="citation">Nat Rev Genet. 2007 Mar; 8(3):206-16.</div>
</div>
</li>
<li class="two_line" reference_id="R10">
<a href="/pubmed/1090005/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Evolution at two levels in humans <mark>and</mark> chimpanzees.</a><span class="one_line_source">[Science.  1975]</span><div class="alt-note">
<div class="authors">King MC, Wilson AC</div>
<div class="citation">Science. 1975 Apr 11; 188(4184):107-16.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P4" name="crb--P4" rid="P4" style="position: absolute; visibility: visible; width: 241px; top: 1964px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-25"><ul>
<li class="two_line" reference_id="R11">
<a href="/pubmed/20531367/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Next-generation genomics: an integrative approach.</a><span class="one_line_source">[Nat Rev Genet.  2010]</span><div class="alt-note">
<div class="authors">Hawkins RD, Hon GC, Ren B</div>
<div class="citation">Nat Rev Genet. 2010 Jul; 11(7):476-86.</div>
</div>
</li>
<li class="two_line" reference_id="R12">
<a href="/pubmed/22955616/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An integrated encyclopedia of DNA elements in the human genome.</a><span class="one_line_source">[Nature.  2012]</span><div class="alt-note">
<div class="authors">ENCODE Project Consortium.</div>
<div class="citation">Nature. 2012 Sep 6; 489(7414):57-74.</div>
</div>
</li>
<li class="two_line" reference_id="R13">
<a href="/pubmed/22955620/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Landscape of transcription in human cells.</a><span class="one_line_source">[Nature.  2012]</span><div class="alt-note">
<div class="authors">Djebali S, Davis CA, Merkel A, Dobin A, Lassmann T, Mortazavi A, Tanzer A, Lagarde J, Lin W, Schlesinger F, Xue C, Marinov GK, Khatun J, Williams BA, Zaleski C, Rozowsky J, Röder M, Kokocinski F, Abdelhamid RF, Alioto T, Antoshechkin I, Baer MT, Bar NS, Batut P, Bell K, Bell I, Chakrabortty S, Chen X, Chrast J, Curado J, Derrien T, Drenkow J, Dumais E, Dumais J, Duttagupta R, Falconnet E, Fastuca M, Fejes-Toth K, Ferreira P, Foissac S, Fullwood MJ, Gao H, Gonzalez D, Gordon A, Gunawardena H, Howald C, Jha S, Johnson R, Kapranov P, King B, Kingswood C, Luo OJ, Park E, Persaud K, Preall JB, Ribeca P, Risk B, Robyr D, Sammeth M, Schaffer L, See LH, Shahab A, Skancke J, Suzuki AM, Takahashi H, Tilgner H, Trout D, Walters N, Wang H, Wrobel J, Yu Y, Ruan X, Hayashizaki Y, Harrow J, Gerstein M, Hubbard T, Reymond A, Antonarakis SE, Hannon G, Giddings MC, Ruan Y, Wold B, Carninci P, Guigó R, Gingeras TR</div>
<div class="citation">Nature. 2012 Sep 6; 489(7414):101-8.</div>
</div>
</li>
<li class="two_line" reference_id="R6">
<a href="/pubmed/19913406/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Evolution of transcriptional control in mammals.</a><span class="one_line_source">[Curr Opin Genet Dev.  2009]</span><div class="alt-note">
<div class="authors">Wilson MD, Odom DT</div>
<div class="citation">Curr Opin Genet Dev. 2009 Dec; 19(6):579-85.</div>
</div>
</li>
<li class="two_line" reference_id="R14">
<a href="/pubmed/17571346/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Identification <mark>and</mark> analysis of functional elements in 1% of the human genome by the ENCODE pilot project.</a><span class="one_line_source">[Nature.  2007]</span><div class="alt-note">
<div class="authors">ENCODE Project Consortium., Birney E, Stamatoyannopoulos JA, Dutta A, Guigó R, Gingeras TR, Margulies EH, Weng Z, Snyder M, Dermitzakis ET, Thurman RE, Kuehn MS, Taylor CM, Neph S, Koch CM, Asthana S, Malhotra A, Adzhubei I, Greenbaum JA, Andrews RM, Flicek P, Boyle PJ, Cao H, Carter NP, Clelland GK, Davis S, Day N, Dhami P, Dillon SC, Dorschner MO, Fiegler H, Giresi PG, Goldy J, Hawrylycz M, Haydock A, Humbert R, James KD, Johnson BE, Johnson EM, Frum TT, Rosenzweig ER, Karnani N, Lee K, Lefebvre GC, Navas PA, Neri F, Parker SC, Sabo PJ, Sandstrom R, Shafer A, Vetrie D, Weaver M, Wilcox S, Yu M, Collins FS, Dekker J, Lieb JD, Tullius TD, Crawford GE, Sunyaev S, Noble WS, Dunham I, Denoeud F, Reymond A, Kapranov P, Rozowsky J, Zheng D, Castelo R, Frankish A, Harrow J, Ghosh S, Sandelin A, Hofacker IL, Baertsch R, Keefe D, Dike S, Cheng J, Hirsch HA, Sekinger EA, Lagarde J, Abril JF, Shahab A, Flamm C, Fried C, Hackermüller J, Hertel J, Lindemeyer M, Missal K, Tanzer A, Washietl S, Korbel J, Emanuelsson O, Pedersen JS, Holroyd N, Taylor R, Swarbreck D, Matthews N, Dickson MC, Thomas DJ, Weirauch MT, Gilbert J, Drenkow J, Bell I, Zhao X, Srinivasan KG, Sung WK, Ooi HS, Chiu KP, Foissac S, Alioto T, Brent M, Pachter L, Tress ML, Valencia A, Choo SW, Choo CY, Ucla C, Manzano C, Wyss C, Cheung E, Clark TG, Brown JB, Ganesh M, Patel S, Tammana H, Chrast J, Henrichsen CN, Kai C, Kawai J, Nagalakshmi U, Wu J, Lian Z, Lian J, Newburger P, Zhang X, Bickel P, Mattick JS, Carninci P, Hayashizaki Y, Weissman S, Hubbard T, Myers RM, Rogers J, Stadler PF, Lowe TM, Wei CL, Ruan Y, Struhl K, Gerstein M, Antonarakis SE, Fu Y, Green ED, Karaöz U, Siepel A, Taylor J, Liefer LA, Wetterstrand KA, Good PJ, Feingold EA, Guyer MS, Cooper GM, Asimenos G, Dewey CN, Hou M, Nikolaev S, Montoya-Burgos JI, Löytynoja A, Whelan S, Pardi F, Massingham T, Huang H, Zhang NR, Holmes I, Mullikin JC, Ureta-Vidal A, Paten B, Seringhaus M, Church D, Rosenbloom K, Kent WJ, Stone EA, NISC Comparative Sequencing Program., Baylor College of Medicine Human Genome Sequencing Center., Washington University Genome Sequencing Center., Broad Institute., Children's Hospital Oakland Research Institute., Batzoglou S, Goldman N, Hardison RC, Haussler D, Miller W, Sidow A, Trinklein ND, Zhang ZD, Barrera L, Stuart R, King DC, Ameur A, Enroth S, Bieda MC, Kim J, Bhinge AA, Jiang N, Liu J, Yao F, Vega VB, Lee CW, Ng P, Shahab A, Yang A, Moqtaderi Z, Zhu Z, Xu X, Squazzo S, Oberley MJ, Inman D, Singer MA, Richmond TA, Munn KJ, Rada-Iglesias A, Wallerman O, Komorowski J, Fowler JC, Couttet P, Bruce AW, Dovey OM, Ellis PD, Langford CF, Nix DA, Euskirchen G, Hartman S, Urban AE, Kraus P, Van Calcar S, Heintzman N, Kim TH, Wang K, Qu C, Hon G, Luna R, Glass CK, Rosenfeld MG, Aldred SF, Cooper SJ, Halees A, Lin JM, Shulha HP, Zhang X, Xu M, Haidar JN, Yu Y, Ruan Y, Iyer VR, Green RD, Wadelius C, Farnham PJ, Ren B, Harte RA, Hinrichs AS, Trumbower H, Clawson H, Hillman-Jackson J, Zweig AS, Smith K, Thakkapallayil A, Barber G, Kuhn RM, Karolchik D, Armengol L, Bird CP, de Bakker PI, Kern AD, Lopez-Bigas N, Martin JD, Stranger BE, Woodroffe A, Davydov E, Dimas A, Eyras E, Hallgrímsdóttir IB, Huppert J, Zody MC, Abecasis GR, Estivill X, Bouffard GG, Guan X, Hansen NF, Idol JR, Maduro VV, Maskeri B, McDowell JC, Park M, Thomas PJ, Young AC, Blakesley RW, Muzny DM, Sodergren E, Wheeler DA, Worley KC, Jiang H, Weinstock GM, Gibbs RA, Graves T, Fulton R, Mardis ER, Wilson RK, Clamp M, Cuff J, Gnerre S, Jaffe DB, Chang JL, Lindblad-Toh K, Lander ES, Koriabine M, Nefedov M, Osoegawa K, Yoshinaga Y, Zhu B, de Jong PJ</div>
<div class="citation">Nature. 2007 Jun 14; 447(7146):799-816.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P5" name="crb--P5" rid="P5" style="position: absolute; visibility: visible; width: 241px; top: 2265px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-26"><ul><li class="two_line" reference_id="R15">
<a href="/pubmed/22889292/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An encyclopedia of mouse DNA elements (Mouse ENCODE).</a><span class="one_line_source">[Genome Biol.  2012]</span><div class="alt-note">
<div class="authors">Mouse ENCODE Consortium., Stamatoyannopoulos JA, Snyder M, Hardison R, Ren B, Gingeras T, Gilbert DM, Groudine M, Bender M, Kaul R, Canfield T, Giste E, Johnson A, Zhang M, Balasundaram G, Byron R, Roach V, Sabo PJ, Sandstrom R, Stehling AS, Thurman RE, Weissman SM, Cayting P, Hariharan M, Lian J, Cheng Y, Landt SG, Ma Z, Wold BJ, Dekker J, Crawford GE, Keller CA, Wu W, Morrissey C, Kumar SA, Mishra T, Jain D, Byrska-Bishop M, Blankenberg D, Lajoie BR, Jain G, Sanyal A, Chen KB, Denas O, Taylor J, Blobel GA, Weiss MJ, Pimkin M, Deng W, Marinov GK, Williams BA, Fisher-Aylor KI, Desalvo G, Kiralusha A, Trout D, Amrhein H, Mortazavi A, Edsall L, McCleary D, Kuan S, Shen Y, Yue F, Ye Z, Davis CA, Zaleski C, Jha S, Xue C, Dobin A, Lin W, Fastuca M, Wang H, Guigo R, Djebali S, Lagarde J, Ryba T, Sasaki T, Malladi VS, Cline MS, Kirkup VM, Learned K, Rosenbloom KR, Kent WJ, Feingold EA, Good PJ, Pazin M, Lowdon RF, Adams LB</div>
<div class="citation">Genome Biol. 2012 Aug 13; 13(8):418.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P12" name="crb--P12" rid="P12" style="position: absolute; visibility: visible; width: 241px; top: 3029px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-27"><ul><li class="two_line" reference_id="R16">
<a href="/pubmed/19952138/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide dynamics of replication timing revealed by in vitro models of mouse embryogenesis.</a><span class="one_line_source">[Genome Res.  2010]</span><div class="alt-note">
<div class="authors">Hiratani I, Ryba T, Itoh M, Rathjen J, Kulik M, Papp B, Fussner E, Bazett-Jones DP, Plath K, Dalton S, Rathjen PD, Gilbert DM</div>
<div class="citation">Genome Res. 2010 Feb; 20(2):155-69.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P13" name="crb--P13" rid="P13" style="position: absolute; visibility: visible; width: 241px; top: 3454px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul>
<li class="two_line" reference_id="R13">
<a href="/pubmed/22955620/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Landscape of transcription in human cells.</a><span class="one_line_source">[Nature.  2012]</span><div class="alt-note">
<div class="authors">Djebali S, Davis CA, Merkel A, Dobin A, Lassmann T, Mortazavi A, Tanzer A, Lagarde J, Lin W, Schlesinger F, Xue C, Marinov GK, Khatun J, Williams BA, Zaleski C, Rozowsky J, Röder M, Kokocinski F, Abdelhamid RF, Alioto T, Antoshechkin I, Baer MT, Bar NS, Batut P, Bell K, Bell I, Chakrabortty S, Chen X, Chrast J, Curado J, Derrien T, Drenkow J, Dumais E, Dumais J, Duttagupta R, Falconnet E, Fastuca M, Fejes-Toth K, Ferreira P, Foissac S, Fullwood MJ, Gao H, Gonzalez D, Gordon A, Gunawardena H, Howald C, Jha S, Johnson R, Kapranov P, King B, Kingswood C, Luo OJ, Park E, Persaud K, Preall JB, Ribeca P, Risk B, Robyr D, Sammeth M, Schaffer L, See LH, Shahab A, Skancke J, Suzuki AM, Takahashi H, Tilgner H, Trout D, Walters N, Wang H, Wrobel J, Yu Y, Ruan X, Hayashizaki Y, Harrow J, Gerstein M, Hubbard T, Reymond A, Antonarakis SE, Hannon G, Giddings MC, Ruan Y, Wold B, Carninci P, Guigó R, Gingeras TR</div>
<div class="citation">Nature. 2012 Sep 6; 489(7414):101-8.</div>
</div>
</li>
<li class="two_line" reference_id="R17">
<a href="/pubmed/19920851/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> The complex eukaryotic transcriptome: unexpected pervasive transcription <mark>and</mark> novel small RNAs.</a><span class="one_line_source">[Nat Rev Genet.  2009]</span><div class="alt-note">
<div class="authors">Jacquier A</div>
<div class="citation">Nat Rev Genet. 2009 Dec; 10(12):833-44.</div>
</div>
</li>
<li class="two_line" reference_id="R18">
<a href="/pubmed/19169243/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Bidirectional promoters generate pervasive transcription in yeast.</a><span class="one_line_source">[Nature.  2009]</span><div class="alt-note">
<div class="authors">Xu Z, Wei W, Gagneur J, Perocchi F, Clauder-Münster S, Camblong J, Guffanti E, Stutz F, Huber W, Steinmetz LM</div>
<div class="citation">Nature. 2009 Feb 19; 457(7232):1033-7.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P14" name="crb--P14" rid="P14" style="position: absolute; visibility: visible; width: 241px; top: 3796px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul>
<li class="two_line" reference_id="R19">
<a href="/pubmed/22703170/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Characterization of enhancer function from genome-wide analyses.</a><span class="one_line_source">[Annu Rev Genomics Hum Genet.  2012]</span><div class="alt-note">
<div class="authors">Maston GA, Landt SG, Snyder M, Green MR</div>
<div class="citation">Annu Rev Genomics Hum Genet. 2012; 13():29-57.</div>
</div>
</li>
<li class="two_line" reference_id="R20">
<a href="/pubmed/22705667/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Genomic approaches towards finding cis-regulatory modules in animals.</a><span class="one_line_source">[Nat Rev Genet.  2012]</span><div class="alt-note">
<div class="authors">Hardison RC, Taylor J</div>
<div class="citation">Nat Rev Genet. 2012 Jun 18; 13(7):469-83.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P15" name="crb--P15" rid="P15" style="position: absolute; visibility: visible; width: 241px; top: 3962px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul><li class="two_line" reference_id="R21">
<a href="/pubmed/22955617/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The accessible chromatin landscape of the human genome.</a><span class="one_line_source">[Nature.  2012]</span><div class="alt-note">
<div class="authors">Thurman RE, Rynes E, Humbert R, Vierstra J, Maurano MT, Haugen E, Sheffield NC, Stergachis AB, Wang H, Vernot B, Garg K, John S, Sandstrom R, Bates D, Boatman L, Canfield TK, Diegel M, Dunn D, Ebersol AK, Frum T, Giste E, Johnson AK, Johnson EM, Kutyavin T, Lajoie B, Lee BK, Lee K, London D, Lotakis D, Neph S, Neri F, Nguyen ED, Qu H, Reynolds AP, Roach V, Safi A, Sanchez ME, Sanyal A, Shafer A, Simon JM, Song L, Vong S, Weaver M, Yan Y, Zhang Z, Zhang Z, Lenhard B, Tewari M, Dorschner MO, Hansen RS, Navas PA, Stamatoyannopoulos G, Iyer VR, Lieb JD, Sunyaev SR, Akey JM, Sabo PJ, Kaul R, Furey TS, Dekker J, Crawford GE, Stamatoyannopoulos JA</div>
<div class="citation">Nature. 2012 Sep 6; 489(7414):75-82.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P17" name="crb--P17" rid="P17" style="position: absolute; visibility: visible; width: 241px; top: 4359px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul><li class="two_line" reference_id="R22">
<a href="/pubmed/23526891/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">RFECS: a random-forest based algorithm for enhancer identification from chromatin state.</a><span class="one_line_source">[PLoS Comput Biol.  2013]</span><div class="alt-note">
<div class="authors">Rajagopal N, Xie W, Li Y, Wagner U, Wang W, Stamatoyannopoulos J, Ernst J, Kellis M, Ren B</div>
<div class="citation">PLoS Comput Biol. 2013; 9(3):e1002968.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P20" name="crb--P20" rid="P20" style="position: absolute; visibility: visible; width: 241px; top: 5207px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-32"><ul>
<li class="two_line" reference_id="R23">
<a href="/pubmed/22373907/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">ChromHMM: automating chromatin-state discovery <mark>and</mark> characterization.</a><span class="one_line_source">[Nat Methods.  2012]</span><div class="alt-note">
<div class="authors">Ernst J, Kellis M</div>
<div class="citation">Nat Methods. 2012 Feb 28; 9(3):215-6.</div>
</div>
</li>
<li class="two_line" reference_id="R24">
<a href="/pubmed/23221638/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Integrative annotation of chromatin elements from ENCODE data.</a><span class="one_line_source">[Nucleic Acids Res.  2013]</span><div class="alt-note">
<div class="authors">Hoffman MM, Ernst J, Wilder SP, Kundaje A, Harris RS, Libbrecht M, Giardine B, Ellenbogen PM, Bilmes JA, Birney E, Hardison RC, Dunham I, Kellis M, Noble WS</div>
<div class="citation">Nucleic Acids Res. 2013 Jan; 41(2):827-41.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P21" name="crb--P21" rid="P21" style="position: absolute; visibility: visible; width: 241px; top: 5593px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-33"><ul>
<li class="two_line" reference_id="R25">
<a href="/pubmed/22495300/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Topological domains in mammalian genomes identified by analysis of chromatin interactions.</a><span class="one_line_source">[Nature.  2012]</span><div class="alt-note">
<div class="authors">Dixon JR, Selvaraj S, Yue F, Kim A, Li Y, Shen Y, Hu M, Liu JS, Ren B</div>
<div class="citation">Nature. 2012 Apr 11; 485(7398):376-80.</div>
</div>
</li>
<li class="two_line" reference_id="R28">
<a href="/pubmed/22496629/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Replication fork polarity gradients revealed by megabase-sized U-shaped replication timing domains in human cell lines.</a><span class="one_line_source">[PLoS Comput Biol.  2012]</span><div class="alt-note">
<div class="authors">Baker A, Audit B, Chen CL, Moindrot B, Leleu A, Guilbaud G, Rappailles A, Vaillant C, Goldar A, Mongelard F, d'Aubenton-Carafa Y, Hyrien O, Thermes C, Arneodo A</div>
<div class="citation">PLoS Comput Biol. 2012; 8(4):e1002443.</div>
</div>
</li>
<li class="two_line" reference_id="R29">
<a href="/pubmed/22879376/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">3D chromatin conformation correlates <mark>with</mark> replication timing <mark>and</mark> is conserved in resting cells.</a><span class="one_line_source">[Nucleic Acids Res.  2012]</span><div class="alt-note">
<div class="authors">Moindrot B, Audit B, Klous P, Baker A, Thermes C, de Laat W, Bouvet P, Mongelard F, Arneodo A</div>
<div class="citation">Nucleic Acids Res. 2012 Oct; 40(19):9470-81.</div>
</div>
</li>
<li class="two_line" reference_id="R30">
<a href="/pubmed/22807480/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Chromatin-interaction compartment switch at developmentally regulated chromosomal domains reveals an unusual principle of chromatin folding.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2012]</span><div class="alt-note">
<div class="authors">Takebayashi S, Dileep V, Ryba T, Dennis JH, Gilbert DM</div>
<div class="citation">Proc Natl Acad Sci U S A. 2012 Jul 31; 109(31):12574-9.</div>
</div>
</li>
<li class="two_line" reference_id="R31">
<a href="/pubmed/19560427/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Shifts in replication timing actively affect histone acetylation during nucleosome reassembly.</a><span class="one_line_source">[Mol Cell.  2009]</span><div class="alt-note">
<div class="authors">Lande-Diner L, Zhang J, Cedar H</div>
<div class="citation">Mol Cell. 2009 Jun 26; 34(6):767-74.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P22" name="crb--P22" rid="P22" style="position: absolute; visibility: visible; width: 241px; top: 6019px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-34"><ul><li class="two_line" reference_id="R33">
<a href="/pubmed/22955988/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, <mark>and</mark> expression.</a><span class="one_line_source">[Genome Res.  2012]</span><div class="alt-note">
<div class="authors">Derrien T, Johnson R, Bussotti G, Tanzer A, Djebali S, Tilgner H, Guernec G, Martin D, Merkel A, Knowles DG, Lagarde J, Veeravalli L, Ruan X, Ruan Y, Lassmann T, Carninci P, Brown JB, Lipovich L, Gonzalez JM, Thomas M, Davis CA, Shiekhattar R, Gingeras TR, Hubbard TJ, Notredame C, Harrow J, Guigó R</div>
<div class="citation">Genome Res. 2012 Sep; 22(9):1775-89.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P23" name="crb--P23" rid="P23" style="position: absolute; visibility: visible; width: 241px; top: 6642px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-35"><ul>
<li class="two_line" reference_id="R34">
<a href="/pubmed/21390129/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Human-specific loss of regulatory DNA <mark>and</mark> the evolution of human-specific traits.</a><span class="one_line_source">[Nature.  2011]</span><div class="alt-note">
<div class="authors">McLean CY, Reno PL, Pollen AA, Bassan AI, Capellini TD, Guenther C, Indjeian VB, Lim X, Menke DB, Schaar BT, Wenger AM, Bejerano G, Kingsley DM</div>
<div class="citation">Nature. 2011 Mar 10; 471(7337):216-9.</div>
</div>
</li>
<li class="two_line" reference_id="R38">
<a href="/pubmed/23539138/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Gene expression drives local adaptation in humans.</a><span class="one_line_source">[Genome Res.  2013]</span><div class="alt-note">
<div class="authors">Fraser HB</div>
<div class="citation">Genome Res. 2013 Jul; 23(7):1089-96.</div>
</div>
</li>
<li class="two_line" reference_id="R39">
<a href="/pubmed/22012392/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The evolution of gene expression levels in mammalian organs.</a><span class="one_line_source">[Nature.  2011]</span><div class="alt-note">
<div class="authors">Brawand D, Soumillon M, Necsulea A, Julien P, Csárdi G, Harrigan P, Weier M, Liechti A, Aximu-Petri A, Kircher M, Albert FW, Zeller U, Khaitovich P, Grützner F, Bergmann S, Nielsen R, Pääbo S, Kaessmann H</div>
<div class="citation">Nature. 2011 Oct 19; 478(7369):343-8.</div>
</div>
</li>
<li class="two_line" reference_id="R41">
<a href="/pubmed/23258890/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The evolutionary landscape of alternative splicing in vertebrate species.</a><span class="one_line_source">[Science.  2012]</span><div class="alt-note">
<div class="authors">Barbosa-Morais NL, Irimia M, Pan Q, Xiong HY, Gueroussov S, Lee LJ, Slobodeniuc V, Kutter C, Watt S, Colak R, Kim T, Misquitta-Ali CM, Wilson MD, Kim PM, Odom DT, Frey BJ, Blencowe BJ</div>
<div class="citation">Science. 2012 Dec 21; 338(6114):1587-93.</div>
</div>
</li>
<li class="two_line" reference_id="R42">
<a href="/pubmed/16778047/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Positive natural selection in the human lineage.</a><span class="one_line_source">[Science.  2006]</span><div class="alt-note">
<div class="authors">Sabeti PC, Schaffner SF, Fry B, Lohmueller J, Varilly P, Shamovsky O, Palma A, Mikkelsen TS, Altshuler D, Lander ES</div>
<div class="citation">Science. 2006 Jun 16; 312(5780):1614-20.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P26" name="crb--P26" rid="P26" style="position: absolute; visibility: visible; width: 241px; top: 9191px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-36"><ul>
<li class="two_line" reference_id="R8">
<a href="/pubmed/21721942/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Regulatory variation within <mark>and</mark> between species.</a><span class="one_line_source">[Annu Rev Genomics Hum Genet.  2011]</span><div class="alt-note">
<div class="authors">Zheng W, Gianoulis TA, Karczewski KJ, Zhao H, Snyder M</div>
<div class="citation">Annu Rev Genomics Hum Genet. 2011; 12():327-46.</div>
</div>
</li>
<li class="two_line" reference_id="R5">
<a href="/pubmed/23911320/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Cooperativity <mark>and</mark> rapid evolution of cobound transcription factors in closely related mammals.</a><span class="one_line_source">[Cell.  2013]</span><div class="alt-note">
<div class="authors">Stefflova K, Thybert D, Wilson MD, Streeter I, Aleksic J, Karagianni P, Brazma A, Adams DJ, Talianidis I, Marioni JC, Flicek P, Odom DT</div>
<div class="citation">Cell. 2013 Aug 1; 154(3):530-40.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P27" name="crb--P27" rid="P27" style="position: absolute; visibility: visible; width: 241px; top: 9445px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-37"><ul><li class="two_line" reference_id="R43">
<a href="/pubmed/12529312/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Human-mouse alignments <mark>with</mark> BLASTZ.</a><span class="one_line_source">[Genome Res.  2003]</span><div class="alt-note">
<div class="authors">Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC, Haussler D, Miller W</div>
<div class="citation">Genome Res. 2003 Jan; 13(1):103-7.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P28" name="crb--P28" rid="P28" style="position: absolute; visibility: visible; width: 241px; top: 10069px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-38"><ul>
<li class="two_line" reference_id="R3">
<a href="/pubmed/17529977/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Tissue-specific transcriptional regulation has diverged significantly between human <mark>and</mark> mouse.</a><span class="one_line_source">[Nat Genet.  2007]</span><div class="alt-note">
<div class="authors">Odom DT, Dowell RD, Jacobsen ES, Gordon W, Danford TW, MacIsaac KD, Rolfe PA, Conboy CM, Gifford DK, Fraenkel E</div>
<div class="citation">Nat Genet. 2007 Jun; 39(6):730-2.</div>
</div>
</li>
<li class="two_line" reference_id="R14">
<a href="/pubmed/17571346/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Identification <mark>and</mark> analysis of functional elements in 1% of the human genome by the ENCODE pilot project.</a><span class="one_line_source">[Nature.  2007]</span><div class="alt-note">
<div class="authors">ENCODE Project Consortium., Birney E, Stamatoyannopoulos JA, Dutta A, Guigó R, Gingeras TR, Margulies EH, Weng Z, Snyder M, Dermitzakis ET, Thurman RE, Kuehn MS, Taylor CM, Neph S, Koch CM, Asthana S, Malhotra A, Adzhubei I, Greenbaum JA, Andrews RM, Flicek P, Boyle PJ, Cao H, Carter NP, Clelland GK, Davis S, Day N, Dhami P, Dillon SC, Dorschner MO, Fiegler H, Giresi PG, Goldy J, Hawrylycz M, Haydock A, Humbert R, James KD, Johnson BE, Johnson EM, Frum TT, Rosenzweig ER, Karnani N, Lee K, Lefebvre GC, Navas PA, Neri F, Parker SC, Sabo PJ, Sandstrom R, Shafer A, Vetrie D, Weaver M, Wilcox S, Yu M, Collins FS, Dekker J, Lieb JD, Tullius TD, Crawford GE, Sunyaev S, Noble WS, Dunham I, Denoeud F, Reymond A, Kapranov P, Rozowsky J, Zheng D, Castelo R, Frankish A, Harrow J, Ghosh S, Sandelin A, Hofacker IL, Baertsch R, Keefe D, Dike S, Cheng J, Hirsch HA, Sekinger EA, Lagarde J, Abril JF, Shahab A, Flamm C, Fried C, Hackermüller J, Hertel J, Lindemeyer M, Missal K, Tanzer A, Washietl S, Korbel J, Emanuelsson O, Pedersen JS, Holroyd N, Taylor R, Swarbreck D, Matthews N, Dickson MC, Thomas DJ, Weirauch MT, Gilbert J, Drenkow J, Bell I, Zhao X, Srinivasan KG, Sung WK, Ooi HS, Chiu KP, Foissac S, Alioto T, Brent M, Pachter L, Tress ML, Valencia A, Choo SW, Choo CY, Ucla C, Manzano C, Wyss C, Cheung E, Clark TG, Brown JB, Ganesh M, Patel S, Tammana H, Chrast J, Henrichsen CN, Kai C, Kawai J, Nagalakshmi U, Wu J, Lian Z, Lian J, Newburger P, Zhang X, Bickel P, Mattick JS, Carninci P, Hayashizaki Y, Weissman S, Hubbard T, Myers RM, Rogers J, Stadler PF, Lowe TM, Wei CL, Ruan Y, Struhl K, Gerstein M, Antonarakis SE, Fu Y, Green ED, Karaöz U, Siepel A, Taylor J, Liefer LA, Wetterstrand KA, Good PJ, Feingold EA, Guyer MS, Cooper GM, Asimenos G, Dewey CN, Hou M, Nikolaev S, Montoya-Burgos JI, Löytynoja A, Whelan S, Pardi F, Massingham T, Huang H, Zhang NR, Holmes I, Mullikin JC, Ureta-Vidal A, Paten B, Seringhaus M, Church D, Rosenbloom K, Kent WJ, Stone EA, NISC Comparative Sequencing Program., Baylor College of Medicine Human Genome Sequencing Center., Washington University Genome Sequencing Center., Broad Institute., Children's Hospital Oakland Research Institute., Batzoglou S, Goldman N, Hardison RC, Haussler D, Miller W, Sidow A, Trinklein ND, Zhang ZD, Barrera L, Stuart R, King DC, Ameur A, Enroth S, Bieda MC, Kim J, Bhinge AA, Jiang N, Liu J, Yao F, Vega VB, Lee CW, Ng P, Shahab A, Yang A, Moqtaderi Z, Zhu Z, Xu X, Squazzo S, Oberley MJ, Inman D, Singer MA, Richmond TA, Munn KJ, Rada-Iglesias A, Wallerman O, Komorowski J, Fowler JC, Couttet P, Bruce AW, Dovey OM, Ellis PD, Langford CF, Nix DA, Euskirchen G, Hartman S, Urban AE, Kraus P, Van Calcar S, Heintzman N, Kim TH, Wang K, Qu C, Hon G, Luna R, Glass CK, Rosenfeld MG, Aldred SF, Cooper SJ, Halees A, Lin JM, Shulha HP, Zhang X, Xu M, Haidar JN, Yu Y, Ruan Y, Iyer VR, Green RD, Wadelius C, Farnham PJ, Ren B, Harte RA, Hinrichs AS, Trumbower H, Clawson H, Hillman-Jackson J, Zweig AS, Smith K, Thakkapallayil A, Barber G, Kuhn RM, Karolchik D, Armengol L, Bird CP, de Bakker PI, Kern AD, Lopez-Bigas N, Martin JD, Stranger BE, Woodroffe A, Davydov E, Dimas A, Eyras E, Hallgrímsdóttir IB, Huppert J, Zody MC, Abecasis GR, Estivill X, Bouffard GG, Guan X, Hansen NF, Idol JR, Maduro VV, Maskeri B, McDowell JC, Park M, Thomas PJ, Young AC, Blakesley RW, Muzny DM, Sodergren E, Wheeler DA, Worley KC, Jiang H, Weinstock GM, Gibbs RA, Graves T, Fulton R, Mardis ER, Wilson RK, Clamp M, Cuff J, Gnerre S, Jaffe DB, Chang JL, Lindblad-Toh K, Lander ES, Koriabine M, Nefedov M, Osoegawa K, Yoshinaga Y, Zhu B, de Jong PJ</div>
<div class="citation">Nature. 2007 Jun 14; 447(7146):799-816.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P29" name="crb--P29" rid="P29" style="position: absolute; visibility: visible; width: 241px; top: 10785px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-39"><ul>
<li class="two_line" reference_id="R44">
<a href="/pubmed/17567996/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Finding cis-regulatory elements using comparative genomics: some lessons from ENCODE data.</a><span class="one_line_source">[Genome Res.  2007]</span><div class="alt-note">
<div class="authors">King DC, Taylor J, Zhang Y, Cheng Y, Lawson HA, Martin J, ENCODE groups for Transcriptional Regulation <mark>and</mark> Multispecies Sequence Analysis., Chiaromonte F, Miller W, Hardison RC</div>
<div class="citation">Genome Res. 2007 Jun; 17(6):775-86.</div>
</div>
</li>
<li class="two_line" reference_id="R2">
<a href="/pubmed/12466850/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Initial sequencing <mark>and</mark> comparative analysis of the mouse genome.</a><span class="one_line_source">[Nature.  2002]</span><div class="alt-note">
<div class="authors">Mouse Genome Sequencing Consortium., Waterston RH, Lindblad-Toh K, Birney E, Rogers J, Abril JF, Agarwal P, Agarwala R, Ainscough R, Alexandersson M, An P, Antonarakis SE, Attwood J, Baertsch R, Bailey J, Barlow K, Beck S, Berry E, Birren B, Bloom T, Bork P, Botcherby M, Bray N, Brent MR, Brown DG, Brown SD, Bult C, Burton J, Butler J, Campbell RD, Carninci P, Cawley S, Chiaromonte F, Chinwalla AT, Church DM, Clamp M, Clee C, Collins FS, Cook LL, Copley RR, Coulson A, Couronne O, Cuff J, Curwen V, Cutts T, Daly M, David R, Davies J, Delehaunty KD, Deri J, Dermitzakis ET, Dewey C, Dickens NJ, Diekhans M, Dodge S, Dubchak I, Dunn DM, Eddy SR, Elnitski L, Emes RD, Eswara P, Eyras E, Felsenfeld A, Fewell GA, Flicek P, Foley K, Frankel WN, Fulton LA, Fulton RS, Furey TS, Gage D, Gibbs RA, Glusman G, Gnerre S, Goldman N, Goodstadt L, Grafham D, Graves TA, Green ED, Gregory S, Guigó R, Guyer M, Hardison RC, Haussler D, Hayashizaki Y, Hillier LW, Hinrichs A, Hlavina W, Holzer T, Hsu F, Hua A, Hubbard T, Hunt A, Jackson I, Jaffe DB, Johnson LS, Jones M, Jones TA, Joy A, Kamal M, Karlsson EK, Karolchik D, Kasprzyk A, Kawai J, Keibler E, Kells C, Kent WJ, Kirby A, Kolbe DL, Korf I, Kucherlapati RS, Kulbokas EJ, Kulp D, Landers T, Leger JP, Leonard S, Letunic I, Levine R, Li J, Li M, Lloyd C, Lucas S, Ma B, Maglott DR, Mardis ER, Matthews L, Mauceli E, Mayer JH, McCarthy M, McCombie WR, McLaren S, McLay K, McPherson JD, Meldrim J, Meredith B, Mesirov JP, Miller W, Miner TL, Mongin E, Montgomery KT, Morgan M, Mott R, Mullikin JC, Muzny DM, Nash WE, Nelson JO, Nhan MN, Nicol R, Ning Z, Nusbaum C, O'Connor MJ, Okazaki Y, Oliver K, Overton-Larty E, Pachter L, Parra G, Pepin KH, Peterson J, Pevzner P, Plumb R, Pohl CS, Poliakov A, Ponce TC, Ponting CP, Potter S, Quail M, Reymond A, Roe BA, Roskin KM, Rubin EM, Rust AG, Santos R, Sapojnikov V, Schultz B, Schultz J, Schwartz MS, Schwartz S, Scott C, Seaman S, Searle S, Sharpe T, Sheridan A, Shownkeen R, Sims S, Singer JB, Slater G, Smit A, Smith DR, Spencer B, Stabenau A, Stange-Thomann N, Sugnet C, Suyama M, Tesler G, Thompson J, Torrents D, Trevaskis E, Tromp J, Ucla C, Ureta-Vidal A, Vinson JP, Von Niederhausern AC, Wade CM, Wall M, Weber RJ, Weiss RB, Wendl MC, West AP, Wetterstrand K, Wheeler R, Whelan S, Wierzbowski J, Willey D, Williams S, Wilson RK, Winter E, Worley KC, Wyman D, Yang S, Yang SP, Zdobnov EM, Zody MC, Lander ES</div>
<div class="citation">Nature. 2002 Dec 5; 420(6915):520-62.</div>
</div>
</li>
<li class="two_line" reference_id="R45">
<a href="/pubmed/18663365/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> The functional repertoires of metazoan genomes.</a><span class="one_line_source">[Nat Rev Genet.  2008]</span><div class="alt-note">
<div class="authors">Ponting CP</div>
<div class="citation">Nat Rev Genet. 2008 Sep; 9(9):689-98.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P30" name="crb--P30" rid="P30" style="position: absolute; visibility: visible; width: 241px; top: 11151px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-40"><ul>
<li class="two_line" reference_id="R46">
<a href="/pubmed/18682548/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Evolution of the mammalian transcription factor binding repertoire via transposable elements.</a><span class="one_line_source">[Genome Res.  2008]</span><div class="alt-note">
<div class="authors">Bourque G, Leong B, Vega VB, Chen X, Lee YL, Srinivasan KG, Chew JL, Ruan Y, Wei CL, Ng HH, Liu ET</div>
<div class="citation">Genome Res. 2008 Nov; 18(11):1752-62.</div>
</div>
</li>
<li class="two_line" reference_id="R48">
<a href="/pubmed/23675311/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The majority of primate-specific regulatory sequences are derived from transposable elements.</a><span class="one_line_source">[PLoS Genet.  2013]</span><div class="alt-note">
<div class="authors">Jacques PÉ, Jeyakani J, Bourque G</div>
<div class="citation">PLoS Genet. 2013 May; 9(5):e1003504.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P33" name="crb--P33" rid="P33" style="position: absolute; visibility: visible; width: 241px; top: 12065px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-41"><ul><li class="two_line" reference_id="R49">
<a href="/pubmed/23473601/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Modification of enhancer chromatin: what, how, <mark>and</mark> why?</a><span class="one_line_source">[Mol Cell.  2013]</span><div class="alt-note">
<div class="authors">Calo E, Wysocka J</div>
<div class="citation">Mol Cell. 2013 Mar 7; 49(5):825-37.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P36" name="crb--P36" rid="P36" style="position: absolute; visibility: visible; width: 241px; top: 13347px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-42"><ul>
<li class="two_line" reference_id="R50">
<a href="/pubmed/20888037/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Systematic protein location mapping reveals five principal chromatin types in Drosophila cells.</a><span class="one_line_source">[Cell.  2010]</span><div class="alt-note">
<div class="authors">Filion GJ, van Bemmel JG, Braunschweig U, Talhout W, Kind J, Ward LD, Brugman W, de Castro IJ, Kerkhoven RM, Bussemaker HJ, van Steensel B</div>
<div class="citation">Cell. 2010 Oct 15; 143(2):212-24.</div>
</div>
</li>
<li class="two_line" reference_id="R12">
<a href="/pubmed/22955616/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An integrated encyclopedia of DNA elements in the human genome.</a><span class="one_line_source">[Nature.  2012]</span><div class="alt-note">
<div class="authors">ENCODE Project Consortium.</div>
<div class="citation">Nature. 2012 Sep 6; 489(7414):57-74.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P37" name="crb--P37" rid="P37" style="position: absolute; visibility: visible; width: 241px; top: 13711px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-43"><ul>
<li class="two_line" reference_id="R51">
<a href="/pubmed/21258342/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Chromatin accessibility pre-determines glucocorticoid receptor binding patterns.</a><span class="one_line_source">[Nat Genet.  2011]</span><div class="alt-note">
<div class="authors">John S, Sabo PJ, Thurman RE, Sung MH, Biddie SC, Johnson TA, Hager GL, Stamatoyannopoulos JA</div>
<div class="citation">Nat Genet. 2011 Mar; 43(3):264-8.</div>
</div>
</li>
<li class="two_line" reference_id="R53">
<a href="/pubmed/21795386/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Dynamics of the epigenetic landscape during erythroid differentiation after GATA1 restoration.</a><span class="one_line_source">[Genome Res.  2011]</span><div class="alt-note">
<div class="authors">Wu W, Cheng Y, Keller CA, Ernst J, Kumar SA, Mishra T, Morrissey C, Dorman CM, Chen KB, Drautz D, Giardine B, Shibata Y, Song L, Pimkin M, Crawford GE, Furey TS, Kellis M, Miller W, Taylor J, Schuster SC, Zhang Y, Chiaromonte F, Blobel GA, Weiss MJ, Hardison RC</div>
<div class="citation">Genome Res. 2011 Oct; 21(10):1659-71.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P38" name="crb--P38" rid="P38" style="position: absolute; visibility: visible; width: 241px; top: 14532px; ">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-44"><ul><li class="two_line" reference_id="R54">
<a href="/pubmed/24170599/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Integrating <mark>and</mark> mining the chromatin landscape of cell-type specificity using self-organizing maps.</a><span class="one_line_source">[Genome Res.  2013]</span><div class="alt-note">
<div class="authors">Mortazavi A, Pepke S, Jansen C, Marinov GK, Ernst J, Kellis M, Hardison RC, Myers RM, Wold BJ</div>
<div class="citation">Genome Res. 2013 Dec; 23(12):2136-48.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P40" name="crb--P40" rid="P40">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-45"><ul>
<li class="expanded" reference_id="R55">
<a href="/pubmed/18842067/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Global reorganization of replication domains during embryonic stem cell differentiation.</a><span class="one_line_source">[PLoS Biol.  2008]</span><div class="alt-note">
<div class="authors">Hiratani I, Ryba T, Itoh M, Yokochi T, Schwaiger M, Chang CW, Lyou Y, Townes TM, Schübeler D, Gilbert DM</div>
<div class="citation">PLoS Biol. 2008 Oct 7; 6(10):e245.</div>
</div>
</li>
<li class="expanded" reference_id="R16">
<a href="/pubmed/19952138/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide dynamics of replication timing revealed by in vitro models of mouse embryogenesis.</a><span class="one_line_source">[Genome Res.  2010]</span><div class="alt-note">
<div class="authors">Hiratani I, Ryba T, Itoh M, Rathjen J, Kulik M, Papp B, Fussner E, Bazett-Jones DP, Plath K, Dalton S, Rathjen PD, Gilbert DM</div>
<div class="citation">Genome Res. 2010 Feb; 20(2):155-69.</div>
</div>
</li>
<li class="expanded" reference_id="R56">
<a href="/pubmed/19966280/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Sequencing newly replicated DNA reveals widespread plasticity in human replication timing.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2010]</span><div class="alt-note">
<div class="authors">Hansen RS, Thomas S, Sandstrom R, Canfield TK, Thurman RE, Weaver M, Dorschner MO, Gartler SM, Stamatoyannopoulos JA</div>
<div class="citation">Proc Natl Acad Sci U S A. 2010 Jan 5; 107(1):139-44.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P41" name="crb--P41" rid="P41">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-46"><ul>
<li class="expanded" reference_id="R26">
<a href="/pubmed/20430782/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Evolutionarily conserved replication timing profiles predict long-range chromatin interactions <mark>and</mark> distinguish closely related cell types.</a><span class="one_line_source">[Genome Res.  2010]</span><div class="alt-note">
<div class="authors">Ryba T, Hiratani I, Lu J, Itoh M, Kulik M, Zhang J, Schulz TC, Robins AJ, Dalton S, Gilbert DM</div>
<div class="citation">Genome Res. 2010 Jun; 20(6):761-70.</div>
</div>
</li>
<li class="expanded" reference_id="R27">
<a href="/pubmed/20617169/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Comparative analysis of DNA replication timing reveals conserved large-scale chromosomal architecture.</a><span class="one_line_source">[PLoS Genet.  2010]</span><div class="alt-note">
<div class="authors">Yaffe E, Farkash-Amar S, Polten A, Yakhini Z, Tanay A, Simon I</div>
<div class="citation">PLoS Genet. 2010 Jul 1; 6(7):e1001011.</div>
</div>
</li>
<li class="expanded" reference_id="R57">
<a href="/pubmed/22028635/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Replication timing: a fingerprint for cell identity <mark>and</mark> pluripotency.</a><span class="one_line_source">[PLoS Comput Biol.  2011]</span><div class="alt-note">
<div class="authors">Ryba T, Hiratani I, Sasaki T, Battaglia D, Kulik M, Zhang J, Dalton S, Gilbert DM</div>
<div class="citation">PLoS Comput Biol. 2011 Oct; 7(10):e1002225.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P43" name="crb--P43" rid="P43">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-47"><ul><li class="expanded" reference_id="R12">
<a href="/pubmed/22955616/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An integrated encyclopedia of DNA elements in the human genome.</a><span class="one_line_source">[Nature.  2012]</span><div class="alt-note">
<div class="authors">ENCODE Project Consortium.</div>
<div class="citation">Nature. 2012 Sep 6; 489(7414):57-74.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P44" name="crb--P44" rid="P44">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-48"><ul>
<li class="expanded" reference_id="R3">
<a href="/pubmed/17529977/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Tissue-specific transcriptional regulation has diverged significantly between human <mark>and</mark> mouse.</a><span class="one_line_source">[Nat Genet.  2007]</span><div class="alt-note">
<div class="authors">Odom DT, Dowell RD, Jacobsen ES, Gordon W, Danford TW, MacIsaac KD, Rolfe PA, Conboy CM, Gifford DK, Fraenkel E</div>
<div class="citation">Nat Genet. 2007 Jun; 39(6):730-2.</div>
</div>
</li>
<li class="expanded" reference_id="R4">
<a href="/pubmed/20378774/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Five-vertebrate ChIP-seq reveals the evolutionary dynamics of transcription factor binding.</a><span class="one_line_source">[Science.  2010]</span><div class="alt-note">
<div class="authors">Schmidt D, Wilson MD, Ballester B, Schwalie PC, Brown GD, Marshall A, Kutter C, Watt S, Martinez-Jimenez CP, Mackay S, Talianidis I, Flicek P, Odom DT</div>
<div class="citation">Science. 2010 May 21; 328(5981):1036-40.</div>
</div>
</li>
<li class="expanded" reference_id="R7">
<a href="/pubmed/17690298/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Divergence of transcription factor binding sites across related yeast species.</a><span class="one_line_source">[Science.  2007]</span><div class="alt-note">
<div class="authors">Borneman AR, Gianoulis TA, Zhang ZD, Yu H, Rozowsky J, Seringhaus MR, Wang LY, Gerstein M, Snyder M</div>
<div class="citation">Science. 2007 Aug 10; 317(5839):815-9.</div>
</div>
</li>
<li class="expanded" reference_id="R58">
<a href="/pubmed/17040121/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Large-scale turnover of functional transcription factor binding sites in Drosophila.</a><span class="one_line_source">[PLoS Comput Biol.  2006]</span><div class="alt-note">
<div class="authors">Moses AM, Pollard DA, Nix DA, Iyer VN, Li XY, Biggin MD, Eisen MB</div>
<div class="citation">PLoS Comput Biol. 2006 Oct; 2(10):e130.</div>
</div>
</li>
<li class="expanded" reference_id="R59">
<a href="/pubmed/14978070/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Of mice <mark>and</mark> not men: differences between mouse <mark>and</mark> human immunology.</a><span class="one_line_source">[J Immunol.  2004]</span><div class="alt-note">
<div class="authors">Mestas J, Hughes CC</div>
<div class="citation">J Immunol. 2004 Mar 1; 172(5):2731-8.</div>
</div>
</li>
<li class="expanded" reference_id="R60">
<a href="/pubmed/23382184/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Conservation <mark>and</mark> divergence in the transcriptional programs of the human <mark>and</mark> mouse immune systems.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2013]</span><div class="alt-note">
<div class="authors">Shay T, Jojic V, Zuk O, Rothamel K, Puyraimond-Zemmour D, Feng T, Wakamatsu E, Benoist C, Koller D, Regev A, ImmGen Consortium.</div>
<div class="citation">Proc Natl Acad Sci U S A. 2013 Feb 19; 110(8):2946-51.</div>
</div>
</li>
<li class="expanded" reference_id="R61">
<a href="/pubmed/23401516/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genomic responses in mouse models poorly mimic human inflammatory diseases.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2013]</span><div class="alt-note">
<div class="authors">Seok J, Warren HS, Cuenca AG, Mindrinos MN, Baker HV, Xu W, Richards DR, McDonald-Smith GP, Gao H, Hennessy L, Finnerty CC, López CM, Honari S, Moore EE, Minei JP, Cuschieri J, Bankey PE, Johnson JL, Sperry J, Nathens AB, Billiar TR, West MA, Jeschke MG, Klein MB, Gamelli RL, Gibran NS, Brownstein BH, Miller-Graziano C, Calvano SE, Mason PH, Cobb JP, Rahme LG, Lowry SF, Maier RV, Moldawer LL, Herndon DN, Davis RW, Xiao W, Tompkins RG, Inflammation <mark>and</mark> Host Response to Injury, Large Scale Collaborative Research Program.</div>
<div class="citation">Proc Natl Acad Sci U S A. 2013 Feb 26; 110(9):3507-12.</div>
</div>
</li>
<li class="expanded" reference_id="R62">
<a href="/pubmed/12826024/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic control of the innate immune response.</a><span class="one_line_source">[BMC Immunol.  2003]</span><div class="alt-note">
<div class="authors">Wells CA, Ravasi T, Faulkner GJ, Carninci P, Okazaki Y, Hayashizaki Y, Sweet M, Wainwright BJ, Hume DA</div>
<div class="citation">BMC Immunol. 2003 Jun 26; 4():5.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P45" name="crb--P45" rid="P45">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-49"><ul>
<li class="expanded" reference_id="R63">
<a href="/pubmed/19377475/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The regulated retrotransposon transcriptome of mammalian cells.</a><span class="one_line_source">[Nat Genet.  2009]</span><div class="alt-note">
<div class="authors">Faulkner GJ, Kimura Y, Daub CO, Wani S, Plessy C, Irvine KM, Schroder K, Cloonan N, Steptoe AL, Lassmann T, Waki K, Hornig N, Arakawa T, Takahashi H, Kawai J, Forrest AR, Suzuki H, Hayashizaki Y, Hume DA, Orlando V, Grimmond SM, Carninci P</div>
<div class="citation">Nat Genet. 2009 May; 41(5):563-71.</div>
</div>
</li>
<li class="expanded" reference_id="R64">
<a href="/pubmed/23664764/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Epigenomic analysis of multilineage differentiation of human embryonic stem cells.</a><span class="one_line_source">[Cell.  2013]</span><div class="alt-note">
<div class="authors">Xie W, Schultz MD, Lister R, Hou Z, Rajagopal N, Ray P, Whitaker JW, Tian S, Hawkins RD, Leung D, Yang H, Wang T, Lee AY, Swanson SA, Zhang J, Zhu Y, Kim A, Nery JR, Urich MA, Kuan S, Yen CA, Klugman S, Yu P, Suknuntha K, Propson NE, Chen H, Edsall LE, Wagner U, Li Y, Ye Z, Kulkarni A, Xuan Z, Chung WY, Chi NC, Antosiewicz-Bourget JE, Slukvin I, Stewart R, Zhang MQ, Wang W, Thomson JA, Ecker JR, Ren B</div>
<div class="citation">Cell. 2013 May 23; 153(5):1134-48.</div>
</div>
</li>
<li class="expanded" reference_id="R65">
<a href="/pubmed/24681886/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The retrovirus HERVH is a long noncoding RNA required for human embryonic stem cell identity.</a><span class="one_line_source">[Nat Struct Mol Biol.  2014]</span><div class="alt-note">
<div class="authors">Lu X, Sachs F, Ramsay L, Jacques PÉ, Göke J, Bourque G, Ng HH</div>
<div class="citation">Nat Struct Mol Biol. 2014 Apr; 21(4):423-5.</div>
</div>
</li>
<li class="expanded" reference_id="R66">
<a href="/pubmed/24777452/" ref="reftype=pubmed&amp;article-id=4266106&amp;issue-id=246497&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Deep transcriptome profiling of mammalian stem cells supports a regulatory role for retrotransposons in pluripotency maintenance.</a><span class="one_line_source">[Nat Genet.  2014]</span><div class="alt-note">
<div class="authors">Fort A, Hashimoto K, Yamada D, Salimullah M, Keya CA, Saxena A, Bonetti A, Voineagu I, Bertin N, Kratz A, Noro Y, Wong CH, de Hoon M, Andersson R, Sandelin A, Suzuki H, Wei CL, Koseki H, FANTOM Consortium., Hasegawa Y, Forrest AR, Carninci P</div>
<div class="citation">Nat Genet. 2014 Jun; 46(6):558-66.</div>
</div>
</li>
</ul></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="">Support Center</a>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2017-08-22T17:55:43-04:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.45&amp;Host=ptpmc101&amp;ncbi_phid=8A1A484999CA71610000000000F300CC&amp;ncbi_session=F4FC36EE99CA85E1_0189SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4266106%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies <mark>and</mark> Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 66px; left: 299px; width: 421px; z-index: 1001; "><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block; "><a class="ui-ncbiautocomplete-link-pref" style="display: none; ">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block; ">Turn off</a></div></div><div style="top: -100px; left: -100px; display: none; " aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div><script async="1" src="https://www.google-analytics.com/analytics.js"></script>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript>&lt;img alt="statistics" src="/stat?jsdisabled=true&amp;amp;ncbi_db=pmc&amp;amp;ncbi_pdid=article&amp;amp;ncbi_acc=&amp;amp;ncbi_domain=nihpa&amp;amp;ncbi_report=record&amp;amp;ncbi_type=fulltext&amp;amp;ncbi_objectid=&amp;amp;ncbi_pcid=/articles/PMC4266106/&amp;amp;ncbi_app=pmc" /&gt;</noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- F4FC36EE99CA85E1_0189SID /projects/PMC/PMCViewer@4.45 ptpmc101 v4.1.r526566 Fri, Feb 03 2017 14:27:04 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4158155/js/3879255/4121861/3818874/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none; " tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block; " class="ui-dialog-content ui-widget-content">  <p>We are experimenting <mark>with</mark> display styles <mark>that</mark> make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems <mark>with</mark> the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><ul class="print-log"><li></li></ul></body></html>